{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os, sys, re, random, time\n",
    "import requests\n",
    "import collections\n",
    "from Bio import Entrez, Medline\n",
    "from com.ibm.watson import DESKTOP, DOCS_DIR\n",
    "import com.ibm.watson.utils.pubmed_utils as pu\n",
    "import urllib\n",
    "import json\n",
    "import pandas as pd\n",
    "from bs4 import BeautifulSoup as bs\n",
    "import pymongo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "base_dir = os.path.join(DESKTOP, \"Levels_Of_Evidence\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "outcome_terms = [\"[Rr]esponse[s]?\", \"[Ss]urvival\", \"[Oo]verall [Ss]urvival\", \"[Pp]artial [Rr]esponse[s]*\", \n",
    "                 \"[Ss]urvival [Rr]ate\", \"[Oo]utcome\", \"PFS\", \"OS\", \"ORR\", \"TTP\", \"RECIST\", \"[pP]rogression\", \"EFS\"]\n",
    "outcome_terms_re = re.compile(\"|\".join(outcome_terms))\n",
    "cancer_terms = [\"cancer\", \"tumor\", \"tumour\", \"lymphoma\", \"leukemia\", \"myeloma\", \"carcinoma\", \n",
    "                \"adenoma\", \"sarcoma\", \"melanoma\"]\n",
    "cancer_terms_re = re.compile(\"|\".join(cancer_terms), re.IGNORECASE)\n",
    "#cancer_terms = \"cancer OR tumor OR tumour OR lymphoma OR leukemia OR myeloma OR carcinoma OR adenoma OR sarcoma\"\n",
    "#patient_terms_re = re.compile(\"[mM]en|[wW]omen|[pP]atient[s]?\")\n",
    "#clinical, trial, demographics, phase IGNORE\n",
    "#remove \"Review\", use Clinical Trial, Case Reports\n",
    "Coding_Substitutions = \"([c]\\.)(\\d+)?[-+*]?\\d+[AGTC]>[AGTC](?!\\w)\"\n",
    "Coding_Deletions = \"[c]\\.\\S+(\\d+|\\?)del[AGCT]*|(c\\.)((\\d+([_+-]\\d+)?)|(\\(.*\\))_\\(.*\\))del[ACTG]*\"\n",
    "Coding_Duplications = \"[gc]\\.\\S+(\\d+|\\?)dup[AGCT]*|([cg]\\.)(\\d+)?(_\\d+)?(\\(\\S*\\)_\\(\\S*\\))?dup[AGCT]*\"\n",
    "Coding_Indels = \"(c\\.)(((\\d+)|((\\d+_\\d+)))+(delins)[AGTC]+|(((\\d+)|(\\d+_\\d+))del[AGCT]+ins[AGTC]+)|(\\[\\d+[AGCT]+>[AGCT]+; \\d+del[AGCT]+\\]))\"\n",
    "Coding_Inversions = \"c\\.\\d+_\\d+inv\"\n",
    "Coding_Conversions = \"g.\\d+_\\d+con(.*):g\\.\\d+_\\d+\"\n",
    "Coding_Translocations = \"t\\((\\d+|[XY]);(\\d+|[XY])\\)\\([pq]\\d+(\\.\\d+)?;[pq]\\d+(\\.\\d+)?\\)(\\(c\\.\\d+\\+\\d+_\\d+\\+\\d+\\))\"\n",
    "Coding_Insertions = \"c\\.((\\d+_\\d+)|\\d+\\+\\d+_\\d+\\+\\d+)ins(([AGTC]+))\"\n",
    "Missense_Alteration = \"([p]\\.)?[A-Z][a-z]{0,2}\\d{2,4}[A-Z][a-z]{0,2}\"\n",
    "Nonsense_Alteration = \"([p]\\.)?[A-Z][a-z]{0,2}\\d{2,4}(\\*|Ter)(?!\\w)\"\n",
    "Upstream_Translation_Initiation = \"p\\.\\S+?\\-\\d+\"\n",
    "Translation_Termination_Codon = \"p\\.\\*\\S+\\*(\\d+|\\?)\"\n",
    "Deletions_Without_Nonsense = \"p\\.\\S+(\\d+|\\?)del\"\n",
    "Duplications_Protein = \"p\\.\\S+(\\d+|\\?)dup\"\n",
    "Insertions_Protein = \"p\\.\\S+(\\d+|\\?)ins\\w+\"\n",
    "Translocations = \"t\\(\\S+p\\.\\S+\\s\\S+\"\n",
    "Complex_Rearrangements = \"p\\.\\S+(\\d+|\\?)delins\\w+\"\n",
    "\n",
    "all_variant_patterns = [Coding_Substitutions, Coding_Deletions, Coding_Duplications, Coding_Indels, Coding_Inversions,\n",
    "                       Coding_Conversions, Coding_Translocations, Coding_Insertions, Missense_Alteration,\n",
    "                       Nonsense_Alteration, Upstream_Translation_Initiation, Translation_Termination_Codon,\n",
    "                       Deletions_Without_Nonsense, Duplications_Protein, Insertions_Protein, Translocations,\n",
    "                       Complex_Rearrangements]\n",
    "all_variant_regex = [re.compile(x) for x in all_variant_patterns]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "client = pymongo.MongoClient()\n",
    "db = client[\"LOE\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def quote(lst):\n",
    "    return [ \"\\\"\" + x.strip() + \"\\\"\" for x in lst ] if isinstance(lst, list) else \"\\\"\" + lst + \"\\\"\"\n",
    "\n",
    "def add_fieldname(lst, name):\n",
    "    return [ x + \"[%s]\" % name for x in lst ] if isinstance(lst, list) else lst + \"[%s]\" % name\n",
    "\n",
    "def make_query(field_name, gene_symbol, therapy_id=None, cancer_umls_cui=None, variant=None, variant_type=None):\n",
    "    OR = \" OR \"\n",
    "    AND = \" AND \"\n",
    "    gene = db[\"GENE\"].find_one({\"SYMBOL\" : gene_symbol})\n",
    "    gene_syns = quote([gene[\"SYMBOL\"]] + gene[\"SYNS\"])\n",
    "    gene_syns = add_fieldname(gene_syns, field_name)\n",
    "    \n",
    "    therapy = db[\"THERAPY\"].find_one({\"THERAPY_ID\" : therapy_id})\n",
    "    therapy_syns = quote([therapy[\"NAME\"]] + therapy[\"SYNS\"])\n",
    "    therapy_syns = add_fieldname(therapy_syns, field_name)\n",
    "    \n",
    "    if cancer_umls_cui is not None:\n",
    "        cancer = db[\"CANCER\"].find_one({\"UMLS_CUI\" : cancer_umls_cui})\n",
    "        cancer_syns = quote([cancer[\"NAME\"]] + cancer[\"SYNS\"]) \n",
    "        cancer_syns = add_fieldname(cancer_syns, field_name)\n",
    "    else:\n",
    "        cancer_syns = None\n",
    "    \n",
    "    if variant:\n",
    "        variant = [add_fieldname(quote(variant), field_name)]\n",
    "        \n",
    "    if variant_type:\n",
    "        variant_type = [add_fieldname(quote(variant_type), field_name)]\n",
    "    \n",
    "    query_components = [ gene_syns, therapy_syns, cancer_syns, variant, variant_type ]\n",
    "    query_components = [\"(\" + OR.join(x) + \")\" for x in query_components if x is not None]\n",
    "    return AND.join(query_components)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def search(gene_symbol, therapy_id, therapy_name, variant, variant_type):\n",
    "    pubmed_query = make_query(\"Title/Abstract\", gene_symbol, therapy_id, therapy_name, variant, variant_type)\n",
    "    pmc_query = make_query(\"Body - All Words\", gene_symbol, therapy_id, therapy_name, variant, variant_type)\n",
    "    print \"Pubmed Query: \"\n",
    "    print pubmed_query\n",
    "    print \"PMC Query: \"\n",
    "    print pmc_query\n",
    "    pubmed_ids = pu.search_pubmed(pubmed_query, retmax=100000)\n",
    "    pmc_ids = pu.search_pmc(pmc_query, retmax=100000)\n",
    "    converted = pu.convert_pmc_to_pmid(pmc_ids)\n",
    "    union = set(converted.values()).union(set(pubmed_ids))\n",
    "    return (union, converted)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_pmc_text(pmc_id):\n",
    "    print \"PMC ID: \" + pmc_id\n",
    "    html = requests.get(\"https://www.ncbi.nlm.nih.gov/pmc/articles/%s/\" % pmc_id).text\n",
    "    soup = bs(html,'html.parser')\n",
    "    sections = soup.find_all(\"div\", class_=\"sec\")\n",
    "    components = {\"abstract\" : \"\", \"body\" : \"\"}\n",
    "    abs_re = re.compile(\"abstract\", re.IGNORECASE)\n",
    "    ref_re = re.compile(\"references\", re.IGNORECASE)\n",
    "\n",
    "    for section in sections:\n",
    "        sec_title = section.find(\"h2\")\n",
    "        if sec_title:\n",
    "            title = sec_title.get_text()\n",
    "            parent = sec_title.parent\n",
    "            if abs_re.search(title):\n",
    "                components[\"abstract\"] = parent.get_text()\n",
    "            elif not ref_re.search(title):\n",
    "                components[\"body\"] += parent.get_text()\n",
    "\n",
    "    return components\n",
    "\n",
    "def get_IBM_pmc_text(pmc_id):\n",
    "    content_url = \"https://watsonpow01.rch.stglabs.ibm.com/services/content/api/v1/content/pubmed/%s/%s?user_id=genomics&token=w4genom&version=2017-03-01\"\n",
    "    response = requests.get(content_url % (\"pmid\", pmc_id)).json()\n",
    "    return response"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{u'body': u'Introduction\\nThe development of numerous targeted small molecule inhibitors represents an important and evolving new approach to cancer therapy. However, as tumours often have defects in multiple oncogenic signalling pathways, single agent anti-tumour activity is modest, and thus combinations of targeted agents are being investigated. Specifically, the MAPK pathway, a major proliferative pathway, and the PI3K pathway, a major survival pathway, are frequently activated in cancer and are being concomitantly targeted.\\n\\n\\nMany MEK inhibitors, such as PD 0325901 and selumetinib (AZD6244), have been developed to target the MAPK pathway and have shown potent growth inhibitory activity in experimental systems [1\\u20134]. A novel orally available small molecule allosteric MEK inhibitor WX-554 (UCB1366554), which potently inhibits MEK1 and MEK2 with a half maximal inhibitory concentration (IC50) of 4.7 and 11\\xa0nM, respectively, has been developed by Wilex and UCB Celltech. WX-554 has demonstrated marked inhibition of ERK1/2 phosphorylation in HT29 cells, and growth inhibition in a range of cell lines in vitro, and tumour growth delay or stasis in vivo, with increased sensitivity in BRAF or RAS mutant cells and tumours [5]. WX-554 has been shown to be safe and tolerable in a dose escalation study in healthy volunteers [6], and a recent Phase I study indicated that WX-554 has very good bioavailability and that it is able to inhibit MEK signal transduction in a dose-dependent manner [3, 7, 8]. WX-554 was investigated in a Phase Ib/II dose escalation study to determine safety and pharmacokinetics/pharmacodynamics in patients with solid tumours [7, 9], which was terminated due to business, rather than clinical, reasons.\\n\\n\\nTargeting the PI3K/AKT pathway has been investigated with a range of PI3K inhibitors which can be isoform-specific inhibitors, pan class I inhibitors (e.g. pictilisib, GDC-0941) or dual PI3K/mTOR inhibitors (e.g. dactolisib, NVP-BEZ235), and promising anti-cancer efficacy has been reported in pre-clinical models [10\\u201316]. Wilex and UCB Celltech have also developed a novel small molecule pan class I PI3K inhibitor, WX-037 (UCB1370037), from an indole series. WX-037 is a potent inhibitor of the \\u03b1 and \\u03b4 isoforms of PI3K (IC50\\xa0=\\xa04.1 and 2.4\\xa0nM, respectively) and a weaker inhibitor of the \\u03b2 and \\u03b3 isoforms of PI3K and DNA-PK (IC50\\xa0=\\xa069, 36 and 28\\xa0nM, respectively) with no detectable inhibition of mTOR (IC50\\xa0=\\xa0>20,000\\xa0nM). In pre-clinical studies, WX-037 demonstrated strong inhibition of AKT phosphorylation, and promising growth inhibition in a range of cell lines in vitro, and tumour growth delay or stasis in vivo, with greater sensitivity observed in PIK3CA mutant or PTEN null cells and tumours [5]. WX-037 was investigated in a Phase I dose escalation study to investigate its safety, pharmacokinetics, pharmacodynamics and clinical activity in patients with solid tumours; however, this trial was also terminated due to business reasons [9, 17].\\n\\n\\nPrevious studies have shown that concomitant inhibition of the PI3K and MAPK pathways by PI3K and MEK inhibition yields promising anti-cancer effects in vitro and in vivo [2, 18\\u201326], and the combination of WX-554 and WX-037 demonstrated synergy in vitro and increased efficacy in vivo [5]. Consequently, the Phase I trial of WX-037 was designed not just to investigate the efficacy of single agent WX-037, but also the combination of WX-554 and WX-037 [9].\\n\\n\\nThe aim of this study was to determine the in vitro and in vivo activity of the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037, alone and in combination, in colorectal carcinoma cell lines and tumour xenograft-bearing mice. These colorectal carcinoma cell lines (HT29 and HCT116) were used as they both contain KRAS/RAF and PI3K pathway mutations. The first objective was to determine the in vitro potency and efficacy of the compounds, alone and in combination, by measuring growth inhibition using an SRB assay, cytotoxicity via a clonogenic assay and cell signalling by Western blotting, and to investigate if the effects of the combinations on cell growth were synergistic, additive or antagonistic using median effect analyses. These results were then used to design in vivo experiments to investigate the pharmacokinetic profile of the compounds and their efficacy, alone and in combination.\\n\\n\\nMethods\\nEthics statement\\nAll in vivo experiments were reviewed and approved by the Newcastle University (UK) animal welfare committee and were performed according to the guidelines for the welfare and use of animals in cancer research [27] and national law, under project license (PPL60/4442) issued by the UK Government Home Office under the animals (scientific procedure) act 1986.\\n\\n\\nInhibitors\\nThe MEK inhibitor WX-554 and the PI3K inhibitor WX-037 were kindly supplied by Wilex, Munich, Germany. For in vitro studies, the inhibitors were dissolved in anhydrous dimethyl sulphoxide (DMSO) and were stored frozen under light-protected conditions at \\u221220\\xa0\\xb0C. For in vivo studies, the MEK inhibitor WX-554 was dissolved in 0.9% NaCl (w/v), 10\\xa0mM Na-citrate pH 3.0 (w/v) and 0.2% Tween 20 (v/v) in sterile distilled water and the PI3K inhibitor WX-037 was suspended in SMEDDS (self-microemulsifying drug delivery system; 25% Capmul MCM EP (glycerol monocaprylocaprate) (v/v), 37.5% Tween 80 (polyoxyethylene(20) sorbitan monooleate) (v/v) and 37.5% PEG 400 (polyethylene glycol 400) (v/v)).\\n\\n\\nCell lines and reagents\\nHCT116 and HT29 human colorectal cancer cells were obtained from the ATCC (American Type Culture Collection). All cell lines were grown in RPMI-1640 medium (supplemented with 10% (v/v) foetal bovine serum, 1% (v/v) penicillin (50\\xa0U/ml)\\u2014streptomycin (50\\xa0mg/ml) and 2\\xa0mM\\xa0L-glutamine) and were confirmed free of mycoplasma contamination by regular testing with Mycoalert (Cambrex, Iowa, USA).\\n\\n\\nAnimals\\nAnimal studies were all carried out using female athymic CD1 nude mice (Charles River, Kent, UK), implanted with HCT116 or HT29 xenografts (1\\xa0\\xd7\\xa0107 cells in 50\\xa0\\xb5l media injected subcutaneously into the right flank), maintained and handled in isolators under specific pathogen-free conditions.\\n\\n\\nGrowth inhibition assay\\nExponentially growing cells were grown in media in 96-well format and were exposed to increasing concentrations of the single agent inhibitors WX-554 or WX-037, or WX-554 combined with WX-037 at 0.25, 0.5, 1, 2 or 4 times their half maximal growth inhibitory concentration (GI50) in DMSO, or 0.5% DMSO alone, for 72\\xa0h. Growth was then measured using the Sulforhodamine B (SRB) method and analysed as described previously [23]. The GI50 concentration was calculated based on a standard point to point curve with 1000 segments using GraphPad Prism software (California, USA). The data were analysed by median effect analysis using CalcuSyn software (Biosoft, Cambridgeshire, UK), which calculates the combination index of multiple drugs by an algebraic estimation algorithm.\\n\\n\\nCytotoxicity assay\\nExponentially growing cells were exposed to increasing concentrations of the single agent inhibitors WX-554 or WX-037, or 10\\xa0\\xb5M of WX-554 combined with 10\\xa0\\xb5M WX-037 in DMSO or 0.5% (v/v) DMSO alone for 72\\xa0h before harvesting and reseeding for colony formation. After growth for 10\\u201314\\xa0days, colonies were fixed in methanol\\u2013acetic acid 3:1 (v/v) and stained with crystal violet (0.4%\\xa0w/v). Colonies consisting of more than 50 cells were counted on an automated colony counter (Oxford Optronix, Oxford, UK). Two-tailed paired t tests were used to compare the different groups. Differences with a p\\xa0<\\xa00.05 were considered statistically significant.\\n\\n\\nWestern blotting\\nCells were treated with WX-554 and WX-037 at 1 or 10 times the half maximal growth inhibitory concentration (GI50) in DMSO, or 0.5% (v/v) DMSO alone, for 24\\xa0h. Western blots were prepared, probed with phospho-4EBP1 (Thr37/46) (#2855), phospho-p44/42 MAPK (Thr202/Tyr204) (#4370), phospho-AKT (Ser473) (#4060) or phospho-S6 ribosomal protein (Ser235/236) (#4858) monoclonal antibodies obtained from Cell Signalling Technology (New England BioLabs (UK) Ltd, Hertfordshire, UK) and developed as described previously [23]. Blots were then stripped (100\\xa0mM 2-mercaptoethanol, 2% (w/v) SDS and 62.5\\xa0mM Tris pH6.8 at 55\\xa0\\xb0C for 30\\xa0min) and re-probed with the respective total monoclonal antibody (4EBP1 (53H11) (#9644), p44/42 MAPK (ERK1/2) (#4695), AKT (pan) (C67E7) (#4691) or S6 ribosomal protein (5g10) (#2217)) obtained from Cell Signalling Technology (New England BioLabs (UK) Ltd, Hertfordshire, UK) and developed as described above.\\n\\n\\nPharmacokinetic (PK) studies\\nMice bearing HCT116 or HT29 human tumour xenografts were treated with 1 or 5\\xa0mg/kg WX-554, or 20 or 100\\xa0mg/kg WX-037, alone or in combination, and were bled by cardiac puncture under terminal anaesthesia at 6 or 24\\xa0h post-treatment (3 mice/time point). Blood was collected into heparinized tubes, and plasma was separated and stored at \\u221220\\xa0\\xb0C until analysed. Tumours were removed, snap frozen in liquid nitrogen and stored at \\u221280\\xa0\\xb0C prior to PK analyses. Samples were extracted with solid phase extraction (SPE) and analysed with high-performance liquid chromatography (HPLC) coupled with tandem mass spectrometry (MS/MS) by Wilex (Munich, Germany). The quantification limit for WX-554 was 1\\xa0ng/mL and within and between day variation was <15%. WinNonlin Software Version 4.0.1 (Pharsight Corporation, Peypin, France) was used for PK/PD modelling and non-compartmental analysis. Paired t tests were used to compare the different treatment groups, and differences with a p value\\xa0\\u22640.05 were considered statistically significant.\\n\\n\\nDetermination of anti-tumour activity\\nMice bearing HCT116 human tumour xenografts were randomized into treatment groups and then treated by oral gavage with either the vehicle (10\\xa0ml/kg), 2\\xa0mg/kg WX-554, 50\\xa0mg/kg WX-037 or the combination of 2\\xa0mg/kg WX-554 and 50\\xa0mg/kg WX-037 once daily for 14\\xa0days. Tumour volume was monitored by calliper measurement using the equation a\\n2\\xa0\\xd7\\xa0b/2, where a is the smallest measurement and b the largest. Data are presented as median relative tumour volumes (RTV), where the tumour volume in each mouse on the initial day of treatment (day 0) is assigned an RTV value of 1. The time to RTV4 for each individual tumour was calculated based on a standard point to point curve with 1000 segments using GraphPad Prism software (CA, USA). Mann\\u2013Whitney U tests were used to compare the different groups, i.e., the control versus each treatment group, the single agents versus each other and each single agent versus their combination. Differences with a p value\\xa0\\u22640.05 were considered statistically significant.\\n\\n\\nResults\\nThe PI3K inhibitor WX-037 and the MEK inhibitor WX-554 are synergistic and exhibit increased cytotoxicity in combination in vitro\\nThe growth inhibitory activity of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, as single agents, in HCT116 and HT29 cells was measured using the SRB assay (Supplementary Figure\\xa01). Both drugs induced over 65% growth inhibition in both the colorectal cell lines. The results were used to determine the half maximal growth inhibitory (GI50) concentration of the drugs after 72-h exposure. The MEK inhibitor WX-554 was found to have GI50 values of 38 and 4.3\\xa0nM, whereas the PI3K inhibitor WX-037 was less potent with GI50 values of 2934 and 112\\xa0nM in the HCT116 and HT29 cell lines, respectively (Supplementary Figure\\xa01).\\n\\n\\nStudies were then performed to determine the effect of combining the PI3K and MEK inhibitors on colorectal carcinoma cell growth over 72\\xa0h. WX-037 and WX-554 were used alone at 0.25x, 0.5x, 1x, 2x and 4x their respective GI50 concentration, as calculated from Supplementary Figure\\xa01, and at equipotent concentrations at the same GI50 ratios in combination. Figure\\xa01 shows that the combination of WX-037 and WX-554 was markedly more growth inhibitory than either compound alone, completely inhibiting growth at the highest concentrations. Data were then evaluated by median effect analysis (CalcuSyn, Biosoft, Great Shelford, UK) to determine whether the greater growth inhibitory activity of the combination of WX-554 and WX-037 reflected an additive or a synergistic effect. The combination of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 was strongly synergistic when combined at the GI50 concentration compared to the compounds alone (Supplementary Table\\xa01).Fig.\\xa01\\nGrowth inhibition induced by the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, alone and in combination, in the HCT116 and HT29 cell lines. HCT116 (a) and HT29 (b) cells were treated with the indicated fractions of the GI50 concentrations of the inhibitors, alone or in combination, derived from Supplementary Figure\\xa01, for 72\\xa0h, and an SRB assay was subsequently performed. Growth is presented as a percentage of the control, in which cells were treated with 0.5% (v/v) DMSO. Points represent the mean of 3 independent experiments \\xb1 standard error. Lines were fitted using nonlinear regression analysis\\n\\n\\n\\n\\n\\nCell survival after 72-h exposure to the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 was also measured using a clonogenic cytotoxicity assay. Single agent WX-554 showed significant cytotoxicity at 10\\xa0\\xb5M with 67% cell kill in the HCT116 cell line and 75% in the HT29 cell line; however, the mean lethal concentration (LC50) of 0.6\\xa0\\xb5M and 1.6\\xa0\\xb5M WX-554 was approximately 16-fold and 372-fold higher than the corresponding GI50- values in the HCT116 and HT29 cell lines, respectively. WX-037 showed no marked cytotoxicity with less than 50% cell death after 72\\xa0h treatment at 10\\xa0\\xb5M (Supplementary Figure\\xa02).\\n\\n\\nThe cytotoxicity of the PI3K and MEK inhibitors in combination after 72\\xa0h treatment was then determined. However, as WX-037 did not produce\\xa0>\\xa050% cytotoxicity at 10\\xa0\\xb5M, it was not possible to determine an LC50 value, and hence the highest concentration previously used of 10\\xa0\\xb5M WX-037 was combined with 10\\xa0\\xb5M WX-554. There was a statistically significant increase in cytotoxicity when the PI3K and MEK inhibitors were combined, compared to the cytotoxicity induced by the drugs as single agents, in the HCT116 (p\\xa0=\\xa00.02) and HT29 (p\\xa0<\\xa00.01) cell lines (Fig.\\xa02). Overall, the interaction of WX-037 and WX-554 resulted in significantly enhanced cell growth inhibition and an increase in cytotoxicity in both cell lines studied.Fig.\\xa02\\nCell survival after 72-h exposure to 10\\xa0\\xb5M of the PI3K inhibitor WX-037 and 10\\xa0\\xb5M of the MEK inhibitor WX-554, alone and in combination, in the HCT116 cell line and HT29 cell lines. HCT116 (a) and HT29 (b) cells were treated with a fixed concentration of each inhibitor alone or in combination for 72\\xa0h, and cell survival was subsequently determined by clonogenic assay after 10\\u201314\\xa0days of colony growth. Survival is presented as a percentage of the control, in which cells were treated with 0.5% (v/v) DMSO. Bars represent the mean of 3 independent replicates \\xb1 standard error. *Significantly different from either agent alone, p\\xa0\\u2264\\xa00.05\\n\\n\\n\\n\\n\\nThe effect of 24-h exposure to the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, both as single agents and in combination, was also investigated by Western blotting to determine the effect on the PI3K/AKT signalling pathway, using total and phospho-specific antibodies for AKT, S6 and 4EBP1 and the effect on MAPK signalling, using total and phospho-specific antibodies for ERK1/2. The compounds were used as single agents or in combination at their respective GI50 concentrations and at 10x the GI50 concentration.\\n\\n\\nSupplementary Figure\\xa03 shows that treatment with the MEK inhibitor WX-554 reduced ERK1/2 phosphorylation at 1 and 10 times the GI50 concentration in the HCT116 cell line, and at 10 times the GI50 concentration in the HT29 cell line. The reduction in ERK1/2 phosphorylation was enhanced with the combination of WX-554 and WX-037 leading to complete inhibition with 10 times the GI50 concentration in both colorectal carcinoma cell lines (Supplementary Figure\\xa03). Additionally, there was a concentration-dependent reduction in AKT phosphorylation in the HCT116 cell line after treatment with the single agent PI3K inhibitor WX-037 which was enhanced, to yield complete inhibition at the GI50 concentration, after combined WX-037 and WX-554 treatment. In the HT29 cell line, there was a reduction in AKT phosphorylation at 10 times the GI50 concentration; however, this reduction was similar with single agent WX-037 and the combination (Supplementary Figure\\xa03).\\n\\n\\nSingle agent WX-037 also caused a concentration-dependent reduction in S6 phosphorylation at 1 and 10 times the GI50 concentration, and treatment with the combination of WX-037 and WX-554 enhanced this inhibition, causing complete inhibition at 10 times the GI50 concentration in both colorectal carcinoma cell lines (Supplementary Figure\\xa03). However, WX-037 had no marked effect on the phosphorylation of 4EBP1 alone or in combination with WX-554 in either colorectal carcinoma cell line (Supplementary Figure\\xa03). Hence, overall, the combination of the MEK inhibitor WX-554 and the PI3K inhibitor WX-037 resulted in enhanced inhibition of ERK1/2 and S6 phosphorylation, and inhibition of AKT phosphorylation, in both colorectal carcinoma cell lines. However, both the single agents and the combination had no major impact on 4EBP1 phosphorylation.\\n\\n\\nThe PI3K inhibitor WX-037 and the MEK inhibitor WX-554 exhibit increased tumour growth delay in combination in vivo\\nA PK study was carried out with samples taken 6 and 24\\xa0h after treatment with a single dose of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, alone and in combination, in HCT116 and HT29 human tumour xenograft-bearing mice. The concentrations of the drugs in the plasma and the tumour tissue were measured using LC\\u2013MS/MS (Figs.\\xa03, 4, Supplementary Figures\\xa04 and 5 and Supplementary Table\\xa02).Fig.\\xa03\\nConcentrations of the MEK inhibitor WX-554 alone and in combination with the PI3K inhibitor WX-037 in tumours from mice bearing HCT116 or HT29 human tumour xenografts. Tumour concentrations of WX-554 measured by LC\\u2013MS/MS from HCT116 (a, c) and HT29 (b, d) tumour xenograft-bearing mice at the indicated time points after a single p.o. dose of 1\\xa0mg/kg (a, b) or 5\\xa0mg/kg (c, d) WX-554 alone or combined with 20 or 100\\xa0mg/kg WX-037. Data are presented as the mean concentration from 3 mice in each group \\xb1 standard error. Horizontal dashed lines indicate the in vitro GI50 concentration for the respective cell line, calculated from Supplementary Figure\\xa01\\n\\n\\nFig.\\xa04\\nConcentrations of the PI3K inhibitor WX-037 alone and in combination with the MEK inhibitor WX-554 in tumours from mice bearing HCT116 or HT29 human tumour xenografts. Tumour concentrations of WX-037 measured by LC\\u2013MS/MS from HCT116 (a, c) and HT29 (b, d) tumour xenograft-bearing mice at the indicated time points after a single p.o. dose of 20\\xa0mg/kg (a, b) or 100\\xa0mg/kg (c, d) WX-037 alone or combined with 1 or 5\\xa0mg/kg WX-554. Data are presented as the mean concentration from 3 mice in each group \\xb1 standard error. Horizontal dashed lines indicate the in vitro GI50 concentration for the respective cell line, calculated from Supplementary Figure\\xa01\\n\\n\\n\\n\\n\\nAfter a single dose of 1 or 5\\xa0mg/kg WX-554, alone or in combination with 20 or 100\\xa0mg/kg WX-037, WX-554 concentrations in the plasma and tumour tissue generally greatly exceeded the in vitro GI50 value of 18\\xa0ng/ml (38\\xa0nM) in the HCT116 cell line and 2\\xa0ng/ml (4\\xa0nM) in the HT29 cell line (determined in Supplementary Figure\\xa01) (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A). The exception to this was that plasma WX-554 concentrations were only approximately equal to the HCT116 GI50 value at 6\\xa0h and were below the HCT116 GI50 value by 24\\xa0h, in the HCT116 tumour xenograft-bearing mice after treatment with 1\\xa0mg/kg WX-554, alone or in combination with 20 or 100\\xa0mg/kg WX-037 (Supplementary Figure\\xa04A and Supplementary Table\\xa02A). Additionally, the plasma WX-554 concentration was also below the HCT116 GI50 value by 24\\xa0h after 5\\xa0mg/kg WX-554 alone; however, WX-554 concentrations still exceeded the GI50 value when 5\\xa0mg/kg WX-554 was combined with WX-037 (Supplementary Figure\\xa04C and Supplementary Table\\xa02A). However, overall, the absolute plasma and tumour levels of WX-554 were generally similar in the HT29 and HCT116 tumour xenograft-bearing mice (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A).\\n\\n\\nThere was no consistent effect of concomitant dosing with 20 or 100\\xa0mg/kg WX-037 on the levels of WX-554 in the tumour or the plasma after administration of 1\\xa0mg/kg WX-554 in either HCT116 or HT29 tumour xenograft-bearing mice. In contrast, after dosing with 5\\xa0mg/kg WX-554, there was generally a WX-037 dose-dependent decrease in the levels of WX-554 at 6\\xa0h in the tumour and the plasma upon concomitant treatment with WX-037, which was significant for the tumour data at 6\\xa0h (p\\xa0<\\xa00.05) (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A). These results suggest that WX-037 may delay the tumour uptake of WX-554 at the higher dose (5\\xa0mg/kg) compared to when WX-554 is administered alone.\\n\\n\\nIn contrast to the data for WX-554, after treatment with 20\\xa0mg/kg WX-037, as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554, WX-037 concentrations in plasma and tumour tissue at 6 and 24\\xa0h were markedly lower than the in vitro GI50 value of 1414\\xa0ng/ml (2934\\xa0nM) in the HCT116 cell line (determined in Supplementary Figure\\xa01) (Fig.\\xa04a, Supplementary Figure\\xa05A and Supplementary Table\\xa02B). Plasma and tumour WX-037 concentrations were also markedly below the HCT116 GI50 value 24\\xa0h after treatment with 100\\xa0mg/kg WX-037, given as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554; however, concentrations in the plasma and tumour tissue were similar to or exceeded the HCT116 GI50 value 6\\xa0h after a 100\\xa0mg/kg dose of WX-037 (Fig.\\xa04c, Supplementary Figure\\xa05C and Supplementary Table\\xa02B).\\n\\n\\nAfter treatment with 20 or 100\\xa0mg/kg WX-037, as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554, concentrations in plasma and tumour tissue greatly exceeded the in vitro GI50 value of 54\\xa0ng/ml (112\\xa0nM) in the HT29 cell line (determined in Supplementary Figure\\xa01) at 6\\xa0h. At 24\\xa0h, although concentrations were similar to or exceeded the HT29 GI50 value in all tumour samples and in plasma after a 100\\xa0mg/kg dose of WX-037, plasma concentrations had generally declined to below the GI50 value after a 20\\xa0mg/kg dose of WX-037 (Fig.\\xa04b\\u2013d, Supplementary Figure\\xa05B and D and Supplementary Table\\xa02B). As with the WX-554 data, the absolute plasma and tumour levels of WX-037 were similar in the HCT116 and HT29 tumour xenograft-bearing mice (Fig.\\xa04, Supplementary Figure\\xa05 and Supplementary Table\\xa02B).\\n\\n\\nThere did not appear to be a consistent effect of concomitant dosing with 1 or 5\\xa0mg/kg WX-554 on the levels of WX-037 in the tumour or the plasma at 6\\xa0h after dosing with 20\\xa0mg/kg in either HCT116 or HT29 tumour xenograft-bearing mice (Fig.\\xa04a, b, Supplementary Figure\\xa05A and B and Supplementary Table\\xa02B). However, after dosing with 100\\xa0mg/kg, there appeared to be a dose-dependent decrease in the levels of WX-037 in the tumour and the plasma upon concomitant dosing with WX-554 at 6\\xa0h, but this effect was only significant in HCT116 tumours (Fig.\\xa04c, d, Supplementary Figure\\xa05C and D and Supplementary Table\\xa02B). Hence, concentrations of WX-037 were generally similar regardless of whether it was administered as a single agent or in combination with WX-554, and levels were consistently higher at 6\\xa0h compared with 24\\xa0h.\\n\\n\\nAs the combination of WX-554 and WX-037 was synergistic in the in vitro studies, it may not be necessary for the drug concentrations in the plasma and the tumour to exceed those of the in vitro GI50 values for the single agents to achieve efficacy with the combination in vivo. Based on the in vitro results, in order to achieve half maximal growth inhibition, less than 1/6th and approximately 1/3rd of the single agent GI50 was required for 50% growth inhibition with the drug combination in the HCT116 and HT29 cell lines, respectively, which equates to GI50 values of 3 and <1\\xa0ng/ml WX-554 and 219 and 20\\xa0ng/ml WX-037 in the HCT116 and HT29 cell lines (calculated from Fig.\\xa01). Therefore, with all the combinations of WX-554 and WX-037, WX-554 levels in the plasma and tumour tissues are at or exceed the GI50 concentration for the combination at both 6 and 24\\xa0h (Fig.\\xa03, Supplementary Figures\\xa04 and Supplementary Table\\xa02A). Furthermore, with all combinations of WX-554 and 100\\xa0mg/kg WX-037, WX-037 levels in the plasma and tumour tissues are at or exceed the GI50 for the combination at both 6 and 24\\xa0h, and with all combinations of WX-554 with 20\\xa0mg/kg WX-037, WX-037 levels in the plasma and tumour tissues are at or exceed the GI50 for the combination at 6\\xa0h, but remain below at 24\\xa0h (Fig.\\xa04, Supplementary Figure\\xa05 and Supplementary Table\\xa02B).\\n\\n\\nBased on the results of the PK study, the efficacy of 50\\xa0mg/kg of the PI3K inhibitor WX-037 and 2\\xa0mg/kg of the MEK inhibitor WX-554 given orally, as single agents and in combination, was assessed in HCT116 human tumour xenograft-bearing mice (Fig.\\xa05). The individual doses of the PI3K and MEK inhibitors were chosen to be approximately equiactive, in order to mirror the in vitro conditions under which synergy had been demonstrated (Supplementary Figure\\xa01). In this study, mice were treated daily for 14\\xa0days and tumour volumes were measured three times a week. Figure\\xa05a demonstrates that treatment with 50\\xa0mg/kg WX-037 and 2\\xa0mg/kg WX-554, alone and in combination, caused tumour growth delay compared to vehicle-treated control tumours, and that growth delay was greater with the combination. Additionally, body weight was monitored daily to assess the tolerability of the therapy, and both single agent and combination treatments were found to be well tolerated as average body weights did not drop below 89% of the starting weight (Fig.\\xa05b). The time for the tumours to quadruple in size (time to RTV4) was calculated (Fig.\\xa05c), and statistical analyses using a Mann\\u2013Whitney test demonstrated a significant difference between vehicle-treated control tumours and the combination group (p\\xa0<\\xa00.01), and between the single agent MEK inhibitor and the combination group (p\\xa0=\\xa00.02).Fig.\\xa05\\nEfficacy and tolerability of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 in mice bearing human HCT116 colorectal tumour xenografts. HCT116 tumour xenografts were treated with either vehicle control, 2\\xa0mg/kg of the MEK inhibitor WX-554 and 50\\xa0mg/kg of the PI3K inhibitor WX-037 alone, or 2\\xa0mg/kg of the MEK inhibitor WX-554 and 50\\xa0mg/kg of the PI3K inhibitor WX-037 in combination, p.o. once daily for 14\\xa0days. a Tumour growth curves: data are presented as the median relative tumour volume (RTV), where the growth is calculated for each tumour relative to its size on day 0. Points represent the median of the 10 mice in each group. The dashed line shows the point at which tumours reached four times the initial volume (RTV4). b Effects on body weight: data are presented as a percentage of starting body weight. Points represent the mean of the mice in each group \\xb1 standard error. c Time taken for xenografts to reach four times the initial volume (time to RTV4): data are presented as the time taken by each individual tumour in each group to quadruple in size, and lines to represent the mean of the mice in each group \\xb1 standard error. p values are given where the combination is significantly different from either agent alone (p\\xa0\\u2264\\xa00.05)\\n\\n\\n\\n\\n\\nDiscussion\\nThe novel PI3K inhibitor WX-037 and the novel MEK inhibitor WX-554 have demonstrated in vitro activity in a range of cancer cell lines including breast, fibrosarcoma, thyroid, melanoma, colorectal, ovarian and pancreas lines with a broad range of GI50 values, where generally the cell lines most sensitive to WX-037 had PIK3CA mutations or PTEN loss, and to WX-554 had BRAF or RAS mutations [5]. Furthermore, the potency of WX-554 determined in the HCT116 and HT29 cell lines in this study (38 and 4.3\\xa0nM, respectively) was similar to that determined in the unpublished Wilex studies (29 and 7.2\\xa0nM) [5], and to the MEK inhibitor, PD 0325901 (21 and 6.5\\xa0nM) using the same assay [23]. Similarly the potency of WX-037 determined here in the HCT116 and HT29 cell lines (2934 and 112\\xa0nM, respectively) was similar to that reported in the unpublished Wilex studies (136\\xa0nM in the HT29 cell line) [5], and to the pan class I PI3K inhibitor, pictilisib (1081 and 157\\xa0nM) using the same assay [23].\\n\\n\\nIn these in vitro studies, the HT29 colorectal carcinoma cell line was consistently more sensitive to WX-037 and WX-554 than the HCT116 cell line (26 and 9-fold difference, respectively), which is consistent with other studies where the HT29 cell line exhibited increased sensitivity to pan class I PI3K and/or MEK inhibition, compared to the HCT116 cell line [23, 28, 29]. The observed difference could be due to the presence of a BRAF, rather than a KRAS, mutation as RAS mutations have been associated with intrinsic resistance in a previous study [30]. Alternatively, differences in cell signalling may be responsible, as sensitivity to MEK inhibition was found to correlate with strong ERK1/2 signalling and weak PI3K signalling [28]. Furthermore, the plethora of other mutations found in the HT29 and HCT116 cell lines may contribute to the difference in sensitivity [29, 31].\\n\\n\\nThe combination of WX-037 and WX-554 was strongly synergistic, which is consistent with previous unpublished studies by UCB Celltech and Wilex where synergy was observed in the SK-MEL-28 melanoma, HT29 colorectal, Mia PaCa-2 pancreatic and SKOV-3 ovarian carcinoma cell lines [5]. The degree of synergy with the WX-037 and WX-554 combination was consistent with in vitro published studies using combinations of other PI3K and MEK inhibitors [2, 18, 23, 24, 32, 33].\\n\\n\\nSingle agent WX-554 not only exhibited growth inhibition but also significant cytotoxicity, albeit at higher concentrations with LC50 values approximately 16-fold and 372-fold higher than the corresponding GI50\\u2212 values in the HCT116 and HT29 cell lines, respectively. This marked cytotoxicity at 10\\xa0\\xb5M is in contrast to the effect of PD 0325901 and selumetinib [23] and also to the general perspective that MEK inhibitors are cytostatic [34]. However, another study has already reported that selumetinib can cause apoptosis in some human tumour xenografts, but not in others, which was suggested to be due to differences in ERK1/2 substrate expression or differential cell signalling networks [35], and hence the result with WX-554 is not entirely unexpected.\\n\\n\\nIn contrast to the effect of the MEK inhibitor, the PI3K inhibitor WX-037 at 10\\xa0\\xb5M showed only limited cytotoxicity, which is consistent with the results observed with another pan class I PI3K inhibitor pictilisib [23]. However, the marked synergy observed in the growth inhibition studies with WX-037 and WX-554 translated into cytotoxicity, albeit at a high concentration (10\\xa0\\xb5M), as there was a statistically significant increase in cytotoxicity with the combination of the Wilex PI3K and MEK inhibitors. This cytotoxicity is consistent with previous unpublished studies by UCB Celltech and Wilex where the observed growth inhibitory synergy with WX-037 and WX-554 in the SK-MEL-28 melanoma, HT29 colorectal, Mia PaCa-2 pancreatic and SKOV-3 ovarian carcinoma cell lines was found to correlate with the induction of apoptosis [5]. Similarly, published in vitro studies have shown that combined inhibition of PI3K and MEK resulted in cell death [33, 36, 37].\\n\\n\\nCell signalling studies showed that the single agent PI3K inhibitor WX-037 caused a concentration-dependent reduction in AKT and S6 phosphorylation, which was generally enhanced by the MEK inhibitor WX-554, but that 4EBP1 phosphorylation was not inhibited, consistent with the results of published studies using pan class I PI3K, as opposed to mixed PI3K/mTOR, inhibitors [23, 32, 37, 38]. Furthermore, single agent MEK inhibitor WX-554 caused a concentration-dependent reduction in ERK phosphorylation, and the combination of WX-554 and WX-037 resulted in enhanced inhibition of ERK1/2 phosphorylation, as observed with other MEK inhibitors [23, 39, 40] and combinations [23, 33].\\n\\n\\nIn the pre-clinical studies presented here, and in both pre-clinical and clinical studies reported elsewhere [5, 6], WX-554 has been shown to be non-toxic. Furthermore, single agent WX-554 induced tumour growth delay at 2\\xa0mg/kg in HCT116 colorectal tumour xenograft models, consistent with previous unpublished studies using WX-554 and published studies using other MEK inhibitors that reported tumour growth delay or stasis in vivo, with increased sensitivity in BRAF or RAS mutant cells and tumours [3\\u20135, 25]. In vivo pharmacokinetic analyses revealed concentrations of WX-554 in plasma and tumour tissue increased in a dose-dependent manner and that tumour WX-554 levels generally exceeded the in vitro GI50 concentration, whereas plasma levels were more variable.\\n\\n\\nSingle agent WX-037 was also found to be non-toxic in these studies and those reported elsewhere [5, 17], which is consistent with data for other pan class I PI3K inhibitors [41\\u201345]. Furthermore, 50\\xa0mg/kg WX-037 generated a tumour growth delay in HCT116 colorectal tumour xenograft models, which is consistent with previous unpublished studies using WX-037 and published studies using other pan class I PI3K inhibitors that reported tumour growth delay, or in some cases tumour stasis, with greater sensitivity in PIK3CA mutant or PTEN null cells and tumours [5, 14, 15, 46\\u201351]. Pharmacokinetic studies indicated that concentrations of WX-037 in plasma and tumour tissue increased dose dependently and that WX-037 tumour and plasma levels generally exceeded the in vitro GI50 concentration at an early time point (6\\xa0h), depending on the cell line and dose, whereas levels generally had decreased below the GI50 value by the later time point (24\\xa0h). These pharmacokinetic data are similar to those observed in HCT116 tumour xenograft-bearing mice treated with pictilisib [25].\\n\\n\\nThe PK analyses indicated that there was no pharmacokinetic interaction between WX-037 and WX-554 at lower doses. However, concomitant dosing of WX-037 with the high dose (5\\xa0mg/kg) of WX-554 caused a dose-dependent decrease in early time point (6\\xa0h) WX-554 tumour and plasma concentrations, and a dose-dependent increase in late time point (24\\xa0h) WX-554 levels, suggesting that the presence of WX-037 may delay the uptake of WX-554, leading to higher WX-554 levels at 24\\xa0h. Furthermore, concomitant WX-554 dosing with high doses (100\\xa0mg/kg) of WX-037 caused a dose-dependent decrease in early time point (6\\xa0h) WX-037 tumour and plasma concentrations; however, this effect was only significant in the HCT116 tumour tissue due to variation in the data. The pharmacokinetic interaction between higher doses of WX-037 and WX-554 is in contrast to the results obtained with the combination of PD 0325901 and pictilisib in HCT116 tumour xenograft-bearing mice [25], where there was no significant pharmacokinetic interaction, and to pre-clinical and clinical reports that there was no pharmacokinetic interaction between the PI3K and MEK inhibitors, pictilisib and cobimetinib [52, 53]. However, a recent Phase Ib study using a combination of the PI3K inhibitor buparlisib and the MEK inhibitor trametinib reported a potential pharmacokinetic interaction, which was suggested to be due to the variability of the accumulation of the MEK inhibitor as no clear mechanism could be identified [54].\\n\\n\\nIn HCT116 colorectal tumour xenografts, the combination of 50\\xa0mg/kg WX-037 and 2\\xa0mg/kg WX-554 was non-toxic and caused tumour stasis and enhanced tumour growth delay compared to either single agent at the same dose. This improved activity is consistent with previous studies using combinations of PI3K and MEK inhibitors in a range of pre-clinical human tumour xenograft and mouse models [19, 21, 22, 25, 55\\u201358]. In a previous study using the MEK inhibitor, PD 0325901, and the PI3K inhibitor, GDC-0941, the cell line-dependent differences in sensitivity to the inhibitors determined in the in vitro studies did not correlate with the results observed in the in vivo studies [23, 25], suggesting that HT29 xenografts would not necessarily of been more sensitive to the inhibitors in vivo. Nevertheless, previous studies by UCB and Wilex have demonstrated that WX-554 treatment is able to induce tumour growth delay in HT29 xenografts [5].\\n\\n\\nThe improved efficacy using combinations of PI3K and MEK inhibitors in pre-clinical studies is being translated into clinical trials [45, 54, 59], and there are currently at least 7 ongoing or completed Phase 1b clinical studies [9]. However, there have been issues with the tolerability of the combination treatment in two phase 1b trials combining the MEK inhibitor trametinib, with either the pan-PI3K inhibitor buparlisib, or the PI3K/mTOR inhibitor GSK2126458, where the former study showed promising anti-tumour activity but frequent dose interruptions and reductions were necessary due to toxicity [54]. The latter study was terminated due to poor tolerability and efficacy [9, 60]. These results suggest that the choice of the PI3K and MEK inhibitor to be combined will be crucial to the success of the treatment, due to subtle differences in the pharmacology of the inhibitors leading to differing toxicity profiles. Overall, these studies characterize, for the first time, the in vitro and in vivo efficacy of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, as single agents and in combination. Furthermore, these studies illustrate that dual targeting of PI3K and MEK can induce synergy in vitro which translates to marked tumour growth delay in vivo.\\n\\n\\nElectronic supplementary material\\nBelow is the link to the electronic supplementary material. \\n\\nSupplementary material 1 (PDF 520\\xa0kb)\\n\\n\\n\\nSupplementary material 2 (PDF 81\\xa0kb)', u'publisher': u'Springer Berlin Heidelberg', u'documentText': u'Cancer Chemother PharmacolCancer Chemother. PharmacolCancer Chemotherapy and Pharmacology0344-57041432-0843Springer Berlin HeidelbergBerlin/Heidelberg278372575114336318610.1007/s00280-016-3186-4Original ArticleEnhanced anti-tumour activity of the combination of the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037HaagensenEmma J.1ThomasHuw D.1SchmalixWolfgang A.2PayneAndrew C.3KevorkianLara3AllenRodger A.3BevanPaul2MaxwellRoss J.1NewellDavid R.(+44) 191 246 4300herbie.newell@ncl.ac.uk11Newcastle Cancer Centre, Northern Institute for Cancer Research, Paul O\\u2019Gorman Building, Medical School, Newcastle University, Framlington Place, Newcastle-upon-Tyne, NE2 4HH UK 2Wilex AG, Grillparzerstrasse 18, 81675 Munich, Germany 3UCB Pharma Ltd, 208 Bath Road, Slough, SL1 3WE UK 111120161111201620167861269128126201631102016\\xa9 The Author(s) 2016\\nOpen AccessThis article is distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made.\\nPurpose\\nTumours frequently have defects in multiple oncogenic pathways, e.g. MAPK and PI3K signalling pathways, and combinations of targeted therapies may be required for optimal activity. This study evaluated the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037, as single agents and in combination, in colorectal carcinoma cell lines and tumour xenograft-bearing mice.\\n\\n\\nMethods\\nIn vitro growth inhibition, survival and signal transduction were measured using the Sulforhodamine B, clonogenic and Western blotting assays, respectively, in HCT116 and HT29 cell lines. In vivo anti-tumour efficacy and pharmacokinetic properties were assessed in HCT116 and HT29 human colorectal cancer xenograft tumour-bearing mice.\\n\\n\\nResults\\nThe combination of WX-554 and WX-037 exhibited marked synergistic growth inhibition in vitro, which was associated with increased cytotoxicity and enhanced inhibition of ERK and S6 phosphorylation, compared to either agent alone. Pharmacokinetic analyses indicated that there was no PK interaction between the two drugs at low doses, but that at higher doses, WX-037 may delay the tumour uptake of WX-554. In vivo efficacy studies revealed that the combination of WX-037 and WX-554 was non-toxic and exhibited marked tumour growth inhibition greater than observed with either agent alone.\\n\\n\\nConclusion\\nThese studies show for the first time that combination treatment with the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037 can induce synergistic growth inhibition in vitro, which translates into enhanced anti-tumour efficacy in vivo.\\n\\n\\nElectronic supplementary material\\nThe online version of this article (doi:10.1007/s00280-016-3186-4) contains supplementary material, which is available to authorized users.\\n\\n\\nKeywords\\nPI3K\\nMEK\\nCombination\\nSynergy; Colorectal cancer\\nhttp://dx.doi.org/10.13039/501100000265Medical Research CouncilUCB Celltech CASE PhD studentship\\nhttp://dx.doi.org/10.13039/501100000289Cancer Research UKDrug Discovery Programme C240/A7409issue-copyright-statement\\xa9 Springer-Verlag Berlin Heidelberg 2016\\nIntroduction\\nThe development of numerous targeted small molecule inhibitors represents an important and evolving new approach to cancer therapy. However, as tumours often have defects in multiple oncogenic signalling pathways, single agent anti-tumour activity is modest, and thus combinations of targeted agents are being investigated. Specifically, the MAPK pathway, a major proliferative pathway, and the PI3K pathway, a major survival pathway, are frequently activated in cancer and are being concomitantly targeted.\\n\\n\\nMany MEK inhibitors, such as PD 0325901 and selumetinib (AZD6244), have been developed to target the MAPK pathway and have shown potent growth inhibitory activity in experimental systems [1\\u20134]. A novel orally available small molecule allosteric MEK inhibitor WX-554 (UCB1366554), which potently inhibits MEK1 and MEK2 with a half maximal inhibitory concentration (IC50) of 4.7 and 11\\xa0nM, respectively, has been developed by Wilex and UCB Celltech. WX-554 has demonstrated marked inhibition of ERK1/2 phosphorylation in HT29 cells, and growth inhibition in a range of cell lines in vitro, and tumour growth delay or stasis in vivo, with increased sensitivity in BRAF or RAS mutant cells and tumours [5]. WX-554 has been shown to be safe and tolerable in a dose escalation study in healthy volunteers [6], and a recent Phase I study indicated that WX-554 has very good bioavailability and that it is able to inhibit MEK signal transduction in a dose-dependent manner [3, 7, 8]. WX-554 was investigated in a Phase Ib/II dose escalation study to determine safety and pharmacokinetics/pharmacodynamics in patients with solid tumours [7, 9], which was terminated due to business, rather than clinical, reasons.\\n\\n\\nTargeting the PI3K/AKT pathway has been investigated with a range of PI3K inhibitors which can be isoform-specific inhibitors, pan class I inhibitors (e.g. pictilisib, GDC-0941) or dual PI3K/mTOR inhibitors (e.g. dactolisib, NVP-BEZ235), and promising anti-cancer efficacy has been reported in pre-clinical models [10\\u201316]. Wilex and UCB Celltech have also developed a novel small molecule pan class I PI3K inhibitor, WX-037 (UCB1370037), from an indole series. WX-037 is a potent inhibitor of the \\u03b1 and \\u03b4 isoforms of PI3K (IC50\\xa0=\\xa04.1 and 2.4\\xa0nM, respectively) and a weaker inhibitor of the \\u03b2 and \\u03b3 isoforms of PI3K and DNA-PK (IC50\\xa0=\\xa069, 36 and 28\\xa0nM, respectively) with no detectable inhibition of mTOR (IC50\\xa0=\\xa0>20,000\\xa0nM). In pre-clinical studies, WX-037 demonstrated strong inhibition of AKT phosphorylation, and promising growth inhibition in a range of cell lines in vitro, and tumour growth delay or stasis in vivo, with greater sensitivity observed in PIK3CA mutant or PTEN null cells and tumours [5]. WX-037 was investigated in a Phase I dose escalation study to investigate its safety, pharmacokinetics, pharmacodynamics and clinical activity in patients with solid tumours; however, this trial was also terminated due to business reasons [9, 17].\\n\\n\\nPrevious studies have shown that concomitant inhibition of the PI3K and MAPK pathways by PI3K and MEK inhibition yields promising anti-cancer effects in vitro and in vivo [2, 18\\u201326], and the combination of WX-554 and WX-037 demonstrated synergy in vitro and increased efficacy in vivo [5]. Consequently, the Phase I trial of WX-037 was designed not just to investigate the efficacy of single agent WX-037, but also the combination of WX-554 and WX-037 [9].\\n\\n\\nThe aim of this study was to determine the in vitro and in vivo activity of the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037, alone and in combination, in colorectal carcinoma cell lines and tumour xenograft-bearing mice. These colorectal carcinoma cell lines (HT29 and HCT116) were used as they both contain KRAS/RAF and PI3K pathway mutations. The first objective was to determine the in vitro potency and efficacy of the compounds, alone and in combination, by measuring growth inhibition using an SRB assay, cytotoxicity via a clonogenic assay and cell signalling by Western blotting, and to investigate if the effects of the combinations on cell growth were synergistic, additive or antagonistic using median effect analyses. These results were then used to design in vivo experiments to investigate the pharmacokinetic profile of the compounds and their efficacy, alone and in combination.\\n\\n\\nMethods\\nEthics statement\\nAll in vivo experiments were reviewed and approved by the Newcastle University (UK) animal welfare committee and were performed according to the guidelines for the welfare and use of animals in cancer research [27] and national law, under project license (PPL60/4442) issued by the UK Government Home Office under the animals (scientific procedure) act 1986.\\n\\n\\nInhibitors\\nThe MEK inhibitor WX-554 and the PI3K inhibitor WX-037 were kindly supplied by Wilex, Munich, Germany. For in vitro studies, the inhibitors were dissolved in anhydrous dimethyl sulphoxide (DMSO) and were stored frozen under light-protected conditions at \\u221220\\xa0\\xb0C. For in vivo studies, the MEK inhibitor WX-554 was dissolved in 0.9% NaCl (w/v), 10\\xa0mM Na-citrate pH 3.0 (w/v) and 0.2% Tween 20 (v/v) in sterile distilled water and the PI3K inhibitor WX-037 was suspended in SMEDDS (self-microemulsifying drug delivery system; 25% Capmul MCM EP (glycerol monocaprylocaprate) (v/v), 37.5% Tween 80 (polyoxyethylene(20) sorbitan monooleate) (v/v) and 37.5% PEG 400 (polyethylene glycol 400) (v/v)).\\n\\n\\nCell lines and reagents\\nHCT116 and HT29 human colorectal cancer cells were obtained from the ATCC (American Type Culture Collection). All cell lines were grown in RPMI-1640 medium (supplemented with 10% (v/v) foetal bovine serum, 1% (v/v) penicillin (50\\xa0U/ml)\\u2014streptomycin (50\\xa0mg/ml) and 2\\xa0mM\\xa0L-glutamine) and were confirmed free of mycoplasma contamination by regular testing with Mycoalert (Cambrex, Iowa, USA).\\n\\n\\nAnimals\\nAnimal studies were all carried out using female athymic CD1 nude mice (Charles River, Kent, UK), implanted with HCT116 or HT29 xenografts (1\\xa0\\xd7\\xa0107 cells in 50\\xa0\\xb5l media injected subcutaneously into the right flank), maintained and handled in isolators under specific pathogen-free conditions.\\n\\n\\nGrowth inhibition assay\\nExponentially growing cells were grown in media in 96-well format and were exposed to increasing concentrations of the single agent inhibitors WX-554 or WX-037, or WX-554 combined with WX-037 at 0.25, 0.5, 1, 2 or 4 times their half maximal growth inhibitory concentration (GI50) in DMSO, or 0.5% DMSO alone, for 72\\xa0h. Growth was then measured using the Sulforhodamine B (SRB) method and analysed as described previously [23]. The GI50 concentration was calculated based on a standard point to point curve with 1000 segments using GraphPad Prism software (California, USA). The data were analysed by median effect analysis using CalcuSyn software (Biosoft, Cambridgeshire, UK), which calculates the combination index of multiple drugs by an algebraic estimation algorithm.\\n\\n\\nCytotoxicity assay\\nExponentially growing cells were exposed to increasing concentrations of the single agent inhibitors WX-554 or WX-037, or 10\\xa0\\xb5M of WX-554 combined with 10\\xa0\\xb5M WX-037 in DMSO or 0.5% (v/v) DMSO alone for 72\\xa0h before harvesting and reseeding for colony formation. After growth for 10\\u201314\\xa0days, colonies were fixed in methanol\\u2013acetic acid 3:1 (v/v) and stained with crystal violet (0.4%\\xa0w/v). Colonies consisting of more than 50 cells were counted on an automated colony counter (Oxford Optronix, Oxford, UK). Two-tailed paired t tests were used to compare the different groups. Differences with a p\\xa0<\\xa00.05 were considered statistically significant.\\n\\n\\nWestern blotting\\nCells were treated with WX-554 and WX-037 at 1 or 10 times the half maximal growth inhibitory concentration (GI50) in DMSO, or 0.5% (v/v) DMSO alone, for 24\\xa0h. Western blots were prepared, probed with phospho-4EBP1 (Thr37/46) (#2855), phospho-p44/42 MAPK (Thr202/Tyr204) (#4370), phospho-AKT (Ser473) (#4060) or phospho-S6 ribosomal protein (Ser235/236) (#4858) monoclonal antibodies obtained from Cell Signalling Technology (New England BioLabs (UK) Ltd, Hertfordshire, UK) and developed as described previously [23]. Blots were then stripped (100\\xa0mM 2-mercaptoethanol, 2% (w/v) SDS and 62.5\\xa0mM Tris pH6.8 at 55\\xa0\\xb0C for 30\\xa0min) and re-probed with the respective total monoclonal antibody (4EBP1 (53H11) (#9644), p44/42 MAPK (ERK1/2) (#4695), AKT (pan) (C67E7) (#4691) or S6 ribosomal protein (5g10) (#2217)) obtained from Cell Signalling Technology (New England BioLabs (UK) Ltd, Hertfordshire, UK) and developed as described above.\\n\\n\\nPharmacokinetic (PK) studies\\nMice bearing HCT116 or HT29 human tumour xenografts were treated with 1 or 5\\xa0mg/kg WX-554, or 20 or 100\\xa0mg/kg WX-037, alone or in combination, and were bled by cardiac puncture under terminal anaesthesia at 6 or 24\\xa0h post-treatment (3 mice/time point). Blood was collected into heparinized tubes, and plasma was separated and stored at \\u221220\\xa0\\xb0C until analysed. Tumours were removed, snap frozen in liquid nitrogen and stored at \\u221280\\xa0\\xb0C prior to PK analyses. Samples were extracted with solid phase extraction (SPE) and analysed with high-performance liquid chromatography (HPLC) coupled with tandem mass spectrometry (MS/MS) by Wilex (Munich, Germany). The quantification limit for WX-554 was 1\\xa0ng/mL and within and between day variation was <15%. WinNonlin Software Version 4.0.1 (Pharsight Corporation, Peypin, France) was used for PK/PD modelling and non-compartmental analysis. Paired t tests were used to compare the different treatment groups, and differences with a p value\\xa0\\u22640.05 were considered statistically significant.\\n\\n\\nDetermination of anti-tumour activity\\nMice bearing HCT116 human tumour xenografts were randomized into treatment groups and then treated by oral gavage with either the vehicle (10\\xa0ml/kg), 2\\xa0mg/kg WX-554, 50\\xa0mg/kg WX-037 or the combination of 2\\xa0mg/kg WX-554 and 50\\xa0mg/kg WX-037 once daily for 14\\xa0days. Tumour volume was monitored by calliper measurement using the equation a\\n2\\xa0\\xd7\\xa0b/2, where a is the smallest measurement and b the largest. Data are presented as median relative tumour volumes (RTV), where the tumour volume in each mouse on the initial day of treatment (day 0) is assigned an RTV value of 1. The time to RTV4 for each individual tumour was calculated based on a standard point to point curve with 1000 segments using GraphPad Prism software (CA, USA). Mann\\u2013Whitney U tests were used to compare the different groups, i.e., the control versus each treatment group, the single agents versus each other and each single agent versus their combination. Differences with a p value\\xa0\\u22640.05 were considered statistically significant.\\n\\n\\nResults\\nThe PI3K inhibitor WX-037 and the MEK inhibitor WX-554 are synergistic and exhibit increased cytotoxicity in combination in vitro\\nThe growth inhibitory activity of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, as single agents, in HCT116 and HT29 cells was measured using the SRB assay (Supplementary Figure\\xa01). Both drugs induced over 65% growth inhibition in both the colorectal cell lines. The results were used to determine the half maximal growth inhibitory (GI50) concentration of the drugs after 72-h exposure. The MEK inhibitor WX-554 was found to have GI50 values of 38 and 4.3\\xa0nM, whereas the PI3K inhibitor WX-037 was less potent with GI50 values of 2934 and 112\\xa0nM in the HCT116 and HT29 cell lines, respectively (Supplementary Figure\\xa01).\\n\\n\\nStudies were then performed to determine the effect of combining the PI3K and MEK inhibitors on colorectal carcinoma cell growth over 72\\xa0h. WX-037 and WX-554 were used alone at 0.25x, 0.5x, 1x, 2x and 4x their respective GI50 concentration, as calculated from Supplementary Figure\\xa01, and at equipotent concentrations at the same GI50 ratios in combination. Figure\\xa01 shows that the combination of WX-037 and WX-554 was markedly more growth inhibitory than either compound alone, completely inhibiting growth at the highest concentrations. Data were then evaluated by median effect analysis (CalcuSyn, Biosoft, Great Shelford, UK) to determine whether the greater growth inhibitory activity of the combination of WX-554 and WX-037 reflected an additive or a synergistic effect. The combination of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 was strongly synergistic when combined at the GI50 concentration compared to the compounds alone (Supplementary Table\\xa01).Fig.\\xa01\\nGrowth inhibition induced by the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, alone and in combination, in the HCT116 and HT29 cell lines. HCT116 (a) and HT29 (b) cells were treated with the indicated fractions of the GI50 concentrations of the inhibitors, alone or in combination, derived from Supplementary Figure\\xa01, for 72\\xa0h, and an SRB assay was subsequently performed. Growth is presented as a percentage of the control, in which cells were treated with 0.5% (v/v) DMSO. Points represent the mean of 3 independent experiments \\xb1 standard error. Lines were fitted using nonlinear regression analysis\\n\\n\\n\\n\\n\\nCell survival after 72-h exposure to the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 was also measured using a clonogenic cytotoxicity assay. Single agent WX-554 showed significant cytotoxicity at 10\\xa0\\xb5M with 67% cell kill in the HCT116 cell line and 75% in the HT29 cell line; however, the mean lethal concentration (LC50) of 0.6\\xa0\\xb5M and 1.6\\xa0\\xb5M WX-554 was approximately 16-fold and 372-fold higher than the corresponding GI50- values in the HCT116 and HT29 cell lines, respectively. WX-037 showed no marked cytotoxicity with less than 50% cell death after 72\\xa0h treatment at 10\\xa0\\xb5M (Supplementary Figure\\xa02).\\n\\n\\nThe cytotoxicity of the PI3K and MEK inhibitors in combination after 72\\xa0h treatment was then determined. However, as WX-037 did not produce\\xa0>\\xa050% cytotoxicity at 10\\xa0\\xb5M, it was not possible to determine an LC50 value, and hence the highest concentration previously used of 10\\xa0\\xb5M WX-037 was combined with 10\\xa0\\xb5M WX-554. There was a statistically significant increase in cytotoxicity when the PI3K and MEK inhibitors were combined, compared to the cytotoxicity induced by the drugs as single agents, in the HCT116 (p\\xa0=\\xa00.02) and HT29 (p\\xa0<\\xa00.01) cell lines (Fig.\\xa02). Overall, the interaction of WX-037 and WX-554 resulted in significantly enhanced cell growth inhibition and an increase in cytotoxicity in both cell lines studied.Fig.\\xa02\\nCell survival after 72-h exposure to 10\\xa0\\xb5M of the PI3K inhibitor WX-037 and 10\\xa0\\xb5M of the MEK inhibitor WX-554, alone and in combination, in the HCT116 cell line and HT29 cell lines. HCT116 (a) and HT29 (b) cells were treated with a fixed concentration of each inhibitor alone or in combination for 72\\xa0h, and cell survival was subsequently determined by clonogenic assay after 10\\u201314\\xa0days of colony growth. Survival is presented as a percentage of the control, in which cells were treated with 0.5% (v/v) DMSO. Bars represent the mean of 3 independent replicates \\xb1 standard error. *Significantly different from either agent alone, p\\xa0\\u2264\\xa00.05\\n\\n\\n\\n\\n\\nThe effect of 24-h exposure to the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, both as single agents and in combination, was also investigated by Western blotting to determine the effect on the PI3K/AKT signalling pathway, using total and phospho-specific antibodies for AKT, S6 and 4EBP1 and the effect on MAPK signalling, using total and phospho-specific antibodies for ERK1/2. The compounds were used as single agents or in combination at their respective GI50 concentrations and at 10x the GI50 concentration.\\n\\n\\nSupplementary Figure\\xa03 shows that treatment with the MEK inhibitor WX-554 reduced ERK1/2 phosphorylation at 1 and 10 times the GI50 concentration in the HCT116 cell line, and at 10 times the GI50 concentration in the HT29 cell line. The reduction in ERK1/2 phosphorylation was enhanced with the combination of WX-554 and WX-037 leading to complete inhibition with 10 times the GI50 concentration in both colorectal carcinoma cell lines (Supplementary Figure\\xa03). Additionally, there was a concentration-dependent reduction in AKT phosphorylation in the HCT116 cell line after treatment with the single agent PI3K inhibitor WX-037 which was enhanced, to yield complete inhibition at the GI50 concentration, after combined WX-037 and WX-554 treatment. In the HT29 cell line, there was a reduction in AKT phosphorylation at 10 times the GI50 concentration; however, this reduction was similar with single agent WX-037 and the combination (Supplementary Figure\\xa03).\\n\\n\\nSingle agent WX-037 also caused a concentration-dependent reduction in S6 phosphorylation at 1 and 10 times the GI50 concentration, and treatment with the combination of WX-037 and WX-554 enhanced this inhibition, causing complete inhibition at 10 times the GI50 concentration in both colorectal carcinoma cell lines (Supplementary Figure\\xa03). However, WX-037 had no marked effect on the phosphorylation of 4EBP1 alone or in combination with WX-554 in either colorectal carcinoma cell line (Supplementary Figure\\xa03). Hence, overall, the combination of the MEK inhibitor WX-554 and the PI3K inhibitor WX-037 resulted in enhanced inhibition of ERK1/2 and S6 phosphorylation, and inhibition of AKT phosphorylation, in both colorectal carcinoma cell lines. However, both the single agents and the combination had no major impact on 4EBP1 phosphorylation.\\n\\n\\nThe PI3K inhibitor WX-037 and the MEK inhibitor WX-554 exhibit increased tumour growth delay in combination in vivo\\nA PK study was carried out with samples taken 6 and 24\\xa0h after treatment with a single dose of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, alone and in combination, in HCT116 and HT29 human tumour xenograft-bearing mice. The concentrations of the drugs in the plasma and the tumour tissue were measured using LC\\u2013MS/MS (Figs.\\xa03, 4, Supplementary Figures\\xa04 and 5 and Supplementary Table\\xa02).Fig.\\xa03\\nConcentrations of the MEK inhibitor WX-554 alone and in combination with the PI3K inhibitor WX-037 in tumours from mice bearing HCT116 or HT29 human tumour xenografts. Tumour concentrations of WX-554 measured by LC\\u2013MS/MS from HCT116 (a, c) and HT29 (b, d) tumour xenograft-bearing mice at the indicated time points after a single p.o. dose of 1\\xa0mg/kg (a, b) or 5\\xa0mg/kg (c, d) WX-554 alone or combined with 20 or 100\\xa0mg/kg WX-037. Data are presented as the mean concentration from 3 mice in each group \\xb1 standard error. Horizontal dashed lines indicate the in vitro GI50 concentration for the respective cell line, calculated from Supplementary Figure\\xa01\\n\\n\\nFig.\\xa04\\nConcentrations of the PI3K inhibitor WX-037 alone and in combination with the MEK inhibitor WX-554 in tumours from mice bearing HCT116 or HT29 human tumour xenografts. Tumour concentrations of WX-037 measured by LC\\u2013MS/MS from HCT116 (a, c) and HT29 (b, d) tumour xenograft-bearing mice at the indicated time points after a single p.o. dose of 20\\xa0mg/kg (a, b) or 100\\xa0mg/kg (c, d) WX-037 alone or combined with 1 or 5\\xa0mg/kg WX-554. Data are presented as the mean concentration from 3 mice in each group \\xb1 standard error. Horizontal dashed lines indicate the in vitro GI50 concentration for the respective cell line, calculated from Supplementary Figure\\xa01\\n\\n\\n\\n\\n\\nAfter a single dose of 1 or 5\\xa0mg/kg WX-554, alone or in combination with 20 or 100\\xa0mg/kg WX-037, WX-554 concentrations in the plasma and tumour tissue generally greatly exceeded the in vitro GI50 value of 18\\xa0ng/ml (38\\xa0nM) in the HCT116 cell line and 2\\xa0ng/ml (4\\xa0nM) in the HT29 cell line (determined in Supplementary Figure\\xa01) (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A). The exception to this was that plasma WX-554 concentrations were only approximately equal to the HCT116 GI50 value at 6\\xa0h and were below the HCT116 GI50 value by 24\\xa0h, in the HCT116 tumour xenograft-bearing mice after treatment with 1\\xa0mg/kg WX-554, alone or in combination with 20 or 100\\xa0mg/kg WX-037 (Supplementary Figure\\xa04A and Supplementary Table\\xa02A). Additionally, the plasma WX-554 concentration was also below the HCT116 GI50 value by 24\\xa0h after 5\\xa0mg/kg WX-554 alone; however, WX-554 concentrations still exceeded the GI50 value when 5\\xa0mg/kg WX-554 was combined with WX-037 (Supplementary Figure\\xa04C and Supplementary Table\\xa02A). However, overall, the absolute plasma and tumour levels of WX-554 were generally similar in the HT29 and HCT116 tumour xenograft-bearing mice (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A).\\n\\n\\nThere was no consistent effect of concomitant dosing with 20 or 100\\xa0mg/kg WX-037 on the levels of WX-554 in the tumour or the plasma after administration of 1\\xa0mg/kg WX-554 in either HCT116 or HT29 tumour xenograft-bearing mice. In contrast, after dosing with 5\\xa0mg/kg WX-554, there was generally a WX-037 dose-dependent decrease in the levels of WX-554 at 6\\xa0h in the tumour and the plasma upon concomitant treatment with WX-037, which was significant for the tumour data at 6\\xa0h (p\\xa0<\\xa00.05) (Fig.\\xa03, Supplementary Figure\\xa04 and Supplementary Table\\xa02A). These results suggest that WX-037 may delay the tumour uptake of WX-554 at the higher dose (5\\xa0mg/kg) compared to when WX-554 is administered alone.\\n\\n\\nIn contrast to the data for WX-554, after treatment with 20\\xa0mg/kg WX-037, as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554, WX-037 concentrations in plasma and tumour tissue at 6 and 24\\xa0h were markedly lower than the in vitro GI50 value of 1414\\xa0ng/ml (2934\\xa0nM) in the HCT116 cell line (determined in Supplementary Figure\\xa01) (Fig.\\xa04a, Supplementary Figure\\xa05A and Supplementary Table\\xa02B). Plasma and tumour WX-037 concentrations were also markedly below the HCT116 GI50 value 24\\xa0h after treatment with 100\\xa0mg/kg WX-037, given as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554; however, concentrations in the plasma and tumour tissue were similar to or exceeded the HCT116 GI50 value 6\\xa0h after a 100\\xa0mg/kg dose of WX-037 (Fig.\\xa04c, Supplementary Figure\\xa05C and Supplementary Table\\xa02B).\\n\\n\\nAfter treatment with 20 or 100\\xa0mg/kg WX-037, as a single agent or in combination with 1 or 5\\xa0mg/kg WX-554, concentrations in plasma and tumour tissue greatly exceeded the in vitro GI50 value of 54\\xa0ng/ml (112\\xa0nM) in the HT29 cell line (determined in Supplementary Figure\\xa01) at 6\\xa0h. At 24\\xa0h, although concentrations were similar to or exceeded the HT29 GI50 value in all tumour samples and in plasma after a 100\\xa0mg/kg dose of WX-037, plasma concentrations had generally declined to below the GI50 value after a 20\\xa0mg/kg dose of WX-037 (Fig.\\xa04b\\u2013d, Supplementary Figure\\xa05B and D and Supplementary Table\\xa02B). As with the WX-554 data, the absolute plasma and tumour levels of WX-037 were similar in the HCT116 and HT29 tumour xenograft-bearing mice (Fig.\\xa04, Supplementary Figure\\xa05 and Supplementary Table\\xa02B).\\n\\n\\nThere did not appear to be a consistent effect of concomitant dosing with 1 or 5\\xa0mg/kg WX-554 on the levels of WX-037 in the tumour or the plasma at 6\\xa0h after dosing with 20\\xa0mg/kg in either HCT116 or HT29 tumour xenograft-bearing mice (Fig.\\xa04a, b, Supplementary Figure\\xa05A and B and Supplementary Table\\xa02B). However, after dosing with 100\\xa0mg/kg, there appeared to be a dose-dependent decrease in the levels of WX-037 in the tumour and the plasma upon concomitant dosing with WX-554 at 6\\xa0h, but this effect was only significant in HCT116 tumours (Fig.\\xa04c, d, Supplementary Figure\\xa05C and D and Supplementary Table\\xa02B). Hence, concentrations of WX-037 were generally similar regardless of whether it was administered as a single agent or in combination with WX-554, and levels were consistently higher at 6\\xa0h compared with 24\\xa0h.\\n\\n\\nAs the combination of WX-554 and WX-037 was synergistic in the in vitro studies, it may not be necessary for the drug concentrations in the plasma and the tumour to exceed those of the in vitro GI50 values for the single agents to achieve efficacy with the combination in vivo. Based on the in vitro results, in order to achieve half maximal growth inhibition, less than 1/6th and approximately 1/3rd of the single agent GI50 was required for 50% growth inhibition with the drug combination in the HCT116 and HT29 cell lines, respectively, which equates to GI50 values of 3 and <1\\xa0ng/ml WX-554 and 219 and 20\\xa0ng/ml WX-037 in the HCT116 and HT29 cell lines (calculated from Fig.\\xa01). Therefore, with all the combinations of WX-554 and WX-037, WX-554 levels in the plasma and tumour tissues are at or exceed the GI50 concentration for the combination at both 6 and 24\\xa0h (Fig.\\xa03, Supplementary Figures\\xa04 and Supplementary Table\\xa02A). Furthermore, with all combinations of WX-554 and 100\\xa0mg/kg WX-037, WX-037 levels in the plasma and tumour tissues are at or exceed the GI50 for the combination at both 6 and 24\\xa0h, and with all combinations of WX-554 with 20\\xa0mg/kg WX-037, WX-037 levels in the plasma and tumour tissues are at or exceed the GI50 for the combination at 6\\xa0h, but remain below at 24\\xa0h (Fig.\\xa04, Supplementary Figure\\xa05 and Supplementary Table\\xa02B).\\n\\n\\nBased on the results of the PK study, the efficacy of 50\\xa0mg/kg of the PI3K inhibitor WX-037 and 2\\xa0mg/kg of the MEK inhibitor WX-554 given orally, as single agents and in combination, was assessed in HCT116 human tumour xenograft-bearing mice (Fig.\\xa05). The individual doses of the PI3K and MEK inhibitors were chosen to be approximately equiactive, in order to mirror the in vitro conditions under which synergy had been demonstrated (Supplementary Figure\\xa01). In this study, mice were treated daily for 14\\xa0days and tumour volumes were measured three times a week. Figure\\xa05a demonstrates that treatment with 50\\xa0mg/kg WX-037 and 2\\xa0mg/kg WX-554, alone and in combination, caused tumour growth delay compared to vehicle-treated control tumours, and that growth delay was greater with the combination. Additionally, body weight was monitored daily to assess the tolerability of the therapy, and both single agent and combination treatments were found to be well tolerated as average body weights did not drop below 89% of the starting weight (Fig.\\xa05b). The time for the tumours to quadruple in size (time to RTV4) was calculated (Fig.\\xa05c), and statistical analyses using a Mann\\u2013Whitney test demonstrated a significant difference between vehicle-treated control tumours and the combination group (p\\xa0<\\xa00.01), and between the single agent MEK inhibitor and the combination group (p\\xa0=\\xa00.02).Fig.\\xa05\\nEfficacy and tolerability of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554 in mice bearing human HCT116 colorectal tumour xenografts. HCT116 tumour xenografts were treated with either vehicle control, 2\\xa0mg/kg of the MEK inhibitor WX-554 and 50\\xa0mg/kg of the PI3K inhibitor WX-037 alone, or 2\\xa0mg/kg of the MEK inhibitor WX-554 and 50\\xa0mg/kg of the PI3K inhibitor WX-037 in combination, p.o. once daily for 14\\xa0days. a Tumour growth curves: data are presented as the median relative tumour volume (RTV), where the growth is calculated for each tumour relative to its size on day 0. Points represent the median of the 10 mice in each group. The dashed line shows the point at which tumours reached four times the initial volume (RTV4). b Effects on body weight: data are presented as a percentage of starting body weight. Points represent the mean of the mice in each group \\xb1 standard error. c Time taken for xenografts to reach four times the initial volume (time to RTV4): data are presented as the time taken by each individual tumour in each group to quadruple in size, and lines to represent the mean of the mice in each group \\xb1 standard error. p values are given where the combination is significantly different from either agent alone (p\\xa0\\u2264\\xa00.05)\\n\\n\\n\\n\\n\\nDiscussion\\nThe novel PI3K inhibitor WX-037 and the novel MEK inhibitor WX-554 have demonstrated in vitro activity in a range of cancer cell lines including breast, fibrosarcoma, thyroid, melanoma, colorectal, ovarian and pancreas lines with a broad range of GI50 values, where generally the cell lines most sensitive to WX-037 had PIK3CA mutations or PTEN loss, and to WX-554 had BRAF or RAS mutations [5]. Furthermore, the potency of WX-554 determined in the HCT116 and HT29 cell lines in this study (38 and 4.3\\xa0nM, respectively) was similar to that determined in the unpublished Wilex studies (29 and 7.2\\xa0nM) [5], and to the MEK inhibitor, PD 0325901 (21 and 6.5\\xa0nM) using the same assay [23]. Similarly the potency of WX-037 determined here in the HCT116 and HT29 cell lines (2934 and 112\\xa0nM, respectively) was similar to that reported in the unpublished Wilex studies (136\\xa0nM in the HT29 cell line) [5], and to the pan class I PI3K inhibitor, pictilisib (1081 and 157\\xa0nM) using the same assay [23].\\n\\n\\nIn these in vitro studies, the HT29 colorectal carcinoma cell line was consistently more sensitive to WX-037 and WX-554 than the HCT116 cell line (26 and 9-fold difference, respectively), which is consistent with other studies where the HT29 cell line exhibited increased sensitivity to pan class I PI3K and/or MEK inhibition, compared to the HCT116 cell line [23, 28, 29]. The observed difference could be due to the presence of a BRAF, rather than a KRAS, mutation as RAS mutations have been associated with intrinsic resistance in a previous study [30]. Alternatively, differences in cell signalling may be responsible, as sensitivity to MEK inhibition was found to correlate with strong ERK1/2 signalling and weak PI3K signalling [28]. Furthermore, the plethora of other mutations found in the HT29 and HCT116 cell lines may contribute to the difference in sensitivity [29, 31].\\n\\n\\nThe combination of WX-037 and WX-554 was strongly synergistic, which is consistent with previous unpublished studies by UCB Celltech and Wilex where synergy was observed in the SK-MEL-28 melanoma, HT29 colorectal, Mia PaCa-2 pancreatic and SKOV-3 ovarian carcinoma cell lines [5]. The degree of synergy with the WX-037 and WX-554 combination was consistent with in vitro published studies using combinations of other PI3K and MEK inhibitors [2, 18, 23, 24, 32, 33].\\n\\n\\nSingle agent WX-554 not only exhibited growth inhibition but also significant cytotoxicity, albeit at higher concentrations with LC50 values approximately 16-fold and 372-fold higher than the corresponding GI50\\u2212 values in the HCT116 and HT29 cell lines, respectively. This marked cytotoxicity at 10\\xa0\\xb5M is in contrast to the effect of PD 0325901 and selumetinib [23] and also to the general perspective that MEK inhibitors are cytostatic [34]. However, another study has already reported that selumetinib can cause apoptosis in some human tumour xenografts, but not in others, which was suggested to be due to differences in ERK1/2 substrate expression or differential cell signalling networks [35], and hence the result with WX-554 is not entirely unexpected.\\n\\n\\nIn contrast to the effect of the MEK inhibitor, the PI3K inhibitor WX-037 at 10\\xa0\\xb5M showed only limited cytotoxicity, which is consistent with the results observed with another pan class I PI3K inhibitor pictilisib [23]. However, the marked synergy observed in the growth inhibition studies with WX-037 and WX-554 translated into cytotoxicity, albeit at a high concentration (10\\xa0\\xb5M), as there was a statistically significant increase in cytotoxicity with the combination of the Wilex PI3K and MEK inhibitors. This cytotoxicity is consistent with previous unpublished studies by UCB Celltech and Wilex where the observed growth inhibitory synergy with WX-037 and WX-554 in the SK-MEL-28 melanoma, HT29 colorectal, Mia PaCa-2 pancreatic and SKOV-3 ovarian carcinoma cell lines was found to correlate with the induction of apoptosis [5]. Similarly, published in vitro studies have shown that combined inhibition of PI3K and MEK resulted in cell death [33, 36, 37].\\n\\n\\nCell signalling studies showed that the single agent PI3K inhibitor WX-037 caused a concentration-dependent reduction in AKT and S6 phosphorylation, which was generally enhanced by the MEK inhibitor WX-554, but that 4EBP1 phosphorylation was not inhibited, consistent with the results of published studies using pan class I PI3K, as opposed to mixed PI3K/mTOR, inhibitors [23, 32, 37, 38]. Furthermore, single agent MEK inhibitor WX-554 caused a concentration-dependent reduction in ERK phosphorylation, and the combination of WX-554 and WX-037 resulted in enhanced inhibition of ERK1/2 phosphorylation, as observed with other MEK inhibitors [23, 39, 40] and combinations [23, 33].\\n\\n\\nIn the pre-clinical studies presented here, and in both pre-clinical and clinical studies reported elsewhere [5, 6], WX-554 has been shown to be non-toxic. Furthermore, single agent WX-554 induced tumour growth delay at 2\\xa0mg/kg in HCT116 colorectal tumour xenograft models, consistent with previous unpublished studies using WX-554 and published studies using other MEK inhibitors that reported tumour growth delay or stasis in vivo, with increased sensitivity in BRAF or RAS mutant cells and tumours [3\\u20135, 25]. In vivo pharmacokinetic analyses revealed concentrations of WX-554 in plasma and tumour tissue increased in a dose-dependent manner and that tumour WX-554 levels generally exceeded the in vitro GI50 concentration, whereas plasma levels were more variable.\\n\\n\\nSingle agent WX-037 was also found to be non-toxic in these studies and those reported elsewhere [5, 17], which is consistent with data for other pan class I PI3K inhibitors [41\\u201345]. Furthermore, 50\\xa0mg/kg WX-037 generated a tumour growth delay in HCT116 colorectal tumour xenograft models, which is consistent with previous unpublished studies using WX-037 and published studies using other pan class I PI3K inhibitors that reported tumour growth delay, or in some cases tumour stasis, with greater sensitivity in PIK3CA mutant or PTEN null cells and tumours [5, 14, 15, 46\\u201351]. Pharmacokinetic studies indicated that concentrations of WX-037 in plasma and tumour tissue increased dose dependently and that WX-037 tumour and plasma levels generally exceeded the in vitro GI50 concentration at an early time point (6\\xa0h), depending on the cell line and dose, whereas levels generally had decreased below the GI50 value by the later time point (24\\xa0h). These pharmacokinetic data are similar to those observed in HCT116 tumour xenograft-bearing mice treated with pictilisib [25].\\n\\n\\nThe PK analyses indicated that there was no pharmacokinetic interaction between WX-037 and WX-554 at lower doses. However, concomitant dosing of WX-037 with the high dose (5\\xa0mg/kg) of WX-554 caused a dose-dependent decrease in early time point (6\\xa0h) WX-554 tumour and plasma concentrations, and a dose-dependent increase in late time point (24\\xa0h) WX-554 levels, suggesting that the presence of WX-037 may delay the uptake of WX-554, leading to higher WX-554 levels at 24\\xa0h. Furthermore, concomitant WX-554 dosing with high doses (100\\xa0mg/kg) of WX-037 caused a dose-dependent decrease in early time point (6\\xa0h) WX-037 tumour and plasma concentrations; however, this effect was only significant in the HCT116 tumour tissue due to variation in the data. The pharmacokinetic interaction between higher doses of WX-037 and WX-554 is in contrast to the results obtained with the combination of PD 0325901 and pictilisib in HCT116 tumour xenograft-bearing mice [25], where there was no significant pharmacokinetic interaction, and to pre-clinical and clinical reports that there was no pharmacokinetic interaction between the PI3K and MEK inhibitors, pictilisib and cobimetinib [52, 53]. However, a recent Phase Ib study using a combination of the PI3K inhibitor buparlisib and the MEK inhibitor trametinib reported a potential pharmacokinetic interaction, which was suggested to be due to the variability of the accumulation of the MEK inhibitor as no clear mechanism could be identified [54].\\n\\n\\nIn HCT116 colorectal tumour xenografts, the combination of 50\\xa0mg/kg WX-037 and 2\\xa0mg/kg WX-554 was non-toxic and caused tumour stasis and enhanced tumour growth delay compared to either single agent at the same dose. This improved activity is consistent with previous studies using combinations of PI3K and MEK inhibitors in a range of pre-clinical human tumour xenograft and mouse models [19, 21, 22, 25, 55\\u201358]. In a previous study using the MEK inhibitor, PD 0325901, and the PI3K inhibitor, GDC-0941, the cell line-dependent differences in sensitivity to the inhibitors determined in the in vitro studies did not correlate with the results observed in the in vivo studies [23, 25], suggesting that HT29 xenografts would not necessarily of been more sensitive to the inhibitors in vivo. Nevertheless, previous studies by UCB and Wilex have demonstrated that WX-554 treatment is able to induce tumour growth delay in HT29 xenografts [5].\\n\\n\\nThe improved efficacy using combinations of PI3K and MEK inhibitors in pre-clinical studies is being translated into clinical trials [45, 54, 59], and there are currently at least 7 ongoing or completed Phase 1b clinical studies [9]. However, there have been issues with the tolerability of the combination treatment in two phase 1b trials combining the MEK inhibitor trametinib, with either the pan-PI3K inhibitor buparlisib, or the PI3K/mTOR inhibitor GSK2126458, where the former study showed promising anti-tumour activity but frequent dose interruptions and reductions were necessary due to toxicity [54]. The latter study was terminated due to poor tolerability and efficacy [9, 60]. These results suggest that the choice of the PI3K and MEK inhibitor to be combined will be crucial to the success of the treatment, due to subtle differences in the pharmacology of the inhibitors leading to differing toxicity profiles. Overall, these studies characterize, for the first time, the in vitro and in vivo efficacy of the PI3K inhibitor WX-037 and the MEK inhibitor WX-554, as single agents and in combination. Furthermore, these studies illustrate that dual targeting of PI3K and MEK can induce synergy in vitro which translates to marked tumour growth delay in vivo.\\n\\n\\nElectronic supplementary material\\nBelow is the link to the electronic supplementary material. \\n\\nSupplementary material 1 (PDF 520\\xa0kb)\\n\\n\\n\\nSupplementary material 2 (PDF 81\\xa0kb)\\n\\n\\n\\n\\n\\nAcknowledgements\\nThe pharmacokinetic studies were performed by employees of Wilex.\\n\\n\\nFunding\\nHDT, RJM and DRN are funded by a CRUK Drug Discovery Programme C240/A7409, and EJH was an MRC CASE Award PhD Student supported by UCB Celltech, Slough, United Kingdom.\\n\\n\\nCompliance with ethical standards\\nConflict of interest\\nPaul Bevan and Wolfgang Schmalix were employees of Wilex at the time that the research was conducted. Andrew Payne, Lara Kevorkian and Rodger Allen are employees of UCB Celltech, and Rodger Allen also consults for UCB Celltech.\\n\\n\\nEthical approval\\nAll in vivo experiments were reviewed and approved by the Newcastle University (UK) animal welfare committee and were performed according to the guidelines for the welfare and use of animals in cancer research [27] and national law, under project license (PPL60/4442) issued by the UK Government Home Office under the animals (scientific procedure) act 1986.\\n\\n\\nReferences1.HennigMYip-SchneiderMTWentzSWuHHekmatyarSKKleinPBansalNSchmidtCMTargeting mitogen-activated protein kinase kinase with the inhibitor PD0325901 decreases hepatocellular carcinoma growth in vitro and in mouse model systemsHepatology20105141218122510.1002/hep.23470201124262.LiuDXingMPotent Inhibition of thyroid cancer cells by the MEK Inhibitor PD0325901 and Its potentiation by suppression of the PI3K and NF-\\u03baB pathwaysThyroid200818885386410.1089/thy.2007.0357186518023.AkinleyeAFurqanMMukhiNRavellaPLiuDMEK and the inhibitors: from bench to bedsideJ Hematol Oncol201362710.1186/1756-8722-6-27235874174.TempletonIEMusibLMEK inhibitors beyond monotherapy: current and future developmentCurr Opin Pharmacol201523616710.1016/j.coph.2015.05.012260572125.Wilex AG (unpublished data)6.MalaCNevilleNGHaindlEBuergleMSchmalixWBevanPA phase I, first-in-human single ascending dose study of the MEK inhibitor WX-554 given to healthy male subjectsJ Clin Oncol201028(suppl; abstr)e136667.Wilex AG (2012). http://www.wilex.de. Accessed 23/07/128.MillerCROliverKEFarleyJHMEK1/2 inhibitors in the treatment of gynecologic malignanciesGynecol Oncol2014133112813710.1016/j.ygyno.2014.01.008244340599.Clinical Trials gov (2016). http://clinicaltrials.gov/. Accessed 22/03/201610.MairaS-MStaufferFBrueggenJFuretPSchnellCFritschCBrachmannSChenePDe PoverASchoemakerKFabbroDGabrielDSimonenMMurphyLFinanPSellersWGarcia-EcheverriaCIdentification and characterization of NVP-BEZ235, a new orally available dual phosphatidylinositol 3-kinase/mammalian target of rapamycin inhibitor with potent in vivo antitumor activityMol Cancer Ther2008771851186310.1158/1535-7163.MCT-08-00171860671711.SerraVMarkmanBScaltritiMEichhornPJAValeroVGuzmanMBoteroMLLlonchEAtzoriFDi CosimoSMairaMGarcia-EcheverriaCParraJLArribasJBaselgaJNVP-BEZ235, a Dual PI3K/mTOR inhibitor, prevents PI3K signaling and inhibits the growth of cancer cells with activating PI3K mutationsCancer Res200868198022803010.1158/0008-5472.CAN-08-13851882956012.BaumannPMandl-WeberSOduncuFSchmidmaierRThe novel orally bioavailable inhibitor of phosphoinositol-3-kinase and mammalian target of rapamycin, NVP-BEZ235, inhibits growth and proliferation in multiple myelomaExp Cell Res2009315348549710.1016/j.yexcr.2008.11.0071907110913.MairaS-MStaufferFSchnellCGarcia-echeverriaCPI3K inhibitors for cancer treatment: where do we stand?Biochem Soc Trans200937126527210.1042/BST03702651914364414.FolkesAJAhmadiKAldertonWKAlixSBakerSJBoxGChuckowreeISClarkePADepledgePEcclesSAFriedmanLSHayesAHancoxTCKugendradasALensunLMoorePOliveroAGPangJPatelSPergl-WilsonGHRaynaudFIRobsonASaghirNSalphatiLSohalSUltschMHValentiMWallweberHJAWanNCWiesmannCWorkmanPZhyvoloupAZvelebilMJShuttleworthSJThe Identification of 2-(1H-Indazol-4-yl)-6-(4-methanesulfonyl-piperazin-1-ylmethyl)-4-morpholin-4-yl-thieno[3,2-d]pyrimidine (GDC-0941) as a potent, selective, orally bioavailable inhibitor of class I PI3Kinase for the treatment of cancerJ Med Chem200851185522553210.1021/jm800295d1875465415.RaynaudFIEcclesSAPatelSAlixSBoxGChuckowreeIFolkesAGowanSDe HavenBrandon ADi StefanoFHayesAHenleyATLensunLPergl-WilsonGRobsonASaghirNZhyvoloupAMcDonaldESheldrakePShuttleworthSValentiMWanNCClarkePAWorkmanPBiological properties of potent inhibitors of class I phosphatidylinositide 3-kinases: from PI-103 through PI-540, PI-620 to the oral agent GDC-0941Mol Cancer Ther2009871725173810.1158/1535-7163.MCT-08-12001958422716.MayerIAArteagaCLThe PI3K/AKT pathway as a target for cancer treatmentAnnu Rev Med201667112810.1146/annurev-med-062913-0513432647341517.Wilex AG (2013). http://www.wilex.de. Accessed 30/03/201618.YuKToral-BarzaLShiCZhangWGZaskAResponse and determinants of cancer cell susceptibility to PI3K inhibitors: combined targeting of PI3K and Mek1 as an effective anticancer strategyCancer Biol Ther20087231031810.4161/cbt.7.2.533419.EngelmanJAChenLTanXCrosbyKGuimaraesARUpadhyayRMairaMMcNamaraKPereraSASongYChirieacLRKaurRLightbownASimendingerJLiTPaderaRFGarcia-EcheverriaCWeisslederRMahmoodUCantleyLCWongK-KEffective use of PI3K and MEK inhibitors to treat mutant Kras G12D and PIK3CA H1047R murine lung cancersNat Med200814121351135610.1038/nm.18901902998120.JokinenEKoivunenJPMEK and PI3K inhibition in solid tumors: rationale and evidence to dateTher Adv Med Oncol20157317018010.1177/17588340155711112667358021.SosMLFischerSUllrichRPeiferMHeuckmannJMKokerMHeynckSSt\\xfcckrathIWeissJFischerFMichelKGoelARegalesLPolitiKAPereraSGetlikMHeukampLCAns\\xe9nSZanderTBeroukhimRKashkarHShokatKMSellersWRRauhDOrrCHoeflichKPFriedmanLWongKKPaoWThomasRKIdentifying genotype-dependent efficacy of single and combined PI3K- and MAPK-pathway inhibition in cancerProc Nat Acad Sci USA200910643183511835610.1073/pnas.09073251061980505122.HoeflichKPMerchantMOrrCChanJDen OtterDBerryLKasmanIKoeppenHRiceKYangN-YEngstSJohnstonSFriedmanLSBelvinMIntermittent administration of MEK inhibitor GDC-0973 plus PI3K inhibitor GDC-0941 triggers robust apoptosis and tumor growth inhibitionCancer Res201272121021910.1158/0008-5472.CAN-11-15152208439623.HaagensenEJKyleSBealeGSMaxwellRJNewellDRThe synergistic interaction of MEK and PI3K inhibitors is modulated by mTOR inhibitionBr J Cancer201210681386139410.1038/bjc.2012.702241523624.HaagensenEJThomasHDMuddCTsonouEWigginsCMMaxwellRJMooreJDNewellDRPre-clinical use of isogenic cell lines and tumours in vitro and in vivo for predictive biomarker discovery; impact of KRAS and PI3KCA mutation status on MEK inhibitor activity is model dependentEur J Cancer (Oxford, England: 1990)201656697610.1016/j.ejca.2015.12.01225.HaagensenEJThomasHDWilsonIHarnorSJPayneSLRennisonTSmithKMMaxwellRJNewellDRThe enhanced in vivo activity of the combination of a MEK and a PI3K inhibitor correlates with [18F]-FLT PET in human colorectal cancer xenograft tumour-bearing micePLoS ONE2013812e8176310.1371/journal.pone.00817632433996326.BrittenCDPI3K and MEK inhibitor combinations: examining the evidence in selected tumor typesCancer Chemother Pharmacol20137161395140910.1007/s00280-013-2121-12344330727.WorkmanPAboagyeEOBalkwillFBalmainABruderGChaplinDJDoubleJAEverittJFarninghamDAHGlennieMJKellandLRRobinsonVStratfordIJTozerGMWatsonSWedgeSREcclesSAGuidelines for the welfare and use of animals in cancer researchBr J Cancer2010102111555157710.1038/sj.bjc.66056422050246028.BalmannoKChellSDGillingsASHayatSCookSJIntrinsic resistance to the MEK1/2 inhibitor AZD6244 (ARRY-142886) is associated with weak ERK1/2 signalling and/or strong PI3K signalling in colorectal cancer cell linesInt J Cancer2009125102332234110.1002/ijc.246041963731229.Genomics of Drug Sensitivity in Cancer (2012). http://www.cancerrxgene.org/. Accessed 05/10/201230.EbiHCorcoranRBSinghAChenZSongYLifshitsERyanDPMeyerhardtJABenesCSettlemanJWongK-KCantleyLCEngelmanJAReceptor tyrosine kinases exert dominant control over PI3K signaling in human KRAS mutant colorectal cancersJ Clin Investig2011121114311432110.1172/JCI579092198578431.GarnettMJEdelmanEJHeidornSJGreenmanCDDasturALauKWGreningerPThompsonIRLuoXSoaresJLiuQIorioFSurdezDChenLMilanoRJBignellGRTamATDaviesHStevensonJABarthorpeSLutzSRKogeraFLawrenceKMcLaren-DouglasAMitropoulosXMironenkoTThiHRichardsonLZhouWJewittFZhangTO\\u2019BrienPBoisvertJLPriceSHurWYangWDengXButlerAChoiHGChangJWBaselgaJStamenkovicIEngelmanJASharmaSVDelattreOSaez-RodriguezJGrayNSSettlemanJFutrealPAHaberDAStrattonMRRamaswamySMcDermottUBenesCHSystematic identification of genomic markers of drug sensitivity in cancer cellsNature2012483739157057510.1038/nature110052246090232.PittsTMNewtonTPBradshaw-PierceELAddisonRArcaroliJJKlauckPJBagbySMHyattSLPurkeyATentlerJJTanACMessersmithWAEckhardtSGLeongSDual pharmacological targeting of the MAP kinase and PI3K/mTOR pathway in preclinical models of colorectal cancerPLoS ONE2014911e11303710.1371/journal.pone.01130372540149933.KuBMJhoEHBaeYHSunJMAhnJSParkKAhnMJBYL719, a selective inhibitor of phosphoinositide 3-Kinase alpha, enhances the effect of selumetinib (AZD6244, ARRY-142886) in KRAS-mutant non-small cell lung cancerInvest New Drugs2015331122110.1007/s10637-014-0163-92534213934.KohnoMPouyssegurJTargeting the ERK signaling pathway in cancer therapyAnn Med200638320021110.1080/078538906005510371672043435.DaviesBRLogieAMcKayJSMartinPSteeleSJenkinsRCockerillMCartlidgeSSmithPDAZD6244 (ARRY-142886), a potent inhibitor of mitogen-activated protein kinase/extracellular signal-regulated kinase kinase 1/2 kinases: mechanism of action in vivo, pharmacokinetic/pharmacodynamic relationship, and potential for combination in preclinical modelsMol Cancer Ther2007682209221910.1158/1535-7163.MCT-07-02311769971836.WeeSJaganiZXiangKXLooADorschMYaoY-MSellersWRLengauerCStegmeierFPI3K pathway activation mediates resistance to MEK inhibitors in KRAS mutant cancersCancer Res200969104286429310.1158/0008-5472.CAN-08-47651940144937.EdgarKAWallinJJBerryMLeeLBPriorWWSampathDFriedmanLSBelvinMIsoform-specific phosphoinositide 3-kinase inhibitors exert distinct effects in solid tumorsCancer Res20107031164117210.1158/0008-5472.CAN-09-25252010364238.SantiskulvongCKonecnyGEFeketeMChenK-YMKaramAMulhollandDEngCWuHSongMDorigoODual targeting of phosphoinositide 3-kinase and mammalian target of rapamycin Using NVP-BEZ235 as a novel therapeutic approach in human ovarian carcinomaClin Cancer Res20111782373238410.1158/1078-0432.CCR-10-22892137222139.HauraEBRicartADLarsonTGStellaPJBazhenovaLMillerVACohenRBEisenbergPDSelaruPWilnerKDGadgeelSMA phase II study of PD-0325901, an Oral MEK inhibitor, in previously treated patients with advanced non-small cell lung cancerClin Cancer Res20101682450245710.1158/1078-0432.CCR-09-19202033232740.YehTCMarshVBernatBABallardJColwellHEvansRJParryJSmithDBrandhuberBJGrossSMarlowAHurleyBLyssikatosJLeePAWinklerJDKochKWallaceEBiological characterization of ARRY-142886 (AZD6244), a potent, highly selective mitogen-activated protein kinase kinase 1/2 inhibitorClin Cancer Res20071351576158310.1158/1078-0432.CCR-06-11501733230441.YapTABjerkeLClarkePAWorkmanPDrugging PI3K in cancer: refining targets and therapeutic strategiesCurr Opin Pharmacol2015239810710.1016/j.coph.2015.05.0162611781942.SarkerDAngJEBairdRKristeleitRShahKMorenoVClarkePARaynaudFILevyGWareJAMazinaKLinRWuJFredricksonJSpoerkeJMLacknerMRYanYFriedmanLSKayeSBDerynckMKWorkmanPde BonoJSFirst-in-human phase I study of pictilisib (GDC-0941), a potent pan-class I phosphatidylinositol-3-kinase (PI3K) inhibitor, in patients with advanced solid tumorsClin Cancer Res2015211778610.1158/1078-0432.CCR-14-09472537047143.BendellJCRodonJBurrisHAde JongeMVerweijJBirleDDemanseDDe BuckSSRuQCPetersMGoldbrunnerMBaselgaJPhase I, dose-escalation study of BKM120, an oral pan-Class I PI3K inhibitor, in patients with advanced solid tumorsJ Clin Oncol201230328229010.1200/JCO.2011.36.13602216258944.HongDSBowlesDWFalchookGSMessersmithWAGeorgeGCO\\u2019BryantCLVoACKlucherKHerbstRSEckhardtSGPetersonSHausmanDFKurzrockRJimenoAA multicenter phase I trial of PX-866, an oral irreversible phosphatidylinositol 3-kinase inhibitor, in patients with advanced solid tumorsClin Cancer Res201218154173418210.1158/1078-0432.CCR-12-07142269335745.ShapiroGIRodonJBedellCKwakELBaselgaJBranaIPandyaSSScheffoldCLairdADNguyenLTXuYEgileCEdelmanGPhase I safety, pharmacokinetic, and pharmacodynamic study of SAR245408 (XL147), an oral pan-class I PI3K inhibitor, in patients with advanced solid tumorsClin Cancer Res201420123324510.1158/1078-0432.CCR-13-17772416690346.FosterPYamaguchiKHsuPPQianFDuXWuJWonKAYuPJaegerCTZhangWMarloweCKKeastPAbulafiaWChenJYoungJPlonowskiAYakesFMChuFEngellKBentzienFLamSTDaleSYturraldeOMatthewsDJLambPLairdADThe selective PI3K inhibitor XL147 (SAR245408) inhibits tumor growth and survival and potentiates the activity of chemotherapeutic agents in preclinical tumor modelsMol Cancer Ther201514493194010.1158/1535-7163.MCT-14-08332563731447.BurgerMTPecchiSWagmanANiZJKnappMHendricksonTAtallahGPfisterKZhangYBartulisSFrazierKNgSSmithAVerhagenJHaznedarJHuhKIwanowiczEXinXMenezesDMerrittHLeeIWiesmannMKaufmanSCrawfordKChinMBussiereDShoemakerKZarorIMairaSMVolivaCFIdentification of NVP-BKM120 as a potent, selective, orally bioavailable class I PI3 kinase inhibitor for treating cancerACS Med Chem Lett201121077477910.1021/ml200156t2490026648.OhwadaJEbiikeHKawadaHTsukazakiMNakamuraMMiyazakiTMorikamiKYoshinariKYoshidaMKondohOKuramotoSOgawaKAokiYShimmaNDiscovery and biological activity of a novel class I PI3K inhibitor, CH5132799Bioorg Med Chem Lett20112161767177210.1016/j.bmcl.2011.01.0652131622949.LiuNRowleyBRBullCOSchneiderCHaegebarthASchatzCAFracassoPRWilkieDPHentemannMWilhelmSMScottWJMumbergDZiegelbauerKBAY 80-6946 is a highly selective intravenous PI3K inhibitor with potent p110alpha and p110delta activities in tumor cell lines and xenograft modelsMol Cancer Ther201312112319233010.1158/1535-7163.MCT-12-0993-T2417076750.IhleNTWilliamsRChowSChewWBerggrenMIPaine-MurrietaGMinionDJHalterRJWipfPAbrahamRKirkpatrickLPowisGMolecular pharmacology and antitumor activity of PX-866, a novel inhibitor of phosphoinositide-3-kinase signalingMol Cancer Ther2004377637721525213751.KongDYamoriTAdvances in development of phosphatidylinositol 3-kinase inhibitorsCurr Med Chem200916222839285410.2174/0929867097888032221968926752.ShapiroGLoRussoPKwakELClearyJMMusibLJonesCde CrespignyABelvinMMcKenzieMGatesMRChanITBendellJCClinical combination of the MEK inhibitor GDC-0973 and the PI3K inhibitor GDC-0941: a first-in-human phase Ib study testing daily and intermittent dosing schedules in patients with advanced solid tumorsJ Clin Oncol201129(suppl; abstr)300553.ChooEFNgCMBerryLBelvinMLewin-KohNMerchantMSalphatiLPK-PD modeling of combination efficacy effect from administration of the MEK inhibitor GDC-0973 and PI3K inhibitor GDC-0941 in A2058 xenograftsCancer Chemother Pharmacol201371113314310.1007/s00280-012-1988-62305327054.BedardPLTaberneroJJankuFWainbergZAPaz-AresLVansteenkisteJVan CutsemEPerez-GarciaJStathisABrittenCDLeNCarterKDemanseDCsonkaDPetersMZubelANauwelaertsHSessaCA phase Ib dose-escalation study of the oral pan-PI3K inhibitor buparlisib (BKM120) in combination with the oral MEK1/2 inhibitor trametinib (GSK1120212) in patients with selected advanced solid tumorsClin Cancer Res201521473073810.1158/1078-0432.CCR-14-18142550005755.RoperJSinnamonMJCoffeeEMBelmontPKeungLGeorgeon-RichardLWangWVFaberACYunJYilmazOHBronsonRTMartinESTsichlisPNHungKECombination PI3K/MEK inhibition promotes tumor apoptosis and regression in PIK3CA wild-type KRAS mutant colorectal cancerCancer Lett2014347220421110.1016/j.canlet.2014.02.0182457662156.MartinelliETroianiTD\\u2019AiutoEMorgilloFVitaglianoDCapassoACostantinoSCiuffredaLPMerollaFVecchioneLDe VriendtVTejparSNappiASforzaVMartiniGBerrinoLDe PalmaRCiardielloFAntitumor activity of pimasertib, a selective MEK 1/2 inhibitor, in combination with PI3K/mTOR inhibitors or with multi-targeted kinase inhibitors in pimasertib-resistant human lung and colorectal cancer cellsInt J Cancer201313392089210110.1002/ijc.282362362972757.KinrossKMBrownDVKleinschmidtMJacksonSChristensenJCullinaneCHicksRJJohnstoneRWMcArthurGAIn vivo activity of combined PI3K/mTOR and MEK Inhibition in a KrasG12D; Pten deletion mouse model of ovarian cancerMol Cancer Ther20111081440144910.1158/1535-7163.MCT-11-02402163246358.FaberACLiDSongYLiangM-CYeapBYBronsonRTLifshitsEChenZMairaS-MGarcia-EcheverriaCWongK-KEngelmanJADifferential induction of apoptosis in HER2 and EGFR addicted cancers following PI3K inhibitionProc Nat Acad Sci200910646195031950810.1073/pnas.09050561061985086959.ShimizuTTolcherAWPapadopoulosKPBeeramMRascoDWSmithLSGunnSSmetzerLMaysTAKaiserBWickMJAlvarezCCavazosAMangoldGLPatnaikAThe clinical effect of the dual-targeting strategy involving PI3K/AKT/mTOR and RAS/MEK/ERK pathways in patients with advanced cancerClin Cancer Res20121882316232510.1158/1078-0432.CCR-11-23812226180060.Grilley-OlsonJEBedardPLFasoloACornfeldMCarteeLRazakARStaynerLAWuYGreenwoodRSinghRLeeCBBendellJBurrisHADel ConteGSessaCInfanteJRA phase Ib dose-escalation study of the MEK inhibitor trametinib in combination with the PI3K/mTOR inhibitor GSK2126458 in patients with advanced solid tumorsInvest New Drugs201627450049', u'issn': u'0344-5704', u'providerFullName': u'Pub Med', u'providerName': u'pubmed', u'author': [u'Haagensen,Emma J.', u'Thomas,Huw D.', u'Schmalix,Wolfgang A.', u'Payne,Andrew C.', u'Kevorkian,Lara', u'Allen,Rodger A.', u'Bevan,Paul', u'Maxwell,Ross J.', u'Newell,David R.'], u'timeStamp': u'2017-03-14T21:46:47Z', u'abstract': u'Purpose\\nTumours frequently have defects in multiple oncogenic pathways, e.g. MAPK and PI3K signalling pathways, and combinations of targeted therapies may be required for optimal activity. This study evaluated the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037, as single agents and in combination, in colorectal carcinoma cell lines and tumour xenograft-bearing mice.\\n\\n\\nMethods\\nIn vitro growth inhibition, survival and signal transduction were measured using the Sulforhodamine B, clonogenic and Western blotting assays, respectively, in HCT116 and HT29 cell lines. In vivo anti-tumour efficacy and pharmacokinetic properties were assessed in HCT116 and HT29 human colorectal cancer xenograft tumour-bearing mice.\\n\\n\\nResults\\nThe combination of WX-554 and WX-037 exhibited marked synergistic growth inhibition in vitro, which was associated with increased cytotoxicity and enhanced inhibition of ERK and S6 phosphorylation, compared to either agent alone. Pharmacokinetic analyses indicated that there was no PK interaction between the two drugs at low doses, but that at higher doses, WX-037 may delay the tumour uptake of WX-554. In vivo efficacy studies revealed that the combination of WX-037 and WX-554 was non-toxic and exhibited marked tumour growth inhibition greater than observed with either agent alone.\\n\\n\\nConclusion\\nThese studies show for the first time that combination treatment with the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037 can induce synergistic growth inhibition in vitro, which translates into enhanced anti-tumour efficacy in vivo.\\n\\n\\nElectronic supplementary material\\nThe online version of this article (doi:10.1007/s00280-016-3186-4) contains supplementary material, which is available to authorized users.', u'title': u'Enhanced anti-tumour activity of the combination of the novel MEK inhibitor WX-554 and the novel PI3K inhibitor WX-037', u'publishDate': u'2016-11-11T00:00:00Z', u'doi': u'10.1007/s00280-016-3186-4', u'externalId': u'27837257::5114336::3186::10.1007/s00280-016-3186-4', u'referenceFiles': True, u'pmc': u'5114336', u'publicationName': u'Cancer Chemotherapy and Pharmacology', u'pmid': u'27837257', u'_id': u'pubmed_27837257__5114336__3186__10_1007_s00280_016_3186_4_pubmed'}]\n"
     ]
    }
   ],
   "source": [
    "print get_IBM_pmc_text(\"27837257\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pubmed Query: \n",
      "(\"KRAS\"[Title/Abstract] OR \"KRAS1\"[Title/Abstract]) AND (\"trametinib\"[Title/Abstract] OR \"gsk1120212\"[Title/Abstract] OR \"n-(3-{3-cyclopropyl-5-[(2-fluoro-4-iodophenyl)amino]-6,8-dimethyl-2,4,7-trioxo-3,4,6,7-tetrahydropyrido[4,3-d]pyrimidin-1(2h)-yl}phenyl)acetamide\"[Title/Abstract] OR \"jtp-74057\"[Title/Abstract] OR \"mekinist\"[Title/Abstract] OR \"mek inhibitor gsk1120212\"[Title/Abstract])\n",
      "PMC Query: \n",
      "(\"KRAS\"[Body - All Words] OR \"KRAS1\"[Body - All Words]) AND (\"trametinib\"[Body - All Words] OR \"gsk1120212\"[Body - All Words] OR \"n-(3-{3-cyclopropyl-5-[(2-fluoro-4-iodophenyl)amino]-6,8-dimethyl-2,4,7-trioxo-3,4,6,7-tetrahydropyrido[4,3-d]pyrimidin-1(2h)-yl}phenyl)acetamide\"[Body - All Words] OR \"jtp-74057\"[Body - All Words] OR \"mekinist\"[Body - All Words] OR \"mek inhibitor gsk1120212\"[Body - All Words])\n",
      "511\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\npubmed_query = make_query(\"Title/Abstract\", gene_symbol, therapy_id, None, None, None)\\nprint pubmed_query\\npubmed_ids = pu.search_pubmed(pubmed_query, retmax=100000)\\nprint len(pubmed_ids)\\nfiltered = dict()\\nfor article in pubmed_articles:\\n    abstract = article.get(\"AB\")\\n    if abstract and outcome_terms_re.search(abstract):\\n        #filtered.append(article[\"PMID\"])\\n        filtered[article[\"PMID\"]] = article\\n        \\nprint len(filtered)\\n'"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_symbol = \"KRAS\"\n",
    "therapy_id = \"DB08911\"\n",
    "(pmids, pmc_to_pmid_dict) = search(gene_symbol, therapy_id, None, None, None)\n",
    "print len(pmids)\n",
    "pubmed_articles = dict( [ (x[\"PMID\"], x) for x in pu.get_medline(list(pmids)) if x[\"PMID\"] != \"\" ] )\n",
    "'''\n",
    "pubmed_query = make_query(\"Title/Abstract\", gene_symbol, therapy_id, None, None, None)\n",
    "print pubmed_query\n",
    "pubmed_ids = pu.search_pubmed(pubmed_query, retmax=100000)\n",
    "print len(pubmed_ids)\n",
    "filtered = dict()\n",
    "for article in pubmed_articles:\n",
    "    abstract = article.get(\"AB\")\n",
    "    if abstract and outcome_terms_re.search(abstract):\n",
    "        #filtered.append(article[\"PMID\"])\n",
    "        filtered[article[\"PMID\"]] = article\n",
    "        \n",
    "print len(filtered)\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_entities(text):\n",
    "    model_id = \"48e69a3e-b31c-43fe-bdf9-6dffae15ec67\"\n",
    "    api_key = \"e0759019e81f7776b95cbebf8b2f2ffe0244a3f9\"\n",
    "    url = \"https://gateway-a.watsonplatform.net/calls/text/TextGetRankedNamedEntities?apikey=%s&verbose=1&model=%s\"\n",
    "    data = {\"outputMode\" : \"json\", \"text\" : text}\n",
    "    return requests.post(url % (api_key, model_id), data=data).json()[\"entities\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "496\n",
      "316\n"
     ]
    }
   ],
   "source": [
    "print len(pmc_to_pmid_dict)\n",
    "non_reviews = []\n",
    "for (pmid, article) in pubmed_articles.items():\n",
    "    pub_types = article.get(\"PT\")\n",
    "    if \"Review\" not in pub_types:\n",
    "        non_reviews.append(pmid)\n",
    "        \n",
    "print len(non_reviews)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0: 24895460\n",
      "1: 28149837\n",
      "2: 28094001\n",
      "3: 26791782\n",
      "4: 27747084\n",
      "5: 28261333\n",
      "6: 27747083\n",
      "7: 26110550\n",
      "8: 27837257\n",
      "9: 27690220\n",
      "10: 24987059\n",
      "11: 27556948\n",
      "12: 21779504\n",
      "13: 27035903\n",
      "14: 26396549\n",
      "15: 24436047\n",
      "16: 25934522\n",
      "17: 24423918\n",
      "18: 22628411\n",
      "19: 26802155\n",
      "20: 23360111\n",
      "21: 24204737\n",
      "22: 25882375\n",
      "23: 25915534\n",
      "24: 26664139\n",
      "25: 26582713\n",
      "26: 28115009\n",
      "27: 24737952\n",
      "28: 25935754\n",
      "29: 22983396\n",
      "30: 24574860\n",
      "31: 26627007\n",
      "32: 25985019\n",
      "33: 27267993\n",
      "34: 28149764\n",
      "35: 22536370\n",
      "36: 24576830\n",
      "37: 26360058\n",
      "38: 26483300\n",
      "39: 26824010\n",
      "40: 27245685\n",
      "41: 24612015\n",
      "42: 26257864\n",
      "43: 26959608\n",
      "44: 26380542\n",
      "45: 26347206\n",
      "46: 26324703\n",
      "47: 27147897\n",
      "48: 26918901\n",
      "49: 26621741\n",
      "50: 25431423\n",
      "51: 25127139\n",
      "52: 22773810\n",
      "53: 26316818\n",
      "54: 25107706\n",
      "55: 24481312\n",
      "56: 24192036\n",
      "57: 26730180\n",
      "58: 26691657\n",
      "59: 26105199\n",
      "60: 23436801\n",
      "61: 27003990\n",
      "62: 26777152\n",
      "63: 26091043\n",
      "64: 27431571\n",
      "65: 26556430\n",
      "66: 27071537\n",
      "67: 24229709\n",
      "68: 23416860\n",
      "69: 26860843\n",
      "70: 26225238\n",
      "71: 25329694\n",
      "72: 27043212\n",
      "73: 26842989\n",
      "74: 27663730\n",
      "75: 27139190\n",
      "76: 28350009\n",
      "77: 23588493\n",
      "78: 25632386\n",
      "79: 26822978\n",
      "80: 26822154\n",
      "81: 26410082\n",
      "82: 25130256\n",
      "83: 22609219\n",
      "84: 27725879\n",
      "85: 24900312\n",
      "86: 28152546\n",
      "87: 28086948\n",
      "88: 26810587\n",
      "89: 24642617\n",
      "90: 27997540\n",
      "91: 26981779\n",
      "92: 27096871\n",
      "93: 22961667\n",
      "94: 24413269\n",
      "95: 25700679\n",
      "96: 25616432\n",
      "97: 21495172\n",
      "98: 28367253\n",
      "99: 26858988\n",
      "100: 25278770\n",
      "101: 25884512\n",
      "102: 24847879\n",
      "103: 24356096\n",
      "104: 25535693\n",
      "105: 22484192\n",
      "106: 24569458\n",
      "107: 26846975\n",
      "108: 25191268\n",
      "109: 26056603\n",
      "110: 27340376\n",
      "111: 19317917\n",
      "112: 24387342\n",
      "113: 25277205\n",
      "114: 26025932\n",
      "115: 24908424\n",
      "116: 25422890\n",
      "117: 24012403\n",
      "118: 23039341\n",
      "119: 27053975\n",
      "120: 25955301\n",
      "121: 28149844\n",
      "122: 26137449\n",
      "123: 27441499\n",
      "124: 27027238\n",
      "125: 25379021\n",
      "126: 21962474\n",
      "127: 23515890\n",
      "128: 27912760\n",
      "129: 24025354\n",
      "130: 25309914\n",
      "131: 27498862\n",
      "132: 23503600\n",
      "133: 27713119\n",
      "134: 25422355\n",
      "135: 27145457\n",
      "136: 26350096\n",
      "137: 21479172\n",
      "138: 24743024\n",
      "139: 25506379\n",
      "140: 23644885\n",
      "141: 28212293\n",
      "142: 25351745\n",
      "143: 27747093\n",
      "144: 25201346\n",
      "145: 27797970\n",
      "146: 27747092\n",
      "147: 27273450\n",
      "148: 24265153\n",
      "149: 27659017\n",
      "150: 25806345\n",
      "151: 22798288\n",
      "152: 27345413\n",
      "153: 23323143\n",
      "154: 25656898\n",
      "155: 26090424\n",
      "156: 25883647\n",
      "157: 23323148\n",
      "158: 27610011\n",
      "159: 25361812\n",
      "160: 26851181\n",
      "161: 26561209\n",
      "162: 26137463\n",
      "163: 25520658\n",
      "164: 25520871\n",
      "165: 26142706\n",
      "166: 27111917\n",
      "167: 25132745\n",
      "168: 26922062\n",
      "169: 24782887\n",
      "170: 25303301\n",
      "171: 26439496\n",
      "172: 23642906\n",
      "173: 25925381\n",
      "174: 26431495\n",
      "175: 24790649\n",
      "176: 28145866\n",
      "177: 25673644\n",
      "178: 24651269\n",
      "179: 24847881\n",
      "180: 25361808\n",
      "181: 26202952\n",
      "182: 23833300\n",
      "183: 25117978\n",
      "184: 23696960\n",
      "185: 27922010\n",
      "186: 25887099\n",
      "187: 21253292\n",
      "188: 26084293\n",
      "189: 24956168\n",
      "190: 25323927\n",
      "191: 26668062\n",
      "192: 28220839\n",
      "193: 23841999\n",
      "194: 25302162\n",
      "195: 26566875\n",
      "196: 25186968\n",
      "197: 26288737\n",
      "198: 25769717\n",
      "199: 25597784\n",
      "200: 24261963\n",
      "201: 21098728\n",
      "202: 23844038\n",
      "203: 28090569\n",
      "204: 25895919\n",
      "205: 23599677\n",
      "206: 26420268\n",
      "207: 24746704\n",
      "208: 27486983\n",
      "209: 26617477\n",
      "210: 24922006\n",
      "211: 25847954\n",
      "212: 25855137\n",
      "213: 27279233\n",
      "214: 28054963\n",
      "215: 27191992\n",
      "216: 25724524\n",
      "217: 28260909\n",
      "218: 28097440\n",
      "219: 25795921\n",
      "220: 26314551\n",
      "221: 27167191\n",
      "222: 25659581\n",
      "223: 26158630\n",
      "224: 27822414\n",
      "225: 26208478\n",
      "226: 27608848\n",
      "227: 22087839\n",
      "228: 23288408\n",
      "229: 26322273\n",
      "230: 22004085\n",
      "231: 23577104\n",
      "232: 26615130\n",
      "233: 25789707\n",
      "234: 27746978\n",
      "235: 25183853\n",
      "236: 23712190\n",
      "237: 26483612\n",
      "238: 22886659\n",
      "239: 26862489\n",
      "240: 24688783\n",
      "241: 22419879\n",
      "242: 21317449\n",
      "243: 25705882\n",
      "244: 24978597\n",
      "245: 23131389\n",
      "246: 25505733\n",
      "247: 26266637\n",
      "248: 25425580\n",
      "249: 21483017\n",
      "250: 27144064\n",
      "251: 23160324\n",
      "252: 25801978\n",
      "253: 26018876\n",
      "254: 22157296\n",
      "255: 23515407\n",
      "256: 26186022\n",
      "257: 24664307\n",
      "258: 25706985\n",
      "259: 25012986\n",
      "260: 24966667\n",
      "261: 26369631\n",
      "262: 27200298\n",
      "263: 25257766\n",
      "264: 20149254\n",
      "265: 26798437\n",
      "266: 24599935\n",
      "267: 26605312\n",
      "268: 26605311\n",
      "269: 26301689\n",
      "270: 26557003\n",
      "271: 28025559\n",
      "272: 22805292\n",
      "273: 23841470\n",
      "274: 26658964\n",
      "275: 27846884\n",
      "276: 28135039\n",
      "277: 26977879\n",
      "278: 22787415\n",
      "279: 26619946\n",
      "280: 27642639\n",
      "281: 23153539\n",
      "282: 27433281\n",
      "283: 25665005\n",
      "284: 27313062\n",
      "285: 26984351\n",
      "286: 27054332\n",
      "287: 27452461\n",
      "288: 23843700\n",
      "289: 25897431\n",
      "290: 26498038\n",
      "291: 25813020\n",
      "292: 27938382\n",
      "293: 25678824\n",
      "294: 27747087\n",
      "295: 24135425\n",
      "296: 27680702\n",
      "297: 26385075\n",
      "298: 26075998\n",
      "299: 26508880\n",
      "300: 26338018\n",
      "301: 26515496\n",
      "302: 25170609\n",
      "303: 25944621\n",
      "304: 27699178\n",
      "305: 26919617\n",
      "306: 26299952\n",
      "307: 24425783\n",
      "308: 26832408\n",
      "309: 27102149\n",
      "310: 25157335\n",
      "311: 27684555\n",
      "312: 25480501\n",
      "313: 25863335\n",
      "314: 25755680\n",
      "315: 22175026\n",
      "316: 26524482\n",
      "317: 25506209\n",
      "318: 26404379\n",
      "319: 27650277\n",
      "320: 21505228\n",
      "321: 25077070\n",
      "322: 26011384\n",
      "323: 26139106\n",
      "324: 25600339\n",
      "325: 27059373\n",
      "326: 25772366\n",
      "327: 26358384\n",
      "328: 25148578\n",
      "329: 25782066\n",
      "330: 26018524\n",
      "331: 27390608\n",
      "332: 23667175\n",
      "333: 27013865\n",
      "334: 25753158\n",
      "335: 27793177\n",
      "336: 26408255\n",
      "337: 27338794\n",
      "338: 25117068\n",
      "339: 25973541\n",
      "340: 26629424\n",
      "341: 24772266\n",
      "342: 26117819\n",
      "343: 23530058\n",
      "344: 26811624\n",
      "345: 27748834\n",
      "346: 25133424\n",
      "347: 25902737\n",
      "348: 27144341\n",
      "349: 27308344\n",
      "350: 26882569\n",
      "351: 24416617\n",
      "352: 27634878\n",
      "353: 25228144\n",
      "354: 25361007\n",
      "355: 26485762\n",
      "356: 26005705\n",
      "357: 25909218\n",
      "358: 26410619\n",
      "359: 26488003\n",
      "360: 24962318\n",
      "361: 28220783\n",
      "362: 25350589\n",
      "363: 25663694\n",
      "364: 26343582\n",
      "365: 26505547\n",
      "366: 24565585\n",
      "367: 28178529\n",
      "368: 27010960\n",
      "369: 25796376\n",
      "370: 27301426\n",
      "371: 27618791\n",
      "372: 26205736\n",
      "373: 24298448\n",
      "374: 25316811\n",
      "375: 26189368\n",
      "376: 26101870\n",
      "377: 25374620\n",
      "378: 27382311\n",
      "379: 26672083\n",
      "380: 26483879\n",
      "381: 25009801\n",
      "382: 26351322\n",
      "383: 24434212\n",
      "384: 24948110\n",
      "385: 27433082\n",
      "386: 25268375\n",
      "387: 23441129\n",
      "388: 26835368\n",
      "389: 27402485\n",
      "390: 25593991\n",
      "391: 27447490\n",
      "392: 25968887\n",
      "393: 27835901\n",
      "394: 27915478\n",
      "395: 25619724\n",
      "396: 26439693\n",
      "397: 25821557\n",
      "398: 26819503\n",
      "399: 25277503\n",
      "400: 25083302\n",
      "401: 24121492\n",
      "402: 25435907\n",
      "403: 27228072\n",
      "404: 26525348\n",
      "405: 27896217\n",
      "406: 27397505\n",
      "407: 27229005\n",
      "408: 24108405\n",
      "409: 25385327\n",
      "410: 26644315\n",
      "411: 24982846\n",
      "412: 27983698\n",
      "413: 26673580\n",
      "414: 26466763\n",
      "415: 27275478\n",
      "416: 24204165\n",
      "417: 26657862\n",
      "418: 27636997\n",
      "419: 27406232\n",
      "420: 26034349\n",
      "421: 23069660\n",
      "422: 25722381\n",
      "423: 24651437\n",
      "424: 25536104\n",
      "425: 26469692\n",
      "426: 26697199\n",
      "427: 25051360\n",
      "428: 26697198\n",
      "429: 23085539\n",
      "430: 24535670\n",
      "431: 27231576\n",
      "432: 25806238\n",
      "433: 28203297\n",
      "434: 28203293\n",
      "435: 25714017\n",
      "436: 26080442\n",
      "437: 26149458\n",
      "438: 26136684\n",
      "439: 23431193\n",
      "440: 27231564\n",
      "441: 26544513\n",
      "442: 25180764\n",
      "443: 25806225\n",
      "444: 25919140\n",
      "445: 25839008\n",
      "446: 26121087\n",
      "447: 23454899\n",
      "448: 24957944\n",
      "449: 25553081\n",
      "450: 28163892\n",
      "451: 26341525\n",
      "452: 28194436\n",
      "453: 27034805\n",
      "454: 25365225\n",
      "455: 24457962\n",
      "456: 26228812\n",
      "457: 27081080\n",
      "458: 24722523\n",
      "459: 26980737\n",
      "460: 26650445\n",
      "461: 27099521\n"
     ]
    },
    {
     "ename": "ValueError",
     "evalue": "No JSON object could be decoded",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-27-36f21f1c78c7>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     31\u001b[0m         \u001b[0mdiseases\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     32\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mabstract\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;34m\"\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 33\u001b[0;31m             \u001b[0mentities\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mget_entities\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mabstract\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     34\u001b[0m             \u001b[0mdiseases\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mentity\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"text\"\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mentity\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mentities\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0mentity\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"type\"\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;34m'Cancer_Entity'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     35\u001b[0m         \u001b[0mvariants_in_abstract\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m<ipython-input-22-d43b2c52c1c1>\u001b[0m in \u001b[0;36mget_entities\u001b[0;34m(text)\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0murl\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m\"https://gateway-a.watsonplatform.net/calls/text/TextGetRankedNamedEntities?apikey=%s&verbose=1&model=%s\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m\"outputMode\"\u001b[0m \u001b[0;34m:\u001b[0m \u001b[0;34m\"json\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"text\"\u001b[0m \u001b[0;34m:\u001b[0m \u001b[0mtext\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 6\u001b[0;31m     \u001b[0;32mreturn\u001b[0m \u001b[0mrequests\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpost\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0murl\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mapi_key\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmodel_id\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjson\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"entities\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m/Library/Python/2.7/site-packages/requests/models.pyc\u001b[0m in \u001b[0;36mjson\u001b[0;34m(self, **kwargs)\u001b[0m\n\u001b[1;32m    806\u001b[0m                     \u001b[0;31m# used.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    807\u001b[0m                     \u001b[0;32mpass\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 808\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0mcomplexjson\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mloads\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtext\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    809\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    810\u001b[0m     \u001b[0;34m@\u001b[0m\u001b[0mproperty\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.pyc\u001b[0m in \u001b[0;36mloads\u001b[0;34m(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)\u001b[0m\n\u001b[1;32m    337\u001b[0m             \u001b[0mparse_int\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0mNone\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mparse_float\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0mNone\u001b[0m \u001b[0;32mand\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    338\u001b[0m             parse_constant is None and object_pairs_hook is None and not kw):\n\u001b[0;32m--> 339\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0m_default_decoder\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdecode\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    340\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mcls\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    341\u001b[0m         \u001b[0mcls\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mJSONDecoder\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.pyc\u001b[0m in \u001b[0;36mdecode\u001b[0;34m(self, s, _w)\u001b[0m\n\u001b[1;32m    362\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    363\u001b[0m         \"\"\"\n\u001b[0;32m--> 364\u001b[0;31m         \u001b[0mobj\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mraw_decode\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0midx\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0m_w\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    365\u001b[0m         \u001b[0mend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_w\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    366\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mend\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.pyc\u001b[0m in \u001b[0;36mraw_decode\u001b[0;34m(self, s, idx)\u001b[0m\n\u001b[1;32m    380\u001b[0m             \u001b[0mobj\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mscan_once\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0midx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    381\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mStopIteration\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 382\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mValueError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"No JSON object could be decoded\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    383\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mobj\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mValueError\u001b[0m: No JSON object could be decoded"
     ]
    }
   ],
   "source": [
    "with open(os.path.join(base_dir, \"LOE_Output_%s_%s-NEW.tsv\" % (gene_symbol, therapy_id)), \"wb\") as f:\n",
    "    f.write(\"PMID\\tPMC ID\\tOutcome terms\\tCancer types\\tVariants (Abstract)\\tVariants (Body)\\tTitle\\tPublication Type\\n\")\n",
    "    count_10 = 0\n",
    "    count = 0\n",
    "    for (pmc_id, pmid) in [ (x, y) for (x, y) in pmc_to_pmid_dict.items() if y in pubmed_articles ]:\n",
    "        print \"%s: %s\" % (count, pmid)\n",
    "        count_10 += 1\n",
    "        count += 1\n",
    "        if count_10 == 10:\n",
    "            time.sleep(random.randint(10, 20))\n",
    "            count_10 = 0\n",
    "        pubmed_article = pubmed_articles[pmid]\n",
    "        '''\n",
    "        raw = requests.get(\"https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/%s/json\" % pmid).text\n",
    "        result = {}\n",
    "        try:\n",
    "            result = json.loads(raw)\n",
    "        except:\n",
    "            print \"Error\"\n",
    "            result = {\"denotations\" : []}\n",
    "        disease_spans = [(x[\"span\"][\"begin\"], x[\"span\"][\"end\"]) for x in result.get(\"denotations\") if x[\"obj\"].startswith(\"Disease\")]\n",
    "        for (begin, end) in disease_spans:\n",
    "            span = abstract[begin:end]\n",
    "            if cancer_terms_re.search(span):\n",
    "                diseases.add(span)\n",
    "        '''\n",
    "        pmc_response = get_IBM_pmc_text(pmc_id)\n",
    "        #body = get_pmc_text(pmc_id)[\"body\"]\n",
    "        #abstract = result.get(\"text\") or pubmed_article.get(\"AB\") or \"\"\n",
    "        abstract = pubmed_article.get(\"AB\", \"\")\n",
    "        diseases = set()\n",
    "        if abstract != \"\":\n",
    "            entities = get_entities(abstract)\n",
    "            diseases = set([entity[\"text\"] for entity in entities if entity[\"type\"] == 'Cancer_Entity'])\n",
    "        variants_in_abstract = set()\n",
    "        variants_in_body = set()\n",
    "        outcomes = set(outcome_terms_re.findall(abstract))\n",
    "        for regex in all_variant_regex:\n",
    "            match = regex.search(abstract)\n",
    "            if match: variants_in_abstract.add(match.group())\n",
    "                \n",
    "            if pmc_response:\n",
    "                body = pmc_response[0].get(\"body\", \"\")\n",
    "                match = regex.search(body)\n",
    "                if match: variants_in_body.add(match.group())\n",
    "\n",
    "        row = [pmid, pmc_id, \", \".join(outcomes), \", \".join(diseases), \", \".join(variants_in_abstract), \n",
    "               \", \".join(variants_in_body), pubmed_article.get(\"TI\"), \", \".join(pubmed_article.get(\"PT\")), \"\\n\"]\n",
    "        f.write(\"\\t\".join(row))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "pmid = \"23938291\"\n",
    "result = requests.get(\"https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/%s/json\" % pmid).json()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{u'sourceid': u'23938291', u'sourcedb': u'PubMed', u'denotations': [{u'span': {u'begin': 144, u'end': 151}, u'obj': u'Species:9606'}, {u'span': {u'begin': 2195, u'end': 2202}, u'obj': u'Species:9606'}, {u'span': {u'begin': 194, u'end': 198}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 203, u'end': 207}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 47, u'end': 51}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 64, u'end': 68}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 69, u'end': 73}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 384, u'end': 388}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 398, u'end': 402}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 415, u'end': 419}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 424, u'end': 428}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 560, u'end': 564}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 577, u'end': 581}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 1316, u'end': 1320}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 1366, u'end': 1370}, u'obj': u'Gene:3845'}, {u'span': {u'begin': 2078, u'end': 2082}, u'obj': u'Gene:1956'}, {u'span': {u'begin': 173, u'end': 192}, u'obj': u'Disease:C538231'}, {u'span': {u'begin': 84, u'end': 103}, u'obj': u'Disease:C538231'}, {u'span': {u'begin': 455, u'end': 470}, u'obj': u'Disease:D000230'}, {u'span': {u'begin': 529, u'end': 544}, u'obj': u'Disease:D000230'}, {u'span': {u'begin': 613, u'end': 619}, u'obj': u'Disease:D009369'}, {u'span': {u'begin': 1347, u'end': 1353}, u'obj': u'Disease:D009369'}, {u'span': {u'begin': 1379, u'end': 1385}, u'obj': u'Disease:D009369'}, {u'span': {u'begin': 1645, u'end': 1659}, u'obj': u'Disease:D000230'}, {u'span': {u'begin': 1683, u'end': 1689}, u'obj': u'Disease:D009369'}, {u'span': {u'begin': 2005, u'end': 2010}, u'obj': u'Disease:D009369'}, {u'span': {u'begin': 2109, u'end': 2124}, u'obj': u'Disease:D000230'}], u'text': u'Identification of transcriptional subgroups in EGFR-mutated and EGFR/KRAS wild-type lung adenocarcinoma reveals gene signatures associated with patient outcome. PURPOSE: In lung adenocarcinoma, EGFR and KRAS mutations dominate the mutational spectrum and have clear therapeutic implications. We sought to determine whether transcriptional subgroups of clinical relevance exist within EGFR-mutated, KRAS-mutated, or EGFR and KRAS wild-type (EGFRwt/KRASwt) adenocarcinomas. EXPERIMENTAL DESIGN: Gene expression profiles from 1,186 adenocarcinomas, including 215 EGFR-mutated, 84 KRAS-mutated, and 219 EGFRwt/KRASwt tumors, were assembled and divided into four discovery (n = 522) and four validation cohorts (n = 664). Subgroups within the mutation groups were identified by unsupervised consensus clustering, significance analysis of microarrays (SAM) analysis, and centroid classification across discovery cohorts. Genomic alterations in identified mutation subgroups were assessed by integration of genomic profiles for 158 cases with concurrent data. Multicohort expression subgroup predictors were built for each mutation group using the discovery cohorts, and validated in the four validation cohorts. RESULTS: Consensus clustering within the mutation groups identified reproducible transcriptional subgroups in EGFR-mutated and EGFRwt/KRASwt tumors, but not in KRAS-mutated tumors. Subgroups displayed differences in genomic alterations, clinicopathologic characteristics, and overall survival. Multicohort gene signatures derived from the mutation subgroups added independent prognostic information in their respective mutation group, for adenocarcinoma in general and stage I tumors specifically, irrespective of mutation status, when applied to the validation cohorts. Consistent with their worse clinical outcome, high-risk subgroups showed higher expression of proliferation-related genes, higher frequency of copy number alterations/amplifications, and association with a poorly differentiated tumor phenotype. CONCLUSIONS: We identified transcriptional subgroups in EGFR-mutated and EGFRwt/KRASwt adenocarcinomas with significant differences in clinicopathologic characteristics and patient outcome, not limited to a mutation-specific setting.'}\n",
      "lung adenocarcinoma\n",
      "lung adenocarcinoma\n",
      "adenocarcinomas\n",
      "adenocarcinomas\n",
      "tumors\n",
      "tumors\n",
      "tumors\n",
      "adenocarcinoma\n",
      "tumors\n",
      "tumor\n",
      "adenocarcinomas\n"
     ]
    }
   ],
   "source": [
    "print result\n",
    "text = result[\"text\"]\n",
    "#entities_span = [x[\"span\"] for x in result[\"denotations\"]]\n",
    "disease_spans = [(x[\"span\"][\"begin\"], x[\"span\"][\"end\"]) for x in result[\"denotations\"] if x[\"obj\"].startswith(\"Disease\")]\n",
    "for (begin, end) in disease_spans:\n",
    "    span = text[begin:end]\n",
    "    print span\n",
    "#mutation_entities = [x for x in entities if x.startswith(\"Mutation\")]\n",
    "#print disease_spans\n",
    "#print mutation_entities"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "GENES=[\"AKT\", \"ALK\", \"KRAS\", \"KIT\", \"MTOR\"]\n",
    "#THERAPIES = \"ARQ751 OR AZD5363 OR GSK2141795\"\n",
    "THERAPIES=[\"AZD5363\", \"AP26113\", \"Binimetinib\", \"Imatinib\", \"Temsirolimus\"]\n",
    "VARIANTS=[\"E17K\", \"EML4-ALK\", \"G12C\", \"W557G\", \"T1977K\"]\n",
    "DISEASES=[\"breast cancer\", \"lung cancer\", \"lung cancer\", \"Gastrointestinal Stromal Tumor\", \"Renal Clear Cell Carcinoma\"]\n",
    "#query = \"AKT1 AND E17K AND (%s)\" % (cancer_terms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "html = requests.get(\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872785/\").text\n",
    "soup = bs(html,'html.parser')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "sections = soup.find_all(\"div\", class_=\"sec\")\n",
    "components = {\"abstract\" : \"\", \"body\" : \"\"}\n",
    "abs_re = re.compile(\"abstract\", re.IGNORECASE)\n",
    "ref_re = re.compile(\"references\", re.IGNORECASE)\n",
    "\n",
    "for section in sections:\n",
    "    sec_title = section.find(\"h2\")\n",
    "    if sec_title:\n",
    "        title = sec_title.get_text()\n",
    "        parent = sec_title.parent\n",
    "        if abs_re.search(title):\n",
    "            components[\"abstract\"] = parent.get_text()\n",
    "        elif not ref_re.search(title):\n",
    "            components[\"body\"] += parent.get_text()\n",
    "            \n",
    "print components[\"abstract\"]\n",
    "print \"\\n\\n\\n\\n\"\n",
    "print components[\"body\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pubmed Query: \n",
      "(\"BRAF\"[Title/Abstract] OR \"BRAF1\"[Title/Abstract]) AND (\"selumetinib\"[Title/Abstract] OR \"mek inhibitor azd6244\"[Title/Abstract] OR \"azd6244\"[Title/Abstract] OR \"arry-142886\"[Title/Abstract] OR \"mek inhibitor azd6244\"[Title/Abstract]) AND (\"ovarian cancer\"[Title/Abstract] OR \"malignant ovarian neoplasm\"[Title/Abstract] OR \"cancer of ovary\"[Title/Abstract] OR \"cancer of the ovary\"[Title/Abstract] OR \"malignant neoplasm of ovary\"[Title/Abstract] OR \"malignant neoplasm of the ovary\"[Title/Abstract] OR \"malignant ovarian tumor\"[Title/Abstract] OR \"malignant tumor of ovary\"[Title/Abstract] OR \"malignant tumor of the ovary\"[Title/Abstract] OR \"ovarian cancer, nos\"[Title/Abstract]) AND (\"CUL1-BRAF\"[Title/Abstract]) AND (\"fusion gene\"[Title/Abstract])\n",
      "PMC Query: \n",
      "(\"BRAF\"[Body - All Words] OR \"BRAF1\"[Body - All Words]) AND (\"selumetinib\"[Body - All Words] OR \"mek inhibitor azd6244\"[Body - All Words] OR \"azd6244\"[Body - All Words] OR \"arry-142886\"[Body - All Words] OR \"mek inhibitor azd6244\"[Body - All Words]) AND (\"ovarian cancer\"[Body - All Words] OR \"malignant ovarian neoplasm\"[Body - All Words] OR \"cancer of ovary\"[Body - All Words] OR \"cancer of the ovary\"[Body - All Words] OR \"malignant neoplasm of ovary\"[Body - All Words] OR \"malignant neoplasm of the ovary\"[Body - All Words] OR \"malignant ovarian tumor\"[Body - All Words] OR \"malignant tumor of ovary\"[Body - All Words] OR \"malignant tumor of the ovary\"[Body - All Words] OR \"ovarian cancer, nos\"[Body - All Words]) AND (\"CUL1-BRAF\"[Body - All Words]) AND (\"fusion gene\"[Body - All Words])\n",
      "Union size: 0\n",
      "Filtered size: 0\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#query = \"AKT1 AND E17K AND AZD5363\"\n",
    "#(union, filtered) = search(\"ALK\", \"5017\", \"Non-Small Cell Lung Cancer\", \"EML4-ALK\", \"fusion gene\") # Good example\n",
    "(union, filtered) = search(\"BRAF\", \"1081\", \"Ovarian Cancer\", \"CUL1-BRAF\", \"fusion gene\") # Nothing found\n",
    "#(union, filtered) = search(\"BRAF\", \"1081\", \"Glioma\", \"KIAA1549-BRAF\", \"fusion gene\")\n",
    "#(union, filtered) = search(\"BRAF\", \"726\", \"melanoma\", \"V600E\", \"mutation\") # XLS PMID refers to non-PUBMED/PMC content\n",
    "#(union, filtered) = search(\"ROS1\", \"5167\", \"Non-Small Cell Lung Cancer\", \"TPM3-ROS1\", \"fusion gene\") # XLS PMID is incorrect. That publication uses Crizotinib\n",
    "#(union, filtered) = search(\"TSC1\", \"DB01590\", \"hepatocellular carcinoma\", \"\", \"\")\n",
    "print \"Union size: %d\" % len(union)\n",
    "print \"Filtered size: %d\" % len(filtered)\n",
    "print \"\\n\".join(filtered)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "for i in range(5):\n",
    "    query = \"%s AND %s\" % (GENES[i], VARIANTS[i])\n",
    "    ids = pu.search_pubmed(query, retmax=100000)\n",
    "    articles = pu.get_medline(ids)\n",
    "    print \"#%s\" % (query)\n",
    "    for article in articles:\n",
    "        if \"AB\" in article:\n",
    "            abstract = article[\"AB\"]\n",
    "            if outcome_terms_re.search(abstract) and patient_terms_re.search(abstract):\n",
    "                #print \"%s: Match found - %s\" % (article[\"PMID\"], \", \".join(outcome_terms_re.findall(abstract)))\n",
    "                print \"%s\\t%s\\t%s\" % (article[\"PMID\"], \", \".join(article[\"PT\"]), article[\"TI\"])\n",
    "    print"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Library/Python/2.7/site-packages/requests/packages/urllib3/connection.py:266: SubjectAltNameWarning: Certificate for watsonpow01.rch.stglabs.ibm.com has no `subjectAltName`, falling back to check for a `commonName` for now. This feature is being removed by major browsers and deprecated by RFC 2818. (See https://github.com/shazow/urllib3/issues/497 for details.)\n",
      "  SubjectAltNameWarning\n"
     ]
    }
   ],
   "source": [
    "journal = \"geno-grp\"\n",
    "doi = \"10.1016/j.cllc.2015.11.004\" # we have access to Elsevier article \"10.1016/j.cllc.2015.11.004\"\n",
    "response = requests.get(\"https://watsonpow01.rch.stglabs.ibm.com/services/content/api/v1/content/%s/fetch?userID=genomics&token=w4genom&field=doi&value=%s&version=2017-03-18\" % (journal, doi)).json()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "serial\n",
      "JL\n",
      "Clinical Lung Cancer\n",
      "CLINICALLUNGCANCER\n",
      "eng\n",
      "JP__|__FR__|__US__|__ES__|__AU\n",
      "\n",
      " Hematology, Oncology and Palliative Medicine\n",
      " Pulmonary/Respiratory\n",
      "\n",
      "280645\n",
      "999999\n",
      "fla\n",
      "article\n",
      "CKIBMJ00000001072/15257304/v17i5/S1525730415002685/main.xml\n",
      "Elsevier Inc.\n",
      "S1525-7304(15)00268-5\n",
      "S1525730415002685\n",
      "1-s2.0-S1525730415002685\n",
      "10.1016/j.cllc.2015.11.004\n",
      "S300\n",
      "S300.1\n",
      "FULL-TEXT\n",
      "2016-10-06T18:07:34\n",
      "2016-10-06T18:07:34\n",
      "2016-10-06T18:07:34\n",
      "NONSQUAMOUSNONSMALLCELLLUNGCANCERPATIENTSCARRYADOUBLEMUTATIONEGFREMLALKKRASFREQUENCYCLINICALPATHOLOGICALCHARACTERISTICSRESPONSETHERAPY\n",
      "ULIVI\n",
      "P\n",
      "\n",
      " \n",
      " TANAKA\n",
      " T\n",
      " 2010\n",
      " 651\n",
      " 655\n",
      " \n",
      " \n",
      " MARCHETTI\n",
      " A\n",
      " 2005\n",
      " 857\n",
      " 865\n",
      " \n",
      " \n",
      " MOK\n",
      " T\n",
      " 2009\n",
      " 947\n",
      " 957\n",
      " \n",
      " \n",
      " ROSELL\n",
      " R\n",
      " 2012\n",
      " 239\n",
      " 246\n",
      " \n",
      " \n",
      " YANG\n",
      " J\n",
      " 2015\n",
      " 141\n",
      " 151\n",
      " \n",
      " \n",
      " SODA\n",
      " M\n",
      " 2007\n",
      " 561\n",
      " 566\n",
      " \n",
      " \n",
      " SHAW\n",
      " A\n",
      " 2013\n",
      " 2385\n",
      " 2394\n",
      " \n",
      " \n",
      " SHAW\n",
      " A\n",
      " 2014\n",
      " 1189\n",
      " 1197\n",
      " \n",
      " \n",
      " GADGEEL\n",
      " S\n",
      " 2014\n",
      " 1119\n",
      " 1128\n",
      " \n",
      " \n",
      " ROTHSCHILD\n",
      " S\n",
      " 2014\n",
      " 379\n",
      " 381\n",
      " \n",
      " \n",
      " TAM\n",
      " I\n",
      " 2006\n",
      " 1647\n",
      " 1653\n",
      " \n",
      " \n",
      " LEE\n",
      " W\n",
      " 2010\n",
      " 473\n",
      " 477\n",
      " \n",
      " \n",
      " ZHANG\n",
      " Y\n",
      " 2015\n",
      " 61\n",
      " null\n",
      " \n",
      " \n",
      " METRO\n",
      " G\n",
      " 2014\n",
      " 86\n",
      " 92\n",
      " \n",
      " \n",
      " SHEPHERD\n",
      " F\n",
      " 2013\n",
      " 2173\n",
      " 2181\n",
      " \n",
      " \n",
      " GAINOR\n",
      " J\n",
      " 2013\n",
      " 4273\n",
      " 4281\n",
      " \n",
      " \n",
      " LI\n",
      " Y\n",
      " 2013\n",
      " e52093\n",
      " null\n",
      " \n",
      " \n",
      " SHAW\n",
      " A\n",
      " 2009\n",
      " 4247\n",
      " 4253\n",
      " \n",
      " \n",
      " ROSSING\n",
      " H\n",
      " 2013\n",
      " 489\n",
      " null\n",
      " \n",
      " \n",
      " BALDI\n",
      " L\n",
      " 2014\n",
      " 291\n",
      " 295\n",
      " \n",
      " \n",
      " POPAT\n",
      " S\n",
      " 2011\n",
      " 1962\n",
      " 1963\n",
      " \n",
      " \n",
      " KUO\n",
      " Y\n",
      " 2010\n",
      " 2039\n",
      " 2040\n",
      " \n",
      " \n",
      " TISEO\n",
      " M\n",
      " 2011\n",
      " 241\n",
      " 243\n",
      " \n",
      " \n",
      " ZHANG\n",
      " X\n",
      " 2010\n",
      " 188\n",
      " null\n",
      " \n",
      " \n",
      " LEE\n",
      " J\n",
      " 2012\n",
      " 460\n",
      " 463\n",
      " \n",
      " \n",
      " WON\n",
      " J\n",
      " 2015\n",
      " 348\n",
      " 354\n",
      " \n",
      " \n",
      " ULIVI\n",
      " P\n",
      " 2013\n",
      " 708\n",
      " 710\n",
      " \n",
      " \n",
      " CABILLIC\n",
      " F\n",
      " 2014\n",
      " 295\n",
      " 306\n",
      " \n",
      " \n",
      " ROSSI\n",
      " G\n",
      " 2015\n",
      " 1035\n",
      " 1036\n",
      " \n",
      " \n",
      " MARUSYK\n",
      " A\n",
      " 2012\n",
      " 323\n",
      " 334\n",
      " \n",
      " \n",
      " YANG\n",
      " J\n",
      " 2014\n",
      " 1383\n",
      " 1392\n",
      " \n",
      " \n",
      " MAR\n",
      " N\n",
      " 2015\n",
      " 220\n",
      " 225\n",
      " \n",
      " \n",
      " LUK\n",
      " P\n",
      " 2015\n",
      " 142\n",
      " 148\n",
      " \n",
      " \n",
      " TUFMAN\n",
      " A\n",
      " 2014\n",
      " 109\n",
      " 113\n",
      " \n",
      " \n",
      " SASAKI\n",
      " T\n",
      " 2010\n",
      " 1773\n",
      " 1780\n",
      " \n",
      " \n",
      " CARGNELUTTI\n",
      " M\n",
      " 2015\n",
      " 5182\n",
      " 5194\n",
      " \n",
      "\n",
      "\n",
      "\n",
      "1525-7304\n",
      "15257304\n",
      "\n",
      "1-s2.0-S1525730416X00058\n",
      "17\n",
      "5\n",
      "Volume 17, Issue 5\n",
      "\n",
      "201609\n",
      "\n",
      "September 2016\n",
      "2016-09-01\n",
      "2016-09-30\n",
      "2016\n",
      "11\n",
      "384\n",
      "390\n",
      "\n",
      "384\n",
      "390\n",
      "\n",
      "Original Studiesitem\n",
      "S1525-7304(15)00268-5\n",
      "S1525730415002685\n",
      "1-s2.0-S1525730415002685\n",
      "10.1016/j.cllc.2015.11.004\n",
      "2016-10-06T18:07:34\n",
      "/15257304/S1525730416X00058/S1525730415002685/\n",
      "\n",
      "201609\n",
      "\n",
      "September 2016\n",
      "2016-09-01\n",
      "2016-09-30\n",
      "2016\n",
      "\n",
      "\n",
      "15257304/S1525730416X00058/cov200h.gif\n",
      "IMAGE-COVER-H200\n",
      "\n",
      "\n",
      "15257304/S1525730416X00058/cov150h.gif\n",
      "IMAGE-COVER-H150\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "1-s2.0-S1525730415002685-main.pdf\n",
      "15257304/S1525730416X00058/S1525730415002685/main.pdf\n",
      "pdf\n",
      "true\n",
      "201020\n",
      "MAIN\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "  \n",
      "    CLLC\n",
      "    421\n",
      "    S1525-7304(15)00268-5\n",
      "    10.1016/j.cllc.2015.11.004\n",
      "    Elsevier Inc.\n",
      "  \n",
      "  \n",
      "    \n",
      "      Table 1\n",
      "      \n",
      "        Clinical-Pathological Characteristics of Patients\n",
      "      \n",
      "      \n",
      "        \n",
      "        \n",
      "        \n",
      "          \n",
      "            Characteristic\n",
      "            n (%)\n",
      "          \n",
      "        \n",
      "        \n",
      "          \n",
      "            \n",
      "              Overall Patient n\n",
      "            \n",
      "            380 (100)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Age, Years\n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              >65\n",
      "            \n",
      "            209 (55)\n",
      "          \n",
      "          \n",
      "            \n",
      "              ≤65\n",
      "            \n",
      "            171 (45)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Sex\n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              Male\n",
      "            \n",
      "            214 (56)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Female\n",
      "            \n",
      "            166 (44)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Smoking Habits\n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              Current\n",
      "            \n",
      "            132 (35)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Former\n",
      "            \n",
      "            104 (27)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Never\n",
      "            \n",
      "            56 (15)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Missing\n",
      "            \n",
      "            88 (23)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Histotype\n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              ADC\n",
      "            \n",
      "            353 (93)\n",
      "          \n",
      "          \n",
      "            \n",
      "              SCC\n",
      "            \n",
      "            2 (1)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Poorly differentiated carcinoma\n",
      "            \n",
      "            25 (6)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Type of Sample\n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              Histological\n",
      "            \n",
      "            251 (66)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Cytological\n",
      "            \n",
      "            129 (34)\n",
      "          \n",
      "        \n",
      "      \n",
      "      \n",
      "        Abbreviations: ADC = adenocarcinoma; SCC = squamous cell carcinoma.\n",
      "      \n",
      "    \n",
      "    \n",
      "      Table 2\n",
      "      \n",
      "        Clinical-Pathological Characteristics of Patients With \n",
      "          EGFR Mutations, \n",
      "          KRAS Mutations, or \n",
      "          EML4-\n",
      "          ALK Translocation\n",
      "        \n",
      "      \n",
      "      \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "          \n",
      "            \n",
      "            \n",
      "              EGFR Mutation\n",
      "            \n",
      "            \n",
      "              EML4-\n",
      "              ALK Translocation\n",
      "            \n",
      "            \n",
      "              KRAS Mutation\n",
      "            \n",
      "          \n",
      "          \n",
      "            Overall\n",
      "            Exon 19 Deletion\n",
      "            Exon 21 (L858R and L861Q)\n",
      "            Exon 18 G719S\n",
      "            FISH Positivity\n",
      "            Overall\n",
      "            G12C\n",
      "            G12V\n",
      "            G12D\n",
      "            Other Codon 12 Mutation\n",
      "            Codon 13 Mutation\n",
      "            Codon 61 Mutation\n",
      "          \n",
      "        \n",
      "        \n",
      "          \n",
      "            \n",
      "              Overall\n",
      "            \n",
      "            44\n",
      "            27 (61)\n",
      "            15 (34)\n",
      "            2 (5)\n",
      "            32\n",
      "            92\n",
      "            44 (48)\n",
      "            19 (21)\n",
      "            9 (10)\n",
      "            9 (10)\n",
      "            7 (8)\n",
      "            4 (4)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Age, Years\n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              >65\n",
      "            \n",
      "            27\n",
      "            17 (63)\n",
      "            9 (33)\n",
      "            1 (4)\n",
      "            14\n",
      "            46\n",
      "            27 (60)\n",
      "            7 (16)\n",
      "            2 (4)\n",
      "            6 (13)\n",
      "            2 (4)\n",
      "            2 (4)\n",
      "          \n",
      "          \n",
      "            \n",
      "              ≤65\n",
      "            \n",
      "            17\n",
      "            10 (59)\n",
      "            6 (35)\n",
      "            1 (6)\n",
      "            18\n",
      "            46\n",
      "            17 (36)\n",
      "            12 (26)\n",
      "            7 (15)\n",
      "            3 (6)\n",
      "            5 (11)\n",
      "            2 (4)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Sex\n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              Male\n",
      "            \n",
      "            14\n",
      "            11 (79)\n",
      "            2 (14)\n",
      "            1 (7)\n",
      "            14\n",
      "            59\n",
      "            31 (53)\n",
      "            10 (17)\n",
      "            4 (7)\n",
      "            8 (14)\n",
      "            3 (5)\n",
      "            3 (5)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Female\n",
      "            \n",
      "            30\n",
      "            16 (53)\n",
      "            13 (43)\n",
      "            1 (3)\n",
      "            18\n",
      "            33\n",
      "            13 (39)\n",
      "            9 (27)\n",
      "            5 (15)\n",
      "            1 (3)\n",
      "            4 (12)\n",
      "            1 (3)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Smoking Habits\n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              Current\n",
      "            \n",
      "            4\n",
      "            1 (25)\n",
      "            2 (50)\n",
      "            1 (25)\n",
      "            10\n",
      "            36\n",
      "            19 (53)\n",
      "            9 (25)\n",
      "            2 (6)\n",
      "            2 (6)\n",
      "            3 (8)\n",
      "            1 (3)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Former\n",
      "            \n",
      "            10\n",
      "            7 (70)\n",
      "            2 (20)\n",
      "            1 (10)\n",
      "            8\n",
      "            36\n",
      "            19 (53)\n",
      "            6 (17)\n",
      "            3 (8)\n",
      "            5 (14)\n",
      "            2 (6)\n",
      "            1 (3)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Never\n",
      "            \n",
      "            18\n",
      "            12 (67)\n",
      "            6 (33)\n",
      "            –\n",
      "            8\n",
      "            3\n",
      "            –\n",
      "            2 (67)\n",
      "            1 (33)\n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "          \n",
      "            \n",
      "              Missing\n",
      "            \n",
      "            12\n",
      "            7 (58)\n",
      "            5 (42)\n",
      "            –\n",
      "            6\n",
      "            17\n",
      "            6 (35)\n",
      "            2 (12)\n",
      "            3 (18)\n",
      "            2 (12)\n",
      "            2 (12)\n",
      "            2 (12)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Histotype\n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              ADC\n",
      "            \n",
      "            44\n",
      "            27 (61)\n",
      "            15 (34)\n",
      "            2 (5)\n",
      "            25\n",
      "            79\n",
      "            36 (46)\n",
      "            19 (24)\n",
      "            7 (9)\n",
      "            9 (12)\n",
      "            4 (5)\n",
      "            4 (5)\n",
      "          \n",
      "          \n",
      "            \n",
      "              Other NSCLC\n",
      "            \n",
      "            –\n",
      "            –\n",
      "            –\n",
      "            –\n",
      "            7\n",
      "            13\n",
      "            8 (57)\n",
      "            –\n",
      "            2 (14)\n",
      "            –\n",
      "            3 (21)\n",
      "            –\n",
      "          \n",
      "        \n",
      "      \n",
      "      \n",
      "        Data are presented as n (%), or n.\n",
      "        Abbreviations: ADC = adenocarcinoma; NSCLC = non–small-cell lung cancer.\n",
      "      \n",
      "    \n",
      "    \n",
      "      Table 3\n",
      "      \n",
      "        Characteristics of Patients With Concomitant Alterations\n",
      "      \n",
      "      \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "        \n",
      "          \n",
      "            Patient Number\n",
      "            Sex\n",
      "            Age, Years\n",
      "            Smoking Habits\n",
      "            Histotype\n",
      "            \n",
      "              EGFR Status\n",
      "            \n",
      "            \n",
      "              EML4-\n",
      "              ALK, %\n",
      "              \n",
      "                a\n",
      "              \n",
      "            \n",
      "            \n",
      "              KRAS Status\n",
      "            \n",
      "            TKI Treatment\n",
      "            Line of TKI Treatment\n",
      "            Best Disease Response\n",
      "            Duration of Response, Months\n",
      "          \n",
      "        \n",
      "        \n",
      "          \n",
      "            \n",
      "              1\n",
      "            \n",
      "            F\n",
      "            72\n",
      "            Never\n",
      "            ADC\n",
      "            E746-A750del\n",
      "            30\n",
      "            wt\n",
      "            Gefitinib\n",
      "            1\n",
      "            CR\n",
      "            >32\n",
      "          \n",
      "          \n",
      "            \n",
      "              2\n",
      "            \n",
      "            F\n",
      "            52\n",
      "            Former\n",
      "            ADC\n",
      "            E746-A750del\n",
      "            33\n",
      "            wt\n",
      "            Gefitinib\n",
      "            1\n",
      "            PR\n",
      "            10\n",
      "              \n",
      "                b\n",
      "              \n",
      "            \n",
      "          \n",
      "          \n",
      "            \n",
      "              3\n",
      "            \n",
      "            F\n",
      "            68\n",
      "            Never\n",
      "            ADC\n",
      "            E746-S752>S\n",
      "            16\n",
      "            wt\n",
      "            Gefitinib\n",
      "            1\n",
      "            PD\n",
      "            2\n",
      "          \n",
      "          \n",
      "            \n",
      "              4\n",
      "            \n",
      "            M\n",
      "            41\n",
      "            Never\n",
      "            ADC\n",
      "            Del 19\n",
      "            37\n",
      "            wt\n",
      "            Gefitinib\n",
      "            1\n",
      "            PD\n",
      "            2\n",
      "          \n",
      "          \n",
      "            \n",
      "              5\n",
      "            \n",
      "            F\n",
      "            73\n",
      "            Former\n",
      "            ADC\n",
      "            E746-A750del\n",
      "            16\n",
      "            wt\n",
      "            Erlotinib\n",
      "            1\n",
      "            PR\n",
      "            40\n",
      "          \n",
      "          \n",
      "            \n",
      "              6\n",
      "            \n",
      "            F\n",
      "            54\n",
      "            Current\n",
      "            ADC\n",
      "            L858R\n",
      "            58\n",
      "            wt\n",
      "            Gefitinib\n",
      "            1\n",
      "            PR\n",
      "            24\n",
      "          \n",
      "          \n",
      "            \n",
      "              7\n",
      "            \n",
      "            M\n",
      "            68\n",
      "            Former\n",
      "            ADC\n",
      "            E746-A750del\n",
      "            0\n",
      "            G12C\n",
      "            Gefitinib\n",
      "            2\n",
      "            SD\n",
      "            6\n",
      "          \n",
      "          \n",
      "            \n",
      "              8\n",
      "            \n",
      "            M\n",
      "            67\n",
      "            Current\n",
      "            ADC\n",
      "            E746-S752 >V\n",
      "            0\n",
      "            G12C\n",
      "            Not treated\n",
      "              \n",
      "                c\n",
      "              \n",
      "            \n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "          \n",
      "            \n",
      "              9\n",
      "            \n",
      "            F\n",
      "            63\n",
      "            Never\n",
      "            Poorly diff.\n",
      "            L858R\n",
      "            0\n",
      "            G12V\n",
      "            Gefitinib\n",
      "            2\n",
      "            SD\n",
      "            14\n",
      "          \n",
      "          \n",
      "            \n",
      "              10\n",
      "            \n",
      "            F\n",
      "            73\n",
      "            Current\n",
      "            ADC\n",
      "            wt\n",
      "            19\n",
      "            G12C\n",
      "            Crizotinib\n",
      "            2\n",
      "            PD\n",
      "            2\n",
      "          \n",
      "          \n",
      "            \n",
      "              11\n",
      "            \n",
      "            M\n",
      "            81\n",
      "            Former\n",
      "            Poorly diff.\n",
      "            wt\n",
      "            22\n",
      "            G12C\n",
      "            Only chemotherapy\n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "          \n",
      "            \n",
      "              12\n",
      "            \n",
      "            F\n",
      "            57\n",
      "            Current\n",
      "            ADC\n",
      "            wt\n",
      "            62\n",
      "            G12V\n",
      "            Crizotinib\n",
      "            2\n",
      "            SD\n",
      "            24\n",
      "          \n",
      "          \n",
      "            \n",
      "              13\n",
      "            \n",
      "            F\n",
      "            46\n",
      "            Current\n",
      "            ADC\n",
      "            wt\n",
      "            30\n",
      "            G12V\n",
      "            Only chemotherapy\n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "          \n",
      "            \n",
      "              14\n",
      "            \n",
      "            F\n",
      "            76\n",
      "            Former\n",
      "            Poorly diff.\n",
      "            wt\n",
      "            34\n",
      "            G13D\n",
      "            Not treated\n",
      "              \n",
      "                c\n",
      "              \n",
      "            \n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "          \n",
      "            \n",
      "              15\n",
      "            \n",
      "            F\n",
      "            77\n",
      "            Former\n",
      "            ADC\n",
      "            wt\n",
      "            48\n",
      "            G12C\n",
      "            Crizotinib\n",
      "            2\n",
      "            PD\n",
      "            1\n",
      "          \n",
      "          \n",
      "            \n",
      "              16\n",
      "            \n",
      "            M\n",
      "            80\n",
      "            Former\n",
      "            ADC\n",
      "            wt\n",
      "            15\n",
      "            G12C\n",
      "            Only chemotherapy\n",
      "            –\n",
      "            –\n",
      "            –\n",
      "          \n",
      "        \n",
      "      \n",
      "      \n",
      "        Abbreviations: ADC = adenocarcinoma; diff. = differentiated; F = female; FISH = fluorescent in situ hybridization; M = male; TKI = tyrosine kinase inhibitor; wt = wild type.\n",
      "      \n",
      "      \n",
      "        a\n",
      "        Evaluated using FISH analysis.\n",
      "      \n",
      "      \n",
      "        b\n",
      "        The patient subsequently underwent surgical resection.\n",
      "      \n",
      "      \n",
      "        c\n",
      "        Not treated because of sudden worsening of clinical conditions.\n",
      "      \n",
      "    \n",
      "  \n",
      "  \n",
      "    \n",
      "      Original Study\n",
      "    \n",
      "    Nonsquamous, Non-Small-Cell Lung Cancer Patients Who Carry a Double Mutation of \n",
      "      EGFR, \n",
      "      EML4-ALK or \n",
      "      KRAS: Frequency, Clinical-Pathological Characteristics, and Response to Therapy\n",
      "    \n",
      "    \n",
      "      \n",
      "        Paola\n",
      "        Ulivi\n",
      "        \n",
      "          1\n",
      "        \n",
      "        \n",
      "          ∗\n",
      "        \n",
      "        paola.ulivi@irst.emr.it\n",
      "      \n",
      "      \n",
      "        Elisa\n",
      "        Chiadini\n",
      "        \n",
      "          1\n",
      "        \n",
      "      \n",
      "      \n",
      "        Claudio\n",
      "        Dazzi\n",
      "        \n",
      "          2\n",
      "        \n",
      "      \n",
      "      \n",
      "        Alessandra\n",
      "        Dubini\n",
      "        \n",
      "          3\n",
      "        \n",
      "      \n",
      "      \n",
      "        Matteo\n",
      "        Costantini\n",
      "        \n",
      "          3\n",
      "        \n",
      "      \n",
      "      \n",
      "        Laura\n",
      "        Medri\n",
      "        \n",
      "          3\n",
      "        \n",
      "      \n",
      "      \n",
      "        Maurizio\n",
      "        Puccetti\n",
      "        \n",
      "          4\n",
      "        \n",
      "      \n",
      "      \n",
      "        Laura\n",
      "        Capelli\n",
      "        \n",
      "          1\n",
      "        \n",
      "      \n",
      "      \n",
      "        Daniele\n",
      "        Calistri\n",
      "        \n",
      "          1\n",
      "        \n",
      "      \n",
      "      \n",
      "        Alberto\n",
      "        Verlicchi\n",
      "        \n",
      "          2\n",
      "        \n",
      "      \n",
      "      \n",
      "        Alessandro\n",
      "        Gamboni\n",
      "        \n",
      "          5\n",
      "        \n",
      "      \n",
      "      \n",
      "        Maximilian\n",
      "        Papi\n",
      "        \n",
      "          6\n",
      "        \n",
      "      \n",
      "      \n",
      "        Marita\n",
      "        Mariotti\n",
      "        \n",
      "          7\n",
      "        \n",
      "      \n",
      "      \n",
      "        Nicoletta\n",
      "        De Luigi\n",
      "        \n",
      "          7\n",
      "        \n",
      "      \n",
      "      \n",
      "        Emanuela\n",
      "        Scarpi\n",
      "        \n",
      "          8\n",
      "        \n",
      "      \n",
      "      \n",
      "        Sara\n",
      "        Bravaccini\n",
      "        \n",
      "          1\n",
      "        \n",
      "      \n",
      "      \n",
      "        Gian Michele\n",
      "        Turolla\n",
      "        \n",
      "          9\n",
      "        \n",
      "      \n",
      "      \n",
      "        Dino\n",
      "        Amadori\n",
      "        \n",
      "          7\n",
      "        \n",
      "      \n",
      "      \n",
      "        Lucio\n",
      "        Crinò\n",
      "        \n",
      "          10\n",
      "        \n",
      "      \n",
      "      \n",
      "        Angelo\n",
      "        Delmonte\n",
      "        \n",
      "          7\n",
      "        \n",
      "      \n",
      "      \n",
      "        1\n",
      "        Biosciences Laboratory, Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS, Meldola, Italy\n",
      "        \n",
      "          Biosciences Laboratory\n",
      "          Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS\n",
      "          Meldola\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        2\n",
      "        Medical Oncology Unit, S.Maria delle Croci Hospital, Ravenna, Italy\n",
      "        \n",
      "          Medical Oncology Unit\n",
      "          S.Maria delle Croci Hospital\n",
      "          Ravenna\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        3\n",
      "        Pathology Unit, Morgagni-Pierantoni Hospital, Forlì, Italy\n",
      "        \n",
      "          Pathology Unit\n",
      "          Morgagni-Pierantoni Hospital\n",
      "          Forlì\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        4\n",
      "        Pathology Unit, S. Maria delle Croci Hospital, Ravenna, Italy\n",
      "        \n",
      "          Pathology Unit\n",
      "          S. Maria delle Croci Hospital\n",
      "          Ravenna\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        5\n",
      "        Medical Oncology Unit, Degli Infermi Hospital, Faenza, Italy\n",
      "        \n",
      "          Medical Oncology Unit\n",
      "          Degli Infermi Hospital\n",
      "          Faenza\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        6\n",
      "        Department of Oncology, Per gli Infermi Hospital, Rimini, Italy\n",
      "        \n",
      "          Department of Oncology\n",
      "          Per gli Infermi Hospital\n",
      "          Rimini\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        7\n",
      "        Medical Oncology Unit, Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS, Meldola, Italy\n",
      "        \n",
      "          Medical Oncology Unit\n",
      "          Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS\n",
      "          Meldola\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        8\n",
      "        Unit of Biostatistics and Clinical Trials, Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS, Meldola, Italy\n",
      "        \n",
      "          Unit of Biostatistics and Clinical Trials\n",
      "          Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS\n",
      "          Meldola\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        9\n",
      "        Oncology Unit, Lugo Hospital, Lugo di Ravenna, Italy\n",
      "        \n",
      "          Oncology Unit\n",
      "          Lugo Hospital\n",
      "          Lugo di Ravenna\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        10\n",
      "        Division of Medical Oncology, Santa Maria della Misericordia Hospital, Perugia, Italy\n",
      "        \n",
      "          Division of Medical Oncology\n",
      "          Santa Maria della Misericordia Hospital\n",
      "          Perugia\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "      \n",
      "        ∗\n",
      "        Address for correspondence: Paola Ulivi, PhD, Biosciences Laboratory, Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS, via Maroncelli 40, 47014 Meldola, Italy. Fax: +39-0543-739221\n",
      "        \n",
      "          Biosciences Laboratory\n",
      "          Istituto Scientifico Romagnolo per lo Studio e la Cura dei Tumori (IRST) IRCCS\n",
      "          via Maroncelli 40\n",
      "          Meldola\n",
      "          47014\n",
      "          Italy\n",
      "        \n",
      "      \n",
      "    \n",
      "    \n",
      "    \n",
      "    \n",
      "    \n",
      "      Abstract\n",
      "      \n",
      "        Background\n",
      "        Epidermal growth factor receptor (\n",
      "          EGFR) and v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog (\n",
      "          KRAS) mutations, and echinoderm microtubule-associated protein-like 4 (\n",
      "          EML4) anaplastic lymphoma kinase (\n",
      "          ALK) translocation are generally considered to be mutually exclusive. However, concomitant mutations are found in a small number of patients and the effect of these on response to targeted therapy is still unknown.\n",
      "        \n",
      "      \n",
      "      \n",
      "        Patients and Methods\n",
      "        We considered 380 non–small-cell lung cancer (NSCLC) patients who underwent nonsequential testing for \n",
      "          EGFR and \n",
      "          EML4-ALK translocation. \n",
      "          KRAS mutation analysis was also performed on 282 patients.\n",
      "        \n",
      "      \n",
      "      \n",
      "        Results\n",
      "        We found 1.6%, 1.1%, and 2.5% of patients who showed a double mutation comprising \n",
      "          EGFR and \n",
      "          EML4-ALK, \n",
      "          EGFR and \n",
      "          KRAS, and \n",
      "          EML4-ALK and \n",
      "          KRAS, respectively. Twenty-eight patients with \n",
      "          EGFR mutation underwent first-line therapy with a tyrosine kinase receptor; a clinical benefit was observed in 81.8% of patients with \n",
      "          EGFR mutations only and in 67% of those who also showed an \n",
      "          EML4-ALK translocation. Twelve patients with an \n",
      "          EML4-ALK translocation received crizotinib and 7 of these had disease progression within 3 months (2 had a concomitant \n",
      "          KRAS mutation and 1 had a concomitant \n",
      "          EGFR mutation). Two patients showed stable disease, 1 of whom also had a \n",
      "          KRAS mutation. Two patients obtained a partial response and 1 had a complete response; all harbored an \n",
      "          EML4-ALK translocation only. The median overall survival of patients who carried an \n",
      "          EML4-ALK translocation alone or concomitant with a \n",
      "          KRAS mutation was 57.1 (range, 10.7-not reached) and 10.7 (range, 4.6-not reached) months, respectively.\n",
      "        \n",
      "      \n",
      "      \n",
      "        Conclusion\n",
      "        Concomitant \n",
      "          EGFR, EML4-ALK, or \n",
      "          KRAS mutations can occur in NSCLC. Concomitant \n",
      "          KRAS mutation and \n",
      "          EML4-ALK translocation represents the most common double alteration and confers a poor prognosis.\n",
      "        \n",
      "      \n",
      "    \n",
      "    \n",
      "      \n",
      "        Micro-Abstract\n",
      "        We assessed the frequency of non–small-cell lung cancer patients with double mutations composed of epidermal growth factor receptor, \n",
      "          v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog (KRAS), or echinoderm microtubule-associated protein-like 4 (\n",
      "          EML4)-anaplastic lymphoma kinase (\n",
      "          ALK) alterations. The most frequent double alteration was \n",
      "          EML4-\n",
      "          ALK translocation and \n",
      "          KRAS mutation, present in approximately 30% of patients with the \n",
      "          EML4-\n",
      "          ALK translocation, which resulted in decreased responsiveness to crizotinib and a poorer prognosis.\n",
      "        \n",
      "      \n",
      "    \n",
      "    \n",
      "      Keywords\n",
      "      \n",
      "        Crizotinib\n",
      "      \n",
      "      \n",
      "        Double mutations\n",
      "      \n",
      "      \n",
      "        NSCLC\n",
      "      \n",
      "      \n",
      "        Prognosis\n",
      "      \n",
      "      \n",
      "        Tyrosine kinase inhibitors\n",
      "      \n",
      "    \n",
      "  \n",
      "  \n",
      "    \n",
      "      \n",
      "        Introduction\n",
      "        Treatment of advanced nonsquamous (NS) non–small-cell lung cancer (NSCLC) patients has changed in recent years as a result of the definition of druggable features and the increased availability of targeted agents. Mutations of the epidermal growth factor receptor (\n",
      "          EGFR) gene are present in approximately 15% of patients in Western countries and in 50% in Asia.\n",
      "          \n",
      "            1,2\n",
      "           These mutations have been shown to be predictive of response to specific tyrosine kinase inhibitors (TKIs) such as gefitinib, erlotinib, or afatinib, which are used as first-line therapy in this setting.\n",
      "          \n",
      "            3-5\n",
      "           Echinoderm microtubule-associated protein-like 4 (\n",
      "          EML4)-anaplastic lymphoma kinase (\n",
      "          ALK) translocation defines another group of patients (approximately 3%-5%)\n",
      "          \n",
      "            6\n",
      "           who are particularly sensitive to targeted therapies such as crizotinib, alectinib, and ceritinib\n",
      "          \n",
      "            7-9\n",
      "          : the first is superior to chemotherapy in first- and advanced-lines of treatment, and the remaining 2 are currently under evaluation and are showing promising activity at recurrence after crizotinib.\n",
      "          \n",
      "            9,10\n",
      "           For these reasons, it is important to determine \n",
      "          EGFR mutations and \n",
      "          EML4-ALK translocation before any treatment decision is made. Finally, \n",
      "          v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations are present in approximately 25% of patients with NS-NSCLC\n",
      "          \n",
      "            11,12\n",
      "          ; data on their ability to predict response to TKIs are conflicting and their correlation with resistance to therapy (TKIs and chemotherapy), has not been unequivocally defined.\n",
      "          \n",
      "            13-15\n",
      "           In general, \n",
      "          EML4-ALK translocation and \n",
      "          EGFR and \n",
      "          KRAS mutations are considered to be mutually exclusive, as reported in several studies.\n",
      "          \n",
      "            16-18\n",
      "           However, some case reports\n",
      "          \n",
      "            19-23\n",
      "           and studies performed prevalently in Asian\n",
      "          \n",
      "            24-26\n",
      "           or Caucasian\n",
      "          \n",
      "            27-29\n",
      "           case series have suggested that these mutations might overlap. Moreover, little is known of the role they play in response to TKIs. In fact, in some studies it was hypothesized that this double alteration might represent a resistance mechanism to TKIs,\n",
      "          \n",
      "            19,23\n",
      "           and others have reported an increased sensitivity to TKIs in patients with an \n",
      "          EGFR mutation and \n",
      "          EML4-ALK translocation.\n",
      "          \n",
      "            21,22\n",
      "           Similarly, few data are available on concomitant \n",
      "          KRAS mutations and \n",
      "          EGFR or \n",
      "          EML4-ALK alterations.\n",
      "        \n",
      "        In the present study we analyzed the prevalence of overlapping mutations among \n",
      "          EGFR, \n",
      "          KRAS, and \n",
      "          EML4-ALK in patients treated for NS-NSCLC and evaluated their clinical response to targeted therapies.\n",
      "        \n",
      "      \n",
      "      \n",
      "        Patients and Methods\n",
      "        We evaluated 380 patients with NS-NSCLC who underwent nonsequential testing for \n",
      "          EGFR and \n",
      "          KRAS mutations and \n",
      "          EML4-ALK translocation in medical oncology units of the wide catchment area of Romagna (Area Vasta Romagna [AVR]) between January 2010 and December 2013. All patients had a histologically or cytologically confirmed diagnosis of NS-NSCLC. The clinical-pathological characteristics of patients are listed in \n",
      "          Table 1\n",
      "          . The study was approved by the AVR institutional review board.\n",
      "        \n",
      "        Formalin-fixed paraffin-embedded (FFPE) histological samples and cytological FFPE (cell block) or cytological smears were available for molecular analysis. Biological samples were evaluated and selected by pathologists from AVR pathology units. Tumor specimens comprised of at least 50% tumor cells were selected and submitted to DNA extraction. Evaluation of \n",
      "          EGFR mutations was performed on the overall case series using direct sequencing (n = 35), pyrosequencing (n = 235), or MassARRAY (Sequenom, San Diego, CA; n = 110) methodologies. \n",
      "          KRAS mutation analysis was carried out in 282 cases using pyrosequencing (n = 172) or MassARRAY (n = 110).\n",
      "        \n",
      "        The status of \n",
      "          EML4-ALK was determined in selected FFPE histological or cytological sections and cytological samples using fluorescence in situ hybridization (FISH) analysis with the Vysis LSI ALK Dual Color, Break Apart Rearrangement Probe (Abbott/Vysis, Downers Grove, IL). With regard to the cytological samples, the FISH analysis was performed on 87 cell blocks and 42 cytological smears. Samples were deemed to be FISH-positive if ≥ 15% of scored tumor cells had split \n",
      "          ALK 5′ and 3′ probe signals or isolated 3′ signals. Slides were evaluated independently by 2 experts blinded to patient history and histological findings.\n",
      "        \n",
      "        \n",
      "          Statistical Analysis\n",
      "          Descriptive statistics were reported as proportions and medians. Response Evaluation Criteria in Solid Tumors guidelines were used to define all responses. Overall survival (OS) was defined as the time from the date of the first treatment cycle to the date of death from any cause. Patients lost to follow-up were censored at the date of last contact. Kaplan–Meier estimates were used to analyze time to event variables and 95% confidence intervals (CIs) were computed. Survival curves were compared using the log rank test. Statistical analyses were carried out using SAS Statistical software (version 9.3; SAS Institute, Cary, NC).\n",
      "        \n",
      "      \n",
      "      \n",
      "        Results\n",
      "        Among the 380 patients analyzed for \n",
      "          EML4-ALK translocation and \n",
      "          EGFR mutations, 32 (8.4%) showed an \n",
      "          EML4-ALK translocation and 44 (11.6%) had an \n",
      "          EGFR mutation: 27 (61.4%) had an exon 19 deletion, 13 (30.0%) an exon 21 L858R mutation (1 of which with a concomitant T790M mutation), 2 (4.6%) an exon 21 L861Q mutation, and 2 (4.6%) an exon 18 G719S mutation. \n",
      "          KRAS mutations were evaluated in 282 patients, 92 of whom (32.6%) showed a mutation: G12C in 44 patients (48%), G12V in 19 (21%), G12D in 9 (10%), G12A in 7 (8%), G12F in 2 (2%), G13C in 3 (3%), G13D in 4 (4%), and Q61H in 4 (4%). Clinical-pathological characteristics of patients with mutations are shown in \n",
      "          Table 2\n",
      "          . The presence of 2 concomitant mutations was observed in 16 patients (\n",
      "          Table 3\n",
      "          ). No statistically significant correlations with clinical features were found.\n",
      "        \n",
      "        Six of the 380 patients (1.6%) showed the concomitant presence of an \n",
      "          EGFR mutation and \n",
      "          EML4-ALK translocation: 5 had an \n",
      "          EGFR exon 19 deletion and 1 had an exon 21 (L858R) point mutation. This group comprised 5 women and 1 men; 3 had never smoked, 2 were former smokers, and 1 was a current smoker (\n",
      "          Table 3). These patients with a double mutation represented 13.6% (6 of 44) and 18.8% (6 of 32) of patients with the EGFR mutation and the \n",
      "          EML4-ALK translocation, respectively.\n",
      "        \n",
      "        Among the 282 patients analyzed for \n",
      "          KRAS mutations, 3 (1.1%) showed a concomitant \n",
      "          EGFR and \n",
      "          KRAS mutation (\n",
      "          Table 3). Two of these had an \n",
      "          EGFR exon 19 deletion with a concomitant G12C \n",
      "          KRAS mutation, and 1 showed an L858R mutation and a G12V \n",
      "          KRAS mutation. Two patients from this group were male (1 current and 1 former smoker) and 1 was female (former smoker; \n",
      "          Table 3). These patients with overlapping \n",
      "          EGFR and \n",
      "          KRAS mutations represented 6.8% (3 of 44) and 3.2% (3 of 92) of the population with single \n",
      "          EGFR or \n",
      "          KRAS mutations, respectively.\n",
      "        \n",
      "        Finally, 7 of the 282 \n",
      "          KRAS-evaluated patients (2.5%) showed the concomitant presence of a \n",
      "          KRAS mutation (4 G12C, 2 G12V, or 1 G13D) and \n",
      "          EML4-ALK translocation (\n",
      "          Table 3). Five of these patients were female (3 current and 2 former smokers) and 2 were male (both former smokers). The median age of patients with the \n",
      "          EML4-ALK translocation with concomitant \n",
      "          KRAS mutation was 76 years (range, 46-81 years), and that of patients with only the \n",
      "          EML4-ALK translocation was 59 years (range, 41-79 years; \n",
      "          P = .133). Patients with overlapping \n",
      "          KRAS mutation and \n",
      "          EML4-ALK translocation represented 29.2% (7 of 24) and 7.6% (7 of 92) of patients in the \n",
      "          EML4-ALK translocation and \n",
      "          KRAS mutation groups, respectively.\n",
      "        \n",
      "        Among the 44 patients with \n",
      "          EGFR mutation, 28 underwent TKI treatment (24 with gefitinib and 4 with erlotinib) as first-line therapy, 7 patients received platinum-based therapy, and 2 patients were not treated. Treatment information was not available for 7 patients. Six of the 28 patients treated with TKIs had a concomitant \n",
      "          EML4-ALK translocation. Of the 22 patients with only an EGFR mutation, 2 (9%) had a complete response (CR), 16 (72.7%) a partial response (PR), and 4 (18%) who had progressive disease (PD). In patients with a double alteration, there was 1 CR (17%), 3 PR (50%), and 2 PD (33%; \n",
      "          P = .289; \n",
      "          Table 3). The clinical benefit to anti-EGFR TKIs, intended as the sum of CR, PR, and stable disease (SD), was 81.7% in patients with a single mutation and 67% in patients with a double mutation. Only 1 patient who harbored a concomitant \n",
      "          EGFR mutation and \n",
      "          EML4-ALK translocation underwent surgery after treatment with gefitinib. No significant differences in term of median duration of response were observed between patients with only \n",
      "          EGFR mutations (16 months; range, 3-42 months) and those with double alterations (27.5 months; range, 10-40 months; \n",
      "          P = .207). No differences in OS were seen between these 2 groups (\n",
      "          P = .934).\n",
      "        \n",
      "        Among the 32 patients who harbored the \n",
      "          EML4-ALK translocation, 6 also carried the \n",
      "          EGFR mutation and were treated with a TKI as first-line therapy, and the remaining 26 received chemotherapy.\n",
      "        \n",
      "        Twelve patients received second-line crizotinib: 7 (58%) showed PD within 3 months of beginning therapy, of whom 2 (17% of the crizotinib group) had a concomitant G12C \n",
      "          KRAS mutation and 1 had a concomitant \n",
      "          EGFR exon 19 deletion for which treatment with an anti-EGFR TKI had previously been given. Two patients, 1 of whom had a G12V \n",
      "          KRAS mutation, had SD for >20 months ( \n",
      "          Table 3). Of the 3 patients with a single \n",
      "          EML4-ALK translocation, 2 had a PR (6 and 12 months), and 1 a CR (6 months). The median OS of patients who carried an \n",
      "          EML4-ALK translocation alone or a concomitant \n",
      "          KRAS mutation was 57.1 (95% CI, 10.7-not reached) and 10.7 (95% CI, 4.6-not reached) months, respectively (\n",
      "          P = .347).\n",
      "        \n",
      "      \n",
      "      \n",
      "        Discussion\n",
      "        The \n",
      "          EGFR and \n",
      "          KRAS mutations and the \n",
      "          EML4-ALK translocations have long been considered mutually exclusive.\n",
      "          \n",
      "            16-18\n",
      "           However, there is increasing evidence that double mutations, albeit rare, can be concomitantly present.\n",
      "          \n",
      "            19-23\n",
      "           This phenomenon can, in fact, be explained by the concept of tumor heterogeneity according to which different mutations of tyrosine kinase receptors might coexist, not only in different clones of tumor cells,\n",
      "          \n",
      "            30\n",
      "           but also in the same tumor cell, as we and others have already shown.\n",
      "          \n",
      "            27,31\n",
      "          \n",
      "        \n",
      "        In the present study we retrospectively evaluated the prevalence of double alterations in a case series of advanced NS-NSCLC patients, and observed that 1.6% of these had concomitant \n",
      "          EGFR and \n",
      "          EML4-ALK modifications, 1.1% had \n",
      "          EGFR and \n",
      "          KRAS alterations, and 2.5% had an \n",
      "          EML4-ALK translocation and \n",
      "          KRAS mutation. The frequency of these events, albeit rare, is similar to that of other druggable alterations such as \n",
      "          v-Raf murine sarcoma viral oncogene homolog B and \n",
      "          HER2 mutations, which are currently being investigated in several clinical trials using specific targeted therapies.\n",
      "          \n",
      "            32,33\n",
      "           Moreover, the importance of these double alterations increases if we consider their percentages within the groups of patients with single mutations: in fact, a double alteration of \n",
      "          EGFR and \n",
      "          EML4-ALK was present in 13.6% and 18.8% of patients with an \n",
      "          EGFR mutation and \n",
      "          EML4-ALK translocation, respectively. The frequency of overlapping \n",
      "          EGFR and \n",
      "          KRAS mutations was 6.8% and 3.2% of the populations with \n",
      "          EGFR and \n",
      "          KRAS mutations, respectively. Finally, patients with both a \n",
      "          KRAS mutation and \n",
      "          EML4-ALK translocation represented 29.2% and 7.6% of patients with an \n",
      "          EML4-ALK translocation and a \n",
      "          KRAS mutation, respectively. Of note, we observed a slightly greater percentage of \n",
      "          EML4-ALK translocation in smokers or former smokers (50%) than that reported in other studies.\n",
      "          \n",
      "            34,35\n",
      "           However, the most of these patients were or had been light smokers.\n",
      "        \n",
      "        Information on the effect of these double mutations on response to therapies is limited. A recent study showed that patients with \n",
      "          EGFR and \n",
      "          EML4-ALK alterations were not sensitive to anti-EGFR TKIs, whereas sensitivity to crizotinib was high.\n",
      "          \n",
      "            26\n",
      "           Although 1 case report in the same setting reported a resistance to erlotinib,\n",
      "          \n",
      "            23\n",
      "           others observed a good response to TKIs.\n",
      "          \n",
      "            21,22\n",
      "          \n",
      "        \n",
      "        In our study no significant correlation in terms of response to first-line anti-EGFR TKIs was found between patients with an \n",
      "          EGFR mutation and those with \n",
      "          EGFR and \n",
      "          EML4-ALK alterations. However, the clinical benefit was 81.7% in patients with a single mutation and 67% in those who showed concomitant alterations. These data, with a lack of statistical significance, which might be because of the limited number of patients, suggest that double mutations might be predictive of no response to anti-EGFR TKIs. Although the same would seem to apply for treatment with crizotinib in this setting, data are too limited to draw any definitive conclusions. In our case series, only 1 patient with a double mutation received second-line crizotinib, but did not respond. No difference in term of progression-free survival or OS were seen among the 2 populations analyzed.\n",
      "        \n",
      "        Limited data are available in the literature on the prevalence and response to specific TKIs in subjects with the concomitant presence of \n",
      "          EML4-ALK translocation and \n",
      "          KRAS mutation. One case report showed that a patient who harbored \n",
      "          EGFR and \n",
      "          KRAS mutations and a \n",
      "          EML4-ALK translocation was resistant to crizotinib.\n",
      "          \n",
      "            19\n",
      "           In a study performed in a large case series it was reported that a concomitant \n",
      "          EML4-ALK translocation and \n",
      "          KRAS mutation was found in 10% of patients, only 1 of whom had been treated with crizotinib. Interestingly, this patient showed an initial response before rapid disease progression.\n",
      "          \n",
      "            28\n",
      "           Recently, Rossi et al\n",
      "          \n",
      "            29\n",
      "           reported that 2 of their 18 patients with an \n",
      "          ALK translocation (11%) also had a \n",
      "          KRAS mutation. However, no data were available on the response to crizotinib in this group. In our study, 3 patients with the double alteration received crizotinib, 2 of whom showed PD and 1 SD. OS was worse, albeit not significantly, in patients with a double mutation than in patients with a single \n",
      "          EML4-ALK translocation (57.1 vs. 10.7 months); the lack of significance was probably because of the low number of cases. Taken together, these data suggest that the double alteration of \n",
      "          KRAS and \n",
      "          EML4-ALK is a marker of poorer prognosis and might be predictive of resistance to anti-ALK agents. This is in agreement with the results from a recent study that showed that the activation of the rat sarcoma viral oncogene homolog family confers resistance to c-ros oncogene 1 inhibitors.\n",
      "          \n",
      "            36\n",
      "          \n",
      "        \n",
      "        However, prospective clinical trials are needed to confirm these hypotheses.\n",
      "      \n",
      "      \n",
      "        Conclusion\n",
      "        Our study showed that concomitant mutations involving \n",
      "          EGFR, \n",
      "          EML4-ALK, and \n",
      "          KRAS are possible, albeit rare, in advanced NS-NSCLC patients and that their presence might have a detrimental effect on response to treatment. For these reasons, the mutational status of NS-NSCLC patients should be determined at the time of diagnosis to allow the best therapeutic strategy to be chosen.\n",
      "        \n",
      "        \n",
      "          Clinical Practice Points\n",
      "          \n",
      "            \n",
      "              \n",
      "                •\n",
      "                Alterations in \n",
      "                  EGFR, \n",
      "                  KRAS, and \n",
      "                  EML4-ALK are usually considered to be mutually exclusive in NSCLC patients. Although a number of studies and case reports have shown that some patients might have concomitant alterations, their effect on response to treatment is not well understood. Most of the literature data describe the effect of concomitant \n",
      "                  EGFR and \n",
      "                  EML4-ALK alterations, whereas no data are available on the concomitant presence of \n",
      "                  EML4-ALK translocation and \n",
      "                  KRAS mutation.\n",
      "                \n",
      "              \n",
      "              \n",
      "                •\n",
      "                In our study the concomitant presence of \n",
      "                  EGFR mutation and \n",
      "                  EML4-ALK translocation was observed in 1.6% of the overall population. This double alteration reduced response to TKI treatment, albeit not significantly. The concomitant presence of \n",
      "                  EML4-ALK translocation and \n",
      "                  KRAS mutation was the most frequent event, present in 2.5% of the overall population and, in particular, in approximately 30% of patients with an \n",
      "                  EML4-ALK translocation. It led to lower responsiveness to crizotinib and a poorer prognosis.\n",
      "                \n",
      "              \n",
      "              \n",
      "                •\n",
      "                Our results could have 2 important consequences for the clinical management of NSCLC. First, it would be advisable to evaluate the presence of the 3 alterations in all patients, not taking for granted an absolute mutual exclusivity between them. Second, if our data are confirmed in a larger case series, the appropriateness of using anti-ALK agents in patients with concomitant \n",
      "                  EML4-ALK translocation and \n",
      "                  KRAS mutation could be brought to question.\n",
      "                \n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "      \n",
      "      \n",
      "        Disclosure\n",
      "        The authors have stated that they have no conflicts of interest.\n",
      "      \n",
      "    \n",
      "    \n",
      "      Acknowledgments\n",
      "      The authors thank Grainne Tierney and Ursula Elbling for editing the manuscript.\n",
      "    \n",
      "  \n",
      "  \n",
      "    \n",
      "      References\n",
      "      \n",
      "        \n",
      "          1\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  T.\n",
      "                  Tanaka\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Matsuoka\n",
      "                \n",
      "                \n",
      "                  A.\n",
      "                  Sutani\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Frequency of and variables associated with the EGFR mutation and its subtypes\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Int J Cancer\n",
      "                  \n",
      "                  126\n",
      "                \n",
      "                2010\n",
      "              \n",
      "              \n",
      "                651\n",
      "                655\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          2\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.\n",
      "                  Marchetti\n",
      "                \n",
      "                \n",
      "                  C.\n",
      "                  Martella\n",
      "                \n",
      "                \n",
      "                  L.\n",
      "                  Felicioni\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                EGFR mutations in non–small-cell lung cancer: analysis of a large series of cases and development of a rapid and sensitive method for diagnostic screening with potential implications on pharmacologic treatment\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Clin Oncol\n",
      "                  \n",
      "                  23\n",
      "                \n",
      "                2005\n",
      "              \n",
      "              \n",
      "                857\n",
      "                865\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          3\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  T.S.\n",
      "                  Mok\n",
      "                \n",
      "                \n",
      "                  Y.L.\n",
      "                  Wu\n",
      "                \n",
      "                \n",
      "                  S.\n",
      "                  Thongprasert\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Gefitinib or carboplatin-paclitaxel in pulmonary adenocarcinoma\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    N Engl J Med\n",
      "                  \n",
      "                  361\n",
      "                \n",
      "                2009\n",
      "              \n",
      "              \n",
      "                947\n",
      "                957\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          4\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  R.\n",
      "                  Rosell\n",
      "                \n",
      "                \n",
      "                  E.\n",
      "                  Carcereny\n",
      "                \n",
      "                \n",
      "                  R.\n",
      "                  Gervais\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Erlotinib versus standard chemotherapy as first-line treatment for European patients with advanced EGFR mutation-positive non–small-cell lung cancer (EURTAC): a multicentre, open-label, randomised phase 3 trial\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lancet Oncol\n",
      "                  \n",
      "                  13\n",
      "                \n",
      "                2012\n",
      "              \n",
      "              \n",
      "                239\n",
      "                246\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          5\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  J.C.\n",
      "                  Yang\n",
      "                \n",
      "                \n",
      "                  Y.L.\n",
      "                  Wu\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Schuler\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Afatinib versus cisplatin-based chemotherapy for EGFR mutation-positive lung adenocarcinoma (LUX-Lung 3 and LUX-Lung 6): analysis of overall survival data from two randomised, phase 3 trials\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lancet Oncol\n",
      "                  \n",
      "                  16\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                141\n",
      "                151\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          6\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  M.\n",
      "                  Soda\n",
      "                \n",
      "                \n",
      "                  Y.L.\n",
      "                  Choi\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Enomoto\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Identification of the transforming \n",
      "                  EML4-ALK fusion gene in non–small-cell lung cancer\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Nature\n",
      "                  \n",
      "                  448\n",
      "                \n",
      "                2007\n",
      "              \n",
      "              \n",
      "                561\n",
      "                566\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          7\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.T.\n",
      "                  Shaw\n",
      "                \n",
      "                \n",
      "                  D.W.\n",
      "                  Kim\n",
      "                \n",
      "                \n",
      "                  K.\n",
      "                  Nakagawa\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Crizotinib versus chemotherapy in advanced ALK-positive lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    N Engl J Med\n",
      "                  \n",
      "                  368\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                2385\n",
      "                2394\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          8\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.T.\n",
      "                  Shaw\n",
      "                \n",
      "                \n",
      "                  D.W.\n",
      "                  Kim\n",
      "                \n",
      "                \n",
      "                  R.\n",
      "                  Mehra\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Ceritinib in ALK-rearranged non–small-cell lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    N Engl J Med\n",
      "                  \n",
      "                  370\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                1189\n",
      "                1197\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          9\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  S.M.\n",
      "                  Gadgeel\n",
      "                \n",
      "                \n",
      "                  L.\n",
      "                  Gandhi\n",
      "                \n",
      "                \n",
      "                  G.J.\n",
      "                  Riely\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Safety and activity of alectinib against systemic disease and brain metastases in patients with crizotinib-resistant ALK-rearranged non–small-cell lung cancer (AF-002JG): results from the dose-finding portion of a phase 1/2 study\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lancet Oncol\n",
      "                  \n",
      "                  15\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                1119\n",
      "                1128\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          10\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  S.I.\n",
      "                  Rothschild\n",
      "                \n",
      "              \n",
      "              \n",
      "                Ceritinib-a second-generation ALK inhibitor overcoming resistance in ALK-rearranged non–small-cell lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Transl Lung Cancer Res\n",
      "                  \n",
      "                  3\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                379\n",
      "                381\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          11\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  I.Y.\n",
      "                  Tam\n",
      "                \n",
      "                \n",
      "                  L.P.\n",
      "                  Chung\n",
      "                \n",
      "                \n",
      "                  W.S.\n",
      "                  Suen\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Distinct epidermal growth factor receptor and KRAS mutation patterns in non–small-cell lung cancer patients with different tobacco exposure and clinicopathologic features\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Clin Cancer Res\n",
      "                  \n",
      "                  12\n",
      "                \n",
      "                2006\n",
      "              \n",
      "              \n",
      "                1647\n",
      "                1653\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          12\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  W.\n",
      "                  Lee\n",
      "                \n",
      "                \n",
      "                  Z.\n",
      "                  Jiang\n",
      "                \n",
      "                \n",
      "                  J.\n",
      "                  Liu\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                The mutation spectrum revealed by paired genome sequences from a lung cancer patient\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Nature\n",
      "                  \n",
      "                  465\n",
      "                \n",
      "                2010\n",
      "              \n",
      "              \n",
      "                473\n",
      "                477\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          13\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  Y.\n",
      "                  Zhang\n",
      "                \n",
      "                \n",
      "                  W.\n",
      "                  Fang\n",
      "                \n",
      "                \n",
      "                  Y.\n",
      "                  Yan\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                The efficacy of first-line chemotherapy is associated with KRAS mutation status in patients with advanced non–small-cell lung cancer: a meta-analysis\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Med Oncol\n",
      "                  \n",
      "                  32\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                61\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          14\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  G.\n",
      "                  Metro\n",
      "                \n",
      "                \n",
      "                  R.\n",
      "                  Chiari\n",
      "                \n",
      "                \n",
      "                  C.\n",
      "                  Bennati\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Clinical outcome with platinum-based chemotherapy in patients with advanced nonsquamous EGFR wild-type non–small-cell lung cancer segregated according to KRAS mutation status\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Clin Lung Cancer\n",
      "                  \n",
      "                  15\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                86\n",
      "                92\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          15\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  F.A.\n",
      "                  Shepherd\n",
      "                \n",
      "                \n",
      "                  C.\n",
      "                  Domerg\n",
      "                \n",
      "                \n",
      "                  P.\n",
      "                  Hainaut\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Pooled analysis of the prognostic and predictive effects of KRAS mutation status and KRAS mutation subtype in early-stage resected non–small-cell lung cancer in four trials of adjuvant chemotherapy\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Clin Oncol\n",
      "                  \n",
      "                  31\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                2173\n",
      "                2181\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          16\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  J.F.\n",
      "                  Gainor\n",
      "                \n",
      "                \n",
      "                  A.M.\n",
      "                  Varghese\n",
      "                \n",
      "                \n",
      "                  S.H.\n",
      "                  Ou\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                ALK rearrangements are mutually exclusive with mutations in EGFR or KRAS: an analysis of 1,683 patients with non–small-cell lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Clin Cancer Res\n",
      "                  \n",
      "                  19\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                4273\n",
      "                4281\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          17\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  Y.\n",
      "                  Li\n",
      "                \n",
      "                \n",
      "                  Y.\n",
      "                  Li\n",
      "                \n",
      "                \n",
      "                  T.\n",
      "                  Yang\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Clinical significance of \n",
      "                  EML4-ALK fusion gene and association with EGFR and KRAS gene mutations in 208 Chinese patients with non–small-cell lung cancer\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    PLoS One\n",
      "                  \n",
      "                  8\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                e52093\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          18\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.T.\n",
      "                  Shaw\n",
      "                \n",
      "                \n",
      "                  B.Y.\n",
      "                  Yeap\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Mino-Kenudson\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Clinical features and outcome of patients with non–small-cell lung cancer who harbor \n",
      "                  EML4-ALK\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Clin Oncol\n",
      "                  \n",
      "                  27\n",
      "                \n",
      "                2009\n",
      "              \n",
      "              \n",
      "                4247\n",
      "                4253\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          19\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  H.H.\n",
      "                  Rossing\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Grauslund\n",
      "                \n",
      "                \n",
      "                  E.M.\n",
      "                  Urbanska\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Concomitant occurrence of EGFR (epidermal growth factor receptor) and KRAS (V-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog) mutations in an ALK (anaplastic lymphoma kinase)-positive lung adenocarcinoma patient with acquired resistance to crizotinib: a case report\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    BMC Res Notes\n",
      "                  \n",
      "                  6\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                489\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          20\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  L.\n",
      "                  Baldi\n",
      "                \n",
      "                \n",
      "                  M.C.\n",
      "                  Mengoli\n",
      "                \n",
      "                \n",
      "                  A.\n",
      "                  Bisagni\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Concomitant EGFR mutation and ALK rearrangement in lung adenocarcinoma is more frequent than expected: report of a case and review of the literature with demonstration of genes alteration into the same tumor cells\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lung Cancer\n",
      "                  \n",
      "                  86\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                291\n",
      "                295\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          21\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  S.\n",
      "                  Popat\n",
      "                \n",
      "                \n",
      "                  A.\n",
      "                  Vieira de Araujo\n",
      "                \n",
      "                \n",
      "                  T.\n",
      "                  Min\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Lung adenocarcinoma with concurrent exon 19 EGFR mutation and ALK rearrangement responding to erlotinib\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Thorac Oncol\n",
      "                  \n",
      "                  6\n",
      "                \n",
      "                2011\n",
      "              \n",
      "              \n",
      "                1962\n",
      "                1963\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          22\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  Y.W.\n",
      "                  Kuo\n",
      "                \n",
      "                \n",
      "                  S.G.\n",
      "                  Wu\n",
      "                \n",
      "                \n",
      "                  C.C.\n",
      "                  Ho\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Good response to gefitinib in lung adenocarcinoma harboring coexisting \n",
      "                  EML4-ALK fusion gene and EGFR mutation\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Thorac Oncol\n",
      "                  \n",
      "                  5\n",
      "                \n",
      "                2010\n",
      "              \n",
      "              \n",
      "                2039\n",
      "                2040\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          23\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  M.\n",
      "                  Tiseo\n",
      "                \n",
      "                \n",
      "                  F.\n",
      "                  Gelsomino\n",
      "                \n",
      "                \n",
      "                  D.\n",
      "                  Boggiani\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                EGFR and \n",
      "                  EML4-ALK gene mutations in NSCLC: a case report of erlotinib-resistant patient with both concomitant mutations\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lung Cancer\n",
      "                  \n",
      "                  71\n",
      "                \n",
      "                2011\n",
      "              \n",
      "              \n",
      "                241\n",
      "                243\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          24\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  X.\n",
      "                  Zhang\n",
      "                \n",
      "                \n",
      "                  S.\n",
      "                  Zhang\n",
      "                \n",
      "                \n",
      "                  X.\n",
      "                  Yang\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Fusion of \n",
      "                  EML4 and ALK is associated with development of lung adenocarcinomas lacking EGFR and KRAS mutations and is correlated with ALK expression\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Mol Cancer\n",
      "                  \n",
      "                  9\n",
      "                \n",
      "                2010\n",
      "              \n",
      "              \n",
      "                188\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          25\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  J.K.\n",
      "                  Lee\n",
      "                \n",
      "                \n",
      "                  T.M.\n",
      "                  Kim\n",
      "                \n",
      "                \n",
      "                  Y.\n",
      "                  Koh\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Differential sensitivities to tyrosine kinase inhibitors in NSCLC harboring EGFR mutation and ALK translocation\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lung Cancer\n",
      "                  \n",
      "                  77\n",
      "                \n",
      "                2012\n",
      "              \n",
      "              \n",
      "                460\n",
      "                463\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          26\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  J.K.\n",
      "                  Won\n",
      "                \n",
      "                \n",
      "                  B.\n",
      "                  Keam\n",
      "                \n",
      "                \n",
      "                  J.\n",
      "                  Koh\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Concomitant ALK translocation and EGFR mutation in lung cancer: a comparison of direct sequencing and sensitive assays and the impact on responsiveness to tyrosine kinase inhibitor\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Ann Oncol\n",
      "                  \n",
      "                  26\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                348\n",
      "                354\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          27\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  P.\n",
      "                  Ulivi\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Puccetti\n",
      "                \n",
      "                \n",
      "                  L.\n",
      "                  Capelli\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Molecular determinations of EGFR and \n",
      "                  EML4-ALK on a single slide of NSCLC tissue\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Clin Pathol\n",
      "                  \n",
      "                  66\n",
      "                \n",
      "                2013\n",
      "              \n",
      "              \n",
      "                708\n",
      "                710\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          28\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  F.\n",
      "                  Cabillic\n",
      "                \n",
      "                \n",
      "                  A.\n",
      "                  Gros\n",
      "                \n",
      "                \n",
      "                  F.\n",
      "                  Dugay\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Parallel FISH and immunohistochemical studies of ALK status in 3244 non–small-cell lung cancers reveal major discordances\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Thorac Oncol\n",
      "                  \n",
      "                  9\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                295\n",
      "                306\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          29\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  G.\n",
      "                  Rossi\n",
      "                \n",
      "                \n",
      "                  L.\n",
      "                  Baldi\n",
      "                \n",
      "                \n",
      "                  F.\n",
      "                  Barbieri\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Concomitant EGFR and KRAS mutations in ALK-rearranged lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Ann Oncol\n",
      "                  \n",
      "                  26\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                1035\n",
      "                1036\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          30\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.\n",
      "                  Marusyk\n",
      "                \n",
      "                \n",
      "                  V.\n",
      "                  Almendro\n",
      "                \n",
      "                \n",
      "                  K.\n",
      "                  Polyak\n",
      "                \n",
      "              \n",
      "              \n",
      "                Intra-tumour heterogeneity: a looking glass for cancer?\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Nat Rev Cancer\n",
      "                  \n",
      "                  12\n",
      "                \n",
      "                2012\n",
      "              \n",
      "              \n",
      "                323\n",
      "                334\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          31\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  J.J.\n",
      "                  Yang\n",
      "                \n",
      "                \n",
      "                  X.C.\n",
      "                  Zhang\n",
      "                \n",
      "                \n",
      "                  J.\n",
      "                  Su\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Lung cancers with concomitant EGFR mutations and ALK rearrangements: diverse responses to EGFR-TKI and crizotinib in relation to diverse receptors phosphorylation\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Clin Cancer Res\n",
      "                  \n",
      "                  20\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                1383\n",
      "                1392\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          32\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  N.\n",
      "                  Mar\n",
      "                \n",
      "                \n",
      "                  J.J.\n",
      "                  Vredenburgh\n",
      "                \n",
      "                \n",
      "                  J.S.\n",
      "                  Wasser\n",
      "                \n",
      "              \n",
      "              \n",
      "                Targeting HER2 in the treatment of non–small-cell lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Lung Cancer\n",
      "                  \n",
      "                  87\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                220\n",
      "                225\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          33\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  P.P.\n",
      "                  Luk\n",
      "                \n",
      "                \n",
      "                  B.\n",
      "                  Yu\n",
      "                \n",
      "                \n",
      "                  C.C.\n",
      "                  Ng\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                BRAF mutations in non–small-cell lung cancer\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Transl Lung Cancer Res\n",
      "                  \n",
      "                  4\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                142\n",
      "                148\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          34\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  A.L.\n",
      "                  Tufman\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Edelmann\n",
      "                \n",
      "                \n",
      "                  F.\n",
      "                  Gamarra\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Preselection based on clinical characteristics in German non–small-cell lung cancer patients screened for \n",
      "                  EML4-ALK translocation\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    J Thorac Oncol\n",
      "                  \n",
      "                  9\n",
      "                \n",
      "                2014\n",
      "              \n",
      "              \n",
      "                109\n",
      "                113\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          35\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  T.\n",
      "                  Sasaki\n",
      "                \n",
      "                \n",
      "                  S.J.\n",
      "                  Rodig\n",
      "                \n",
      "                \n",
      "                  L.R.\n",
      "                  Chirieac\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                The biology and treatment of \n",
      "                  EML4-ALK non–small-cell lung cancer\n",
      "                \n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Eur J Cancer\n",
      "                  \n",
      "                  46\n",
      "                \n",
      "                2010\n",
      "              \n",
      "              \n",
      "                1773\n",
      "                1780\n",
      "              \n",
      "            \n",
      "          \n",
      "        \n",
      "        \n",
      "          36\n",
      "          \n",
      "            \n",
      "              \n",
      "                \n",
      "                  M.\n",
      "                  Cargnelutti\n",
      "                \n",
      "                \n",
      "                  S.\n",
      "                  Corso\n",
      "                \n",
      "                \n",
      "                  M.\n",
      "                  Pergolizzi\n",
      "                \n",
      "                \n",
      "              \n",
      "              \n",
      "                Activation of RAS family members confers resistance to ROS1 targeting drugs\n",
      "              \n",
      "            \n",
      "            \n",
      "              \n",
      "                \n",
      "                  \n",
      "                    Oncotarget\n",
      "                  \n",
      "                  6\n",
      "                \n",
      "                2015\n",
      "              \n",
      "              \n",
      "                5182\n",
      "                5194\n"
     ]
    }
   ],
   "source": [
    "print response[0][\"documentText\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
